{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext memory_profiler"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Short tutorial: context-specific GRN inference from 10x multiome dataset\n",
    "This is a [Dictys](https://github.com/pinellolab/dictys) tutorial to reconstruct and analyze context-specific GRNs from a [10x multiome dataset on PBMC](https://www.10xgenomics.com/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0). For a full version with details of individual steps, see tutorial [full-multiome](https://github.com/pinellolab/dictys/tree/master/doc/tutorials/full-multiome).\n",
    "\n",
    "Before running, you need to **change the runtime configurations** for your machine at [2. Network inference configuration](#2.-Network-inference-configuration) and [3. Network inference](#3.-Network-inference).\n",
    "\n",
    "If you face any issues or need any assistance, see [FAQ](https://github.com/pinellolab/dictys#faq) and [Issues](https://github.com/pinellolab/dictys#issues).\n",
    "\n",
    "By using this notebook, you imply that you have already accepted the terms of use and privacy policy on the [10x dataset webpage](https://www.10xgenomics.com/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0). The dataset summary webpage is also available [here](https://cf.10xgenomics.com/samples/cell-arc/2.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_web_summary.html).\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Preparation of individual input files in `data` folder\n",
    "After data processing with CellRanger, you need the following files:\n",
    "* RNA read count matrix (\"Filtered feature barcode matrix MEX (DIR)\")\n",
    "* ATAC reads (\"ATAC Position-sorted alignments (BAM)\")\n",
    "* Optionally or from other softwares: cell clustering (inside \"Secondary analysis outputs (DIR)\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.conda/envs/dictys/bin:/opt/gridware/apps/gcc/R/4.3.2/bin:/opt/gridware/compilers/gcc/11.2.0/bin:/opt/gridware2/apps/binapps/anaconda3/2019.07/bin:/opt/gridware2/apps/binapps/anaconda3/2019.07/condabin:/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.local/bin:/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/bin:/usr/share/Modules/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "# 更新 PATH 环境变量，添加 Conda 环境的 bin 目录\n",
    "os.environ['PATH'] = os.path.expanduser('~/.conda/envs/dictys/bin') + ':' + os.environ['PATH']\n",
    "\n",
    "# 确保路径已更新\n",
    "print(os.getenv('PATH'))\n",
    "\n",
    "# 尝试运行命令\n",
    "#os.system('dictys_helper expression_mtx.py -h')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys\n"
     ]
    }
   ],
   "source": [
    "# Removes CPU usage limit by some jupyter versions\n",
    "import os\n",
    "os.environ['KMP_AFFINITY'] = ''\n",
    "#Create input data folder\n",
    "current_path = os.getcwd()\n",
    "print(current_path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "current_path = \"/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k\"\n",
    "os.chdir(current_path )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k'"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### expression.tsv.gz\n",
    "Read count matrix of RNA-profiled cells in compressed tsv format. Downloaded and converted from \"Filtered feature barcode matrix MEX (DIR)\"."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:root:Skipped duplicate occurrence of gene names: TBCE,LINC01238,CYB561D2,MATR3,LINC01505,HSPA14,GOLGA8M,GGT1,ARMCX5-GPRASP2,TMSB15B\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Download expression data in mtx.gz format\n",
    "wget -q -o /dev/null https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_3k/pbmc_granulocyte_sorted_3k_filtered_feature_bc_matrix.tar.gz\n",
    "tar xf pbmc_granulocyte_sorted_3k_filtered_feature_bc_matrix.tar.gz\n",
    "rm pbmc_granulocyte_sorted_3k_filtered_feature_bc_matrix.tar.gz\n",
    "# Convert from mtx.gz to tsv.gz format using helper script `expression_mtx.py`.\n",
    "dictys_helper expression_mtx.py filtered_feature_bc_matrix expression.tsv.gz\n",
    "rm -Rf filtered_feature_bc_matrix\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### bams\n",
    "This folder contains one bam file for each cell with chromatin accessibility measurement. File name should be cell name. Downloaded and converted from \"ATAC Position-sorted alignments (BAM)\".\n",
    "* **This step can take hours or even over a day**\n",
    "* The default setting will need ~30GB of memory for this dataset. Specify a lower `--buffer_size 1000` or smaller for `dictys_helper split_bam.sh` if you have less memory.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Setting --progress (progress) to bar:force:noscroll\n",
      "Setting --output-document (outputdocument) to bams.bam\n",
      "DEBUG output created by Wget 1.21.4 on linux-gnu.\n",
      "\n",
      "Reading HSTS entries from /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.wget-hsts\n",
      "URI encoding = ‘UTF-8’\n",
      "--2024-08-01 15:03:26--  https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_3k/pbmc_granulocyte_sorted_3k_atac_possorted_bam.bam\n",
      "Resolving cf.10xgenomics.com (cf.10xgenomics.com)... 104.18.1.173, 104.18.0.173, 2606:4700::6812:ad, ...\n",
      "Caching cf.10xgenomics.com => 104.18.1.173 104.18.0.173 2606:4700::6812:ad 2606:4700::6812:1ad\n",
      "Connecting to cf.10xgenomics.com (cf.10xgenomics.com)|104.18.1.173|:443... connected.\n",
      "Created socket 4.\n",
      "Releasing 0x0000558dffbeaf70 (new refcount 1).\n",
      "Initiating SSL handshake.\n",
      "Handshake successful; connected socket 4 to SSL handle 0x0000558dffbf69d0\n",
      "certificate:\n",
      "  subject: CN=10xgenomics.com\n",
      "  issuer:  CN=WE1,O=Google Trust Services,C=US\n",
      "X509 certificate successfully verified and matches host cf.10xgenomics.com\n",
      "\n",
      "---request begin---\n",
      "GET /samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_3k/pbmc_granulocyte_sorted_3k_atac_possorted_bam.bam HTTP/1.1\n",
      "Host: cf.10xgenomics.com\n",
      "User-Agent: Wget/1.21.4\n",
      "Accept: */*\n",
      "Accept-Encoding: identity\n",
      "Connection: Keep-Alive\n",
      "\n",
      "---request end---\n",
      "HTTP request sent, awaiting response... \n",
      "---response begin---\n",
      "HTTP/1.1 200 OK\n",
      "Date: Thu, 01 Aug 2024 14:03:27 GMT\n",
      "Content-Type: binary/octet-stream\n",
      "Content-Length: 8468267067\n",
      "Connection: keep-alive\n",
      "last-modified: Sun, 06 Sep 2020 06:51:03 GMT\n",
      "etag: \"5a1d2e3a06057ff2fa4ee9c3c27d0e5b-1010\"\n",
      "x-amz-storage-class: INTELLIGENT_TIERING\n",
      "x-amz-version-id: 4c7byHEQTUIqmfXZOpLjwRD9WI2NuY2I\n",
      "accept-ranges: bytes\n",
      "x-cache: Miss from cloudfront\n",
      "via: 1.1 31ef019c1ddee9b68951720b335aec36.cloudfront.net (CloudFront)\n",
      "x-amz-cf-pop: LHR5-P5\n",
      "x-amz-cf-id: uQ18niaNwUmD959phECVqBVhcvs-A33zdwzfuJJt90paaS5piXVuaA==\n",
      "CF-Cache-Status: DYNAMIC\n",
      "Server: cloudflare\n",
      "CF-RAY: 8ac65f24be91ccaa-MAN\n",
      "\n",
      "---response end---\n",
      "200 OK\n",
      "Registered socket 4 for persistent reuse.\n",
      "Length: 8468267067 (7.9G) [binary/octet-stream]\n",
      "Saving to: ‘bams.bam’\n",
      "\n",
      "bams.bam            100%[===================>]   7.89G  7.56MB/s    in 24m 40s \n",
      "\n",
      "2024-08-01 15:28:07 (5.46 MB/s) - ‘bams.bam’ saved [8468267067/8468267067]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "set -eo pipefail\n",
    "cd ./data\n",
    "# Download chromatin accessibility reads in bam format\n",
    "wget --debug --progress=bar:force:noscroll -O bams.bam https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_3k/pbmc_granulocyte_sorted_3k_atac_possorted_bam.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"dictys_helper split_bam.sh bams.bam bams --section CB:Z: --ref_expression expression.tsv.gz\"\n",
      "\tUser time (seconds): 1097.66\n",
      "\tSystem time (seconds): 187.20\n",
      "\tPercent of CPU this job got: 60%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 35:15.91\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 8777140\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 18\n",
      "\tMinor (reclaiming a frame) page faults: 85689530\n",
      "\tVoluntary context switches: 17729036\n",
      "\tInvoluntary context switches: 5392\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 122180616\n",
      "\tFile system outputs: 128438344\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "set -eo pipefail\n",
    "cd ./data\n",
    "/usr/bin/time -v dictys_helper split_bam.sh bams.bam bams --section \"CB:Z:\" --ref_expression expression.tsv.gz\n",
    "rm bams.bam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### subsets & subsets.txt\n",
    "* subsets.txt: Names of cell subsets. For each cell subset, a GRN is reconstructed.\n",
    "* subsets: Folder containing one subfolder for each cell subset as in `subsets.txt`. Each subfolder contains two files:\n",
    "    - names_rna.txt: Names of cells that belong to this subset and have transcriptome measurement\n",
    "    - names_atac.txt: Names of cells that belong to this subset and have chromatin accessibility measurement\n",
    "    - For joint measurements of RNA and ATAC, these two files should be identical in every folder.\n",
    "    \n",
    "Option 1: download the clustering from 10x \"Secondary analysis outputs (DIR)\":"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "wget -q -o /dev/null https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_3k/pbmc_granulocyte_sorted_3k_analysis.tar.gz\n",
    "#Extract cell names for each cluster\n",
    "tar xf pbmc_granulocyte_sorted_3k_analysis.tar.gz \n",
    "mv analysis/clustering/gex/graphclust/clusters.csv clusters.csv\n",
    "rm -Rf pbmc_granulocyte_sorted_3k_analysis.tar.gz analysis\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Option 2: cluster with other softwares (here [scanpy](https://scanpy-tutorials.readthedocs.io/en/latest/pbmc3k.html) below).\n",
    "\n",
    "**Note**:\n",
    "* The cell below can be hidden if you use a renderer other than jupyter (e.g. github).\n",
    "* If you don't have scanpy installed, the kernel would automatically restart to finish scanpy installation. Please continue by running this cell again for clustering."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#Installs scanpy if not present\n",
    "a=!pip show scanpy &> /dev/null; echo $?\n",
    "a=''.join(a)\n",
    "if a=='0':\n",
    "\tprint('Scanpy found. Continuing notebook execution.')\n",
    "elif a=='1':\n",
    "\ta=!pip install 'scanpy[leiden]'; echo $?\n",
    "\ta=''.join(a)\n",
    "\tif a!='0':\n",
    "\t\traise RuntimeError('Installing scanpy failed.')\n",
    "\tprint('Installing scanpy completed. Restarting kernel. Please continue from here.')\n",
    "\timport os\n",
    "\tos._exit(00)\n",
    "else:\n",
    "\traise ValueError('Unknown return for: pip show scanpy')\n",
    "\n",
    "#Clustering with scanpy\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import scanpy as sc\n",
    "adata = sc.read_10x_mtx('../data/filtered_feature_bc_matrix/',var_names='gene_symbols',cache=False)\n",
    "adata.var_names_make_unique()\n",
    "sc.pp.filter_cells(adata, min_genes=200)\n",
    "sc.pp.filter_genes(adata, min_cells=3)\n",
    "adata.var['mt'] = adata.var_names.str.startswith('MT-')  # annotate the group of mitochondrial genes as 'mt'\n",
    "sc.pp.calculate_qc_metrics(adata, qc_vars=['mt'], percent_top=None, log1p=False, inplace=True)\n",
    "sc.pl.scatter(adata, x='total_counts', y='pct_counts_mt')\n",
    "sc.pl.scatter(adata, x='total_counts', y='n_genes_by_counts')\n",
    "adata = adata[adata.obs.n_genes_by_counts < 6000, :]\n",
    "adata = adata[adata.obs.pct_counts_mt < 20, :]\n",
    "sc.pp.normalize_total(adata, target_sum=1e4)\n",
    "sc.pp.log1p(adata)\n",
    "sc.pp.highly_variable_genes(adata, min_mean=0.0125, max_mean=3, min_disp=0.5)\n",
    "sc.pl.highly_variable_genes(adata)\n",
    "adata.raw = adata\n",
    "adata = adata[:, adata.var.highly_variable]\n",
    "sc.pp.regress_out(adata, ['total_counts', 'pct_counts_mt'])\n",
    "sc.pp.scale(adata, max_value=10)\n",
    "sc.tl.pca(adata, svd_solver='arpack')\n",
    "sc.pp.neighbors(adata, n_neighbors=10, n_pcs=40)\n",
    "sc.tl.leiden(adata)\n",
    "adata.obs.leiden.to_csv('../data/clusters.csv',header=['Cluster'],index=True,index_label='Barcode')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Finally, reformat clusters.csv for input:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "subsets=\"$(tail -n +2 clusters.csv | awk -F , '{print $2}' | sort -u)\"\n",
    "echo \"$subsets\" | awk '{print \"Subset\"$1}' > subsets.txt\n",
    "for x in $subsets; do\n",
    "\tmkdir -p \"subsets/Subset$x\"\n",
    "\tgrep \",$x\"'$' clusters.csv | awk -F , '{print $1}' > \"subsets/Subset$x/names_rna.txt\"\n",
    "\t# RNA and ATAC barcodes are the same for joint quantifications\n",
    "\tcp \"subsets/Subset$x/names_rna.txt\" \"subsets/Subset$x/names_atac.txt\"\n",
    "done\n",
    "rm clusters.csv\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Other files\n",
    "Motif, reference genome, gene transcriptional start site"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "\n",
    "# Motifs (file motifs.motif)\n",
    "# Option 1: from HOCOMOCO (https://hocomoco11.autosome.org/)\n",
    "wget -q -o /dev/null -O motifs.motif 'https://hocomoco11.autosome.org/final_bundle/hocomoco11/full/HUMAN/mono/HOCOMOCOv11_full_HUMAN_mono_homer_format_0.0001.motif'\n",
    "# Option 2: from HOMER\n",
    "#dictys_helper motif_homer.sh > motifs.motif\n",
    "# Option 3: provide your custom motifs\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">dKhGCGTGh\tAHR_HUMAN.H11MO.0.B\t3.3775000000000004\r\n",
      "0.262728374765856\t0.1227600511842322\t0.362725638699551\t0.25178593535036087\r\n",
      "0.07633328991810645\t0.08258130543118362\t0.22593295481662123\t0.6151524498340887\r\n",
      "0.14450570038747923\t0.28392173880411337\t0.13815442099009081\t0.4334181398183167\r\n",
      "0.023935814057894068\t0.016203821748029118\t0.9253278681170539\t0.03453249607702277\r\n",
      "0.007919544273173793\t0.953597675415874\t0.017308392078009837\t0.021174388232942286\r\n",
      "0.02956192959210962\t0.012890110758086997\t0.9474192747166682\t0.010128684933135217\r\n",
      "0.007919544273173797\t0.029561929592109615\t0.012337825593096645\t0.9501807005416201\r\n",
      "0.007919544273173793\t0.007919544273173793\t0.9762413671804787\t0.007919544273173793\r\n",
      "0.27886589130660366\t0.4285328543459993\t0.10955683916661985\t0.18304441518077724\r\n",
      ">hnnGGWWnddWWGGdbWh\tAIRE_HUMAN.H11MO.0.C\t5.64711\r\n",
      "0.38551919443239085\t0.2604245534178759\t0.1353299124033618\t0.21872633974637148\r\n",
      "0.18745267949274294\t0.18745267949274294\t0.14575446582123766\t0.4793401751932764\r\n",
      "0.14575446582123777\t0.14575446582123777\t0.2083017863284958\t0.5001892820290286\r\n",
      "0.0623580384782284\t0.020659824806723626\t0.8337749914010671\t0.08320714531398087\r\n",
      "0.020659824806723633\t0.020659824806723633\t0.7712276708938095\t0.18745267949274314\r\n",
      "0.3125473205072573\t0.062358038478228406\t0.04150893164247602\t0.5835857093720382\r\n",
      "0.37509464101451423\t0.10405625214973326\t0.04150893164247601\t0.4793401751932766\r\n"
     ]
    }
   ],
   "source": [
    "!head -n 18 ./data/motifs.motif"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Process is interrupted.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Reference genome (folder genome)\n",
    "# Note: You need the same reference genome version with chromatin accessibility reads\n",
    "# Option 1: download genome from HOMER\n",
    "dictys_helper genome_homer.sh hg38 genome\n",
    "# Option 2: provide your custom genome\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 4.3G\n",
      "113K annotations\n",
      " 64K chrom.sizes\n",
      "3.7G genome.fa\n",
      "4.2M hg38.aug\n",
      " 51M hg38.basic.annotation\n",
      "258M hg38.full.annotation\n",
      "272K hg38.miRNA\n",
      "233M hg38.repeats\n",
      " 16M hg38.rna\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -h1s ./data/genome | head"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Bed file for TSS (file gene.bed)\n",
    "# Download gtf file from ensembl\n",
    "#wget -q -o /dev/null -O gene.gtf.gz http://ftp.ensembl.org/pub/release-107/gtf/homo_sapiens/Homo_sapiens.GRCh38.107.gtf.gz\n",
    "gunzip gene.gtf.gz\n",
    "# Convert to bed\n",
    "dictys_helper gene_gtf.sh gene.gtf gene.bed\n",
    "rm gene.gtf\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1\t11869\t14409\tDDX11L1\t.\t+\r\n",
      "chr1\t14404\t29570\tWASH7P\t.\t-\r\n",
      "chr1\t17369\t17436\tMIR6859-1\t.\t-\r\n",
      "chr1\t29554\t31109\tMIR1302-2HG\t.\t+\r\n",
      "chr1\t30366\t30503\tMIR1302-2\t.\t+\r\n",
      "chr1\t34554\t36081\tFAM138A\t.\t-\r\n",
      "chr1\t52473\t53312\tOR4G4P\t.\t+\r\n",
      "chr1\t57598\t64116\tOR4G11P\t.\t+\r\n",
      "chr1\t65419\t71585\tOR4F5\t.\t+\r\n",
      "chr1\t131025\t134836\tCICP27\t.\t+\r\n"
     ]
    }
   ],
   "source": [
    "!head ./data/gene.bed"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Clean up"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "!rm -Rf ./data/filtered_feature_bc_matrix\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Network inference configuration\n",
    "### Generate configurations\n",
    "Please adjust them for your own machine and dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k'"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "# Generate configuration template\n",
    "rm -Rf ./makefiles\n",
    "mkdir ./makefiles\n",
    "cd ./makefiles\n",
    "dictys_helper makefile_template.sh common.mk config.mk env_none.mk static.mk\n",
    "\n",
    "# Update configurations, such as:\n",
    "# DEVICE: pytorch device, e.g. cpu, cuda:0. If you do not have a GPU, use 'cpu' and expect LONG computing time.\n",
    "# GENOME_MACS2: effective genome size for macs2. See https://deeptools.readthedocs.io/en/develop/content/feature/effectiveGenomeSize.html\n",
    "# JOINT: whether dataset is joint profiling of RNA and ATAC.\n",
    "# Other configurations include quality control thresholds, number of threads in each job, number of hidden confounders, etc.\n",
    "# They can be obtained in the full-multiome tutorial.\n",
    "dictys_helper makefile_update.py ../makefiles/config.mk '{\"DEVICE\": \"cpu\", \"GENOME_MACS2\": \"hs\", \"JOINT\": \"1\"}'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Lingfei Wang, 2022. All rights reserved.\r\n",
      "#This file contains parameters for whole run and individual steps to be edited for your dataset\r\n",
      "#This file should be edited to configure the run\r\n",
      "#This file should NOT be directly used for any run with `makefile -f` \r\n",
      "\r\n",
      "############################################################\r\n",
      "# Run environment settings\r\n",
      "############################################################\r\n",
      "#Which environment to use, corresponding to env_$(ENVMODE).mk file\r\n",
      "ENVMODE=none\r\n",
      "#Maximum number of CPU threads for each job\r\n",
      "#This is only nominated and passed through to other softwares without any guarantee.\r\n",
      "NTH=4\r\n",
      "#Device name for pyro/pytorch\r\n",
      "#Note: cuda devices other than cuda:0 could be incompatible with singularity environment\r\n",
      "DEVICE=cpu\r\n",
      "\r\n",
      "############################################################\r\n",
      "# Dataset settings\r\n",
      "############################################################\r\n",
      "\r\n",
      "#Genome size for Macs2, accept shortcuts like mm & hs\r\n",
      "GENOME_MACS2=hs\r\n",
      "#Whether dataset is joint profiling of RNA & ATAC of same cell. Separate measurements of two modalities in different cells: 0. Joint measurements: 1.\r\n",
      "JOINT=1\r\n",
      "\r\n",
      "#######################################################################################\r\n",
      "# Parameters of individual steps shared by cell-type specific and dynamic GRN inference\r\n",
      "#######################################################################################\r\n",
      "\r\n",
      "PARAMS-PREPROC-QC_READS:=50 10 0 200 100 0\r\n",
      "PARAMS-CHROMATIN-MACS2:=$(GENOME_MACS2)\r\n",
      "PARAMS-CHROMATIN-BINLINKING:=20\r\n",
      "KPARAMS-PREPROC-SELECTSC_RNA:=\r\n",
      "KPARAMS-PREPROC-QC_READS:=\r\n",
      "KPARAMS-PREPROC-SELECTSC_ATAC:=\r\n",
      "KPARAMS-CHROMATIN-MACS2:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-WELLINGTON:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-HOMER:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-BINDING:=\r\n",
      "KPARAMS-CHROMATIN-TSSDIST:=\r\n",
      "KPARAMS-CHROMATIN-LINKING:=\r\n",
      "KPARAMS-CHROMATIN-BINLINKING:=\r\n",
      "KPARAMS-NETWORK-RECONSTRUCT:=--device $(DEVICE) --nth $(NTH)\r\n",
      "KPARAMS-NETWORK-INDIRECT:=--nth $(NTH)\r\n",
      "KPARAMS-NETWORK-NORMALIZE:=--nth $(NTH)\r\n",
      "KPARAMS-NETWORK-TOFILE:=$(KPARAMS-NETWORK-TOFILE_EXTRA)\r\n",
      "\r\n",
      "#######################################################################################\r\n",
      "# Parameters of individual steps only in dynamic GRN inference\r\n",
      "#######################################################################################\r\n",
      "\r\n",
      "PARAMS-DYNAMIC-SUBSETS_RNA=1000 875 0.05\r\n",
      "PARAMS-DYNAMIC-SUBSETS_ATAC=$(firstword $(PARAMS-DYNAMIC-SUBSETS_RNA))\r\n"
     ]
    }
   ],
   "source": [
    "!cat ./makefiles/config.mk\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Validate input data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"cd .\"\n",
      "\tUser time (seconds): 0.00\n",
      "\tSystem time (seconds): 0.00\n",
      "\tPercent of CPU this job got: 50%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:00.00\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 3328\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 0\n",
      "\tMinor (reclaiming a frame) page faults: 189\n",
      "\tVoluntary context switches: 8\n",
      "\tInvoluntary context switches: 0\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 8\n",
      "\tFile system outputs: 0\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n",
      "WARNING:root:Chromosomes not found in reference genome: chrGL000194.1,chrMT\n",
      "WARNING:root:Cannot find dynamic.mk or traj_node.h5. Skipping dynamic network inference checks.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Joint profile: True\n",
      "Found 2714 cells with RNA profile\n",
      "Found 24036 genes with RNA profile\n",
      "Found 2714 cells with ATAC profile\n",
      "Found 769 motifs\n",
      "Found 678 TFs\n",
      "Found 461 TFs in current dataset\n",
      "Missing 217 TFs in current dataset: ANDR,AP2A,AP2B,AP2C,AP2D,ARI3A,ARI5B,ATF6A,BARH1,BARH2,BC11A,BHA15,BHE22,BHE23,BHE40,BHE41,BMAL1,BRAC,BSH,COE1,COT1,COT2,CR3L1,CR3L2,ERR1,ERR2,ERR3,EVI1,GCR,HEN1,HMBX1,HME1,HME2,HNF6,HTF4,HXA1,HXA10,HXA11,HXA13,HXA2,HXA5,HXA7,HXA9,HXB1,HXB13,HXB2,HXB3,HXB4,HXB6,HXB7,HXB8,HXC10,HXC11,HXC12,HXC13,HXC6,HXC8,HXC9,HXD10,HXD11,HXD12,HXD13,HXD3,HXD4,HXD8,HXD9,ITF2,KAISO,MCR,MGAP,MLXPL,MYBA,MYBB,NDF1,NDF2,NF2L1,NF2L2,NFAC1,NFAC2,NFAC3,NFAC4,NGN2,NKX21,NKX22,NKX23,NKX25,NKX28,NKX31,NKX32,NKX61,NKX62,ONEC2,ONEC3,OZF,P53,P5F1B,P63,P73,PEBB,PHX2A,PHX2B,PIT1,PKNX1,PLAL1,PO2F1,PO2F2,PO2F3,PO3F1,PO3F2,PO3F3,PO3F4,PO4F1,PO4F2,PO4F3,PO5F1,PO6F1,PO6F2,PRD14,PRGR,RHXF1,RORG,RX,SMCA1,SMCA5,SRBP1,SRBP2,STA5A,STA5B,STF1,SUH,TF2LX,TF65,TF7L1,TF7L2,TFE2,THA,THA11,THB,TWST1,TYY1,TYY2,UBIP1,UNC4,Z324A,Z354A,ZBT14,ZBT17,ZBT18,ZBT48,ZBT49,ZBT7A,ZBT7B,ZEP1,ZEP2,ZF64A,ZKSC1,ZKSC3,ZN121,ZN134,ZN136,ZN140,ZN143,ZN148,ZN214,ZN219,ZN232,ZN250,ZN257,ZN260,ZN263,ZN264,ZN274,ZN281,ZN282,ZN317,ZN320,ZN322,ZN329,ZN331,ZN333,ZN335,ZN341,ZN350,ZN382,ZN384,ZN394,ZN410,ZN418,ZN423,ZN436,ZN449,ZN467,ZN490,ZN502,ZN524,ZN528,ZN547,ZN549,ZN554,ZN563,ZN582,ZN586,ZN589,ZN652,ZN667,ZN680,ZN708,ZN713,ZN740,ZN768,ZN770,ZN784,ZN816,ZSC16,ZSC22,ZSC31,ZSCA4\n",
      "Found 23533 genes with TSS information\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"dictys_helper makefile_check.py\"\n",
      "\tUser time (seconds): 17.75\n",
      "\tSystem time (seconds): 5.53\n",
      "\tPercent of CPU this job got: 54%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:42.88\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 7923800\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 6\n",
      "\tMinor (reclaiming a frame) page faults: 4280792\n",
      "\tVoluntary context switches: 65301\n",
      "\tInvoluntary context switches: 791\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 6408392\n",
      "\tFile system outputs: 0\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/usr/bin/time -v cd . &&  /usr/bin/time -v dictys_helper makefile_check.py"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Network inference\n",
    "* **This step can take a day or longer**. You are strongly suggested to use a GPU. If it takes over a day, see [FAQ](https://github.com/pinellolab/dictys#faq) to improve speed.\n",
    "* Please adjust `-j n_jobs` for your machine. The total thread count is n_jobs*n_threads. By default n_threads=4.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.cpu_count())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "peak memory: 74.64 MiB, increment: 0.00 MiB\n"
     ]
    }
   ],
   "source": [
    "%memit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir -p tmp_static/Subset1/\n",
      "cp data/subsets/Subset1/names_rna.txt tmp_static/Subset1/names_rna.txt\n",
      "mkdir -p tmp_static/Subset2/\n",
      "mkdir -p tmp_static/Subset3/\n",
      "cp data/subsets/Subset2/names_rna.txt tmp_static/Subset2/names_rna.txt\n",
      "cp data/subsets/Subset3/names_rna.txt tmp_static/Subset3/names_rna.txt\n",
      "mkdir -p tmp_static/Subset4/\n",
      "mkdir -p tmp_static/Subset5/\n",
      "cp data/subsets/Subset4/names_rna.txt tmp_static/Subset4/names_rna.txt\n",
      "cp data/subsets/Subset5/names_rna.txt tmp_static/Subset5/names_rna.txt\n",
      "mkdir -p tmp_static/Subset6/\n",
      "cp data/subsets/Subset6/names_rna.txt tmp_static/Subset6/names_rna.txt\n",
      "mkdir -p tmp_static/Subset7/\n",
      "cp data/subsets/Subset7/names_rna.txt tmp_static/Subset7/names_rna.txt\n",
      "mkdir -p tmp_static/Subset8/\n",
      "cp data/subsets/Subset8/names_rna.txt tmp_static/Subset8/names_rna.txt\n",
      "mkdir -p tmp_static/Subset9/\n",
      "cp data/subsets/Subset9/names_rna.txt tmp_static/Subset9/names_rna.txt\n",
      "cp tmp_static/Subset1/names_rna.txt tmp_static/Subset1/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset1/names_rna.txt tmp_static/Subset1/expression0.tsv.gz\n",
      "cp tmp_static/Subset2/names_rna.txt tmp_static/Subset2/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset2/names_rna.txt tmp_static/Subset2/expression0.tsv.gz\n",
      "cp tmp_static/Subset3/names_rna.txt tmp_static/Subset3/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset3/names_rna.txt tmp_static/Subset3/expression0.tsv.gz\n",
      "cp tmp_static/Subset4/names_rna.txt tmp_static/Subset4/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset4/names_rna.txt tmp_static/Subset4/expression0.tsv.gz\n",
      "cp tmp_static/Subset5/names_rna.txt tmp_static/Subset5/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset5/names_rna.txt tmp_static/Subset5/expression0.tsv.gz\n",
      "cp tmp_static/Subset6/names_rna.txt tmp_static/Subset6/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset6/names_rna.txt tmp_static/Subset6/expression0.tsv.gz\n",
      "cp tmp_static/Subset7/names_rna.txt tmp_static/Subset7/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset7/names_rna.txt tmp_static/Subset7/expression0.tsv.gz\n",
      "cp tmp_static/Subset8/names_rna.txt tmp_static/Subset8/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset8/names_rna.txt tmp_static/Subset8/expression0.tsv.gz\n",
      "cp tmp_static/Subset9/names_rna.txt tmp_static/Subset9/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset9/names_rna.txt tmp_static/Subset9/expression0.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset8/expression0.tsv.gz tmp_static/Subset8/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset6/expression0.tsv.gz tmp_static/Subset6/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset7/expression0.tsv.gz tmp_static/Subset7/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset9/expression0.tsv.gz tmp_static/Subset9/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset5/expression0.tsv.gz tmp_static/Subset5/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset4/expression0.tsv.gz tmp_static/Subset4/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/names_atac0.txt tmp_static/Subset8/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset1/expression0.tsv.gz tmp_static/Subset1/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset2/expression0.tsv.gz tmp_static/Subset2/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset3/expression0.tsv.gz tmp_static/Subset3/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset8/names_atac.txt data/bams tmp_static/Subset8/reads.bam tmp_static/Subset8/reads.bai tmp_static/Subset8/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/names_atac0.txt tmp_static/Subset6/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/names_atac0.txt tmp_static/Subset7/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset6/names_atac.txt data/bams tmp_static/Subset6/reads.bam tmp_static/Subset6/reads.bai tmp_static/Subset6/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/names_atac0.txt tmp_static/Subset5/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/names_atac0.txt tmp_static/Subset9/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset7/names_atac.txt data/bams tmp_static/Subset7/reads.bam tmp_static/Subset7/reads.bai tmp_static/Subset7/peaks.bed hs\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/names_atac0.txt tmp_static/Subset4/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/names_atac0.txt tmp_static/Subset1/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/names_atac0.txt tmp_static/Subset2/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset9/names_atac.txt data/bams tmp_static/Subset9/reads.bam tmp_static/Subset9/reads.bai tmp_static/Subset9/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset5/names_atac.txt data/bams tmp_static/Subset5/reads.bam tmp_static/Subset5/reads.bai tmp_static/Subset5/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset4/names_atac.txt data/bams tmp_static/Subset4/reads.bam tmp_static/Subset4/reads.bai tmp_static/Subset4/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset1/names_atac.txt data/bams tmp_static/Subset1/reads.bam tmp_static/Subset1/reads.bai tmp_static/Subset1/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset2/names_atac.txt data/bams tmp_static/Subset2/reads.bam tmp_static/Subset2/reads.bai tmp_static/Subset2/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/names_atac0.txt tmp_static/Subset3/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset3/names_atac.txt data/bams tmp_static/Subset3/reads.bam tmp_static/Subset3/reads.bai tmp_static/Subset3/peaks.bed hs\n",
      "[bam_sort_core] merging from 0 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:08:18: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset8/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset8/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:18: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:18: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:08:18: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:18: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:18: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:21:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:24:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: 2957766 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #1  total tags in treatment: 2957766 \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:27: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:08:27: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:08:31: access pq hash for 6868406 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:31: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:47: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:48: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:48: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:08:48: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset8/reads.bam tmp_static/Subset8/reads.bai tmp_static/Subset8/peaks.bed tmp_static/Subset8/footprints.bed\n",
      "[bam_sort_core] merging from 0 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:09:08: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset9/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset9/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:08: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:08: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:08: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:08: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:08: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:11:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:14:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:17:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: 3860901 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #1  total tags in treatment: 3860901 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:20: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:20: Start to calculate pvalue stat... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DEBUG @ Thu, 01 Aug 2024 17:09:24: access pq hash for 8985502 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:24: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:45: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:45: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:45: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:45: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset9/reads.bam tmp_static/Subset9/reads.bai tmp_static/Subset9/peaks.bed tmp_static/Subset9/footprints.bed\n",
      "[bam_sort_core] merging from 1 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:09:37: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset6/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset6/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:37: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:37: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:37: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:37: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:37: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:40:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:43:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:46:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:49:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: 4693961 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #1  total tags in treatment: 4693961 \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:52: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:52: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:58: access pq hash for 10106809 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:58: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:16: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:17: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:17: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:17: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset6/reads.bam tmp_static/Subset6/reads.bai tmp_static/Subset6/peaks.bed tmp_static/Subset6/footprints.bed\n",
      "[bam_sort_core] merging from 1 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:09:55: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset7/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset7/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:55: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:55: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:09:55: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:55: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:55: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:09:58:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:01:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:04:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:07:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:11:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:14:  6000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: 6311723 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #1  total tags in treatment: 6311723 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:10:15: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:10:23: access pq hash for 13789663 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:23: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:43: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:43: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:43: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:43: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset7/reads.bam tmp_static/Subset7/reads.bai tmp_static/Subset7/peaks.bed tmp_static/Subset7/footprints.bed\n",
      "[bam_sort_core] merging from 1 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:10:06: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset5/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset5/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:06: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:06: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:10:06: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:06: Detected format is: BAM \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Thu, 01 Aug 2024 17:10:06: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:09:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:12:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:15:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:18:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:21:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: 5346533 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #1  total tags in treatment: 5346533 \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:22: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:10:22: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:10:29: access pq hash for 11250699 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:29: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:50: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:50: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:50: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:10:50: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset5/reads.bam tmp_static/Subset5/reads.bai tmp_static/Subset5/peaks.bed tmp_static/Subset5/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset2/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset2/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:08: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:11:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:14:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:18:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:24:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:28:  6000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:31:  7000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:34:  8000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37:  9000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: 9104121 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #1  total tags in treatment: 9104121 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:37: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:48: access pq hash for 17953568 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:48: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:12: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:13: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:13: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:13: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset2/reads.bam tmp_static/Subset2/reads.bai tmp_static/Subset2/peaks.bed tmp_static/Subset2/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset1/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset1/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:17: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:20:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:23:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:26:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:30:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:33:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:36:  6000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:39:  7000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42:  8000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:45:  9000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:48:  10000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: 10971622 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #1  total tags in treatment: 10971622 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:51: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:51: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:13:05: access pq hash for 21638339 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:05: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:27: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:27: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:27: #4 Write summits bed file... 04_summits.bed \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Thu, 01 Aug 2024 17:13:27: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset1/reads.bam tmp_static/Subset1/reads.bai tmp_static/Subset1/peaks.bed tmp_static/Subset1/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset4/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset4/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:08: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:08: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:11:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:14:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:17:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:20:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:23:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:27:  6000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:30:  7000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:34:  8000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37:  9000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:40:  10000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: 10670307 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #1 tag size is determined as 49 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #1 tag size = 49.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #1  total tags in treatment: 10670307 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:42: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:42: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:56: access pq hash for 23710636 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:56: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:30: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:31: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:31: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:31: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset4/reads.bam tmp_static/Subset4/reads.bai tmp_static/Subset4/peaks.bed tmp_static/Subset4/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset3/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/tmp_static/Subset3/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21: #1 read tag files... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21: #1 read treatment tags... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:21: Testing format BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21: Detected format is: BAM \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:21: * Input file is gzipped. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:24:  1000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:27:  2000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:30:  3000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:34:  4000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:37:  5000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:40:  6000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:43:  7000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:46:  8000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:50:  9000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:53:  10000000 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: 10595817 reads have been read. \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #1 tag size is determined as 48 bps \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #1 tag size = 48.0 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #1  total tags in treatment: 10595817 \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #1 finished! \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #2 Build Peak Model... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #2 Skipped... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #2 Use 150 as fragment length \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #3 Call peaks... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Thu, 01 Aug 2024 17:12:55: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:12:55: Start to calculate pvalue stat... \n",
      "DEBUG @ Thu, 01 Aug 2024 17:13:09: access pq hash for 22343420 times \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:09: #3 Call peaks for each chromosome... \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:43: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:44: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:44: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Thu, 01 Aug 2024 17:13:44: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset3/reads.bam tmp_static/Subset3/reads.bai tmp_static/Subset3/peaks.bed tmp_static/Subset3/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset8/footprints.bed data/motifs.motif data/genome tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/motifs.bed tmp_static/Subset8/wellington.tsv.gz tmp_static/Subset8/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5768\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5768\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 563 sequences from chr1\n",
      "\tExtracting 211 sequences from chr10\n",
      "\tExtracting 351 sequences from chr11\n",
      "\tExtracting 320 sequences from chr12\n",
      "\tExtracting 109 sequences from chr13\n",
      "\tExtracting 218 sequences from chr14\n",
      "\tExtracting 164 sequences from chr15\n",
      "\tExtracting 283 sequences from chr16\n",
      "\tExtracting 397 sequences from chr17\n",
      "\tExtracting 66 sequences from chr18\n",
      "\tExtracting 568 sequences from chr19\n",
      "\tExtracting 369 sequences from chr2\n",
      "\tExtracting 122 sequences from chr20\n",
      "\tExtracting 85 sequences from chr21\n",
      "\tExtracting 131 sequences from chr22\n",
      "\tExtracting 261 sequences from chr3\n",
      "\tExtracting 198 sequences from chr4\n",
      "\tExtracting 207 sequences from chr5\n",
      "\tExtracting 316 sequences from chr6\n",
      "\tExtracting 285 sequences from chr7\n",
      "\tExtracting 162 sequences from chr8\n",
      "\tExtracting 246 sequences from chr9\n",
      "\tExtracting 109 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5741 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5767\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5767\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 613 sequences from chr1\n",
      "\tExtracting 221 sequences from chr10\n",
      "\tExtracting 289 sequences from chr11\n",
      "\tExtracting 303 sequences from chr12\n",
      "\tExtracting 102 sequences from chr13\n",
      "\tExtracting 204 sequences from chr14\n",
      "\tExtracting 149 sequences from chr15\n",
      "\tExtracting 286 sequences from chr16\n",
      "\tExtracting 368 sequences from chr17\n",
      "\tExtracting 81 sequences from chr18\n",
      "\tExtracting 546 sequences from chr19\n",
      "\tExtracting 373 sequences from chr2\n",
      "\tExtracting 133 sequences from chr20\n",
      "\tExtracting 101 sequences from chr21\n",
      "\tExtracting 142 sequences from chr22\n",
      "\tExtracting 283 sequences from chr3\n",
      "\tExtracting 195 sequences from chr4\n",
      "\tExtracting 228 sequences from chr5\n",
      "\tExtracting 296 sequences from chr6\n",
      "\tExtracting 278 sequences from chr7\n",
      "\tExtracting 200 sequences from chr8\n",
      "\tExtracting 235 sequences from chr9\n",
      "\tExtracting 124 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5750 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5768\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5768\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 561 sequences from chr1\n",
      "\tExtracting 204 sequences from chr10\n",
      "\tExtracting 302 sequences from chr11\n",
      "\tExtracting 304 sequences from chr12\n",
      "\tExtracting 96 sequences from chr13\n",
      "\tExtracting 218 sequences from chr14\n",
      "\tExtracting 153 sequences from chr15\n",
      "\tExtracting 271 sequences from chr16\n",
      "\tExtracting 416 sequences from chr17\n",
      "\tExtracting 77 sequences from chr18\n",
      "\tExtracting 609 sequences from chr19\n",
      "\tExtracting 346 sequences from chr2\n",
      "\tExtracting 151 sequences from chr20\n",
      "\tExtracting 126 sequences from chr21\n",
      "\tExtracting 139 sequences from chr22\n",
      "\tExtracting 275 sequences from chr3\n",
      "\tExtracting 187 sequences from chr4\n",
      "\tExtracting 215 sequences from chr5\n",
      "\tExtracting 281 sequences from chr6\n",
      "\tExtracting 274 sequences from chr7\n",
      "\tExtracting 192 sequences from chr8\n",
      "\tExtracting 248 sequences from chr9\n",
      "\tExtracting 94 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5739 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 5768\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 5768\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 626 sequences from chr1\n",
      "\tExtracting 179 sequences from chr10\n",
      "\tExtracting 304 sequences from chr11\n",
      "\tExtracting 291 sequences from chr12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 78 sequences from chr13\n",
      "\tExtracting 201 sequences from chr14\n",
      "\tExtracting 130 sequences from chr15\n",
      "\tExtracting 277 sequences from chr16\n",
      "\tExtracting 390 sequences from chr17\n",
      "\tExtracting 61 sequences from chr18\n",
      "\tExtracting 674 sequences from chr19\n",
      "\tExtracting 354 sequences from chr2\n",
      "\tExtracting 136 sequences from chr20\n",
      "\tExtracting 156 sequences from chr21\n",
      "\tExtracting 131 sequences from chr22\n",
      "\tExtracting 303 sequences from chr3\n",
      "\tExtracting 177 sequences from chr4\n",
      "\tExtracting 204 sequences from chr5\n",
      "\tExtracting 283 sequences from chr6\n",
      "\tExtracting 298 sequences from chr7\n",
      "\tExtracting 160 sequences from chr8\n",
      "\tExtracting 245 sequences from chr9\n",
      "\tExtracting 59 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t5718 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset8/wellington.tsv.gz tmp_static/Subset8/homer.tsv.gz tmp_static/Subset8/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/wellington.tsv.gz data/gene.bed tmp_static/Subset8/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset8/binding.tsv.gz tmp_static/Subset8/tssdist.tsv.gz tmp_static/Subset8/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset8/linking.tsv.gz tmp_static/Subset8/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/binlinking.tsv.gz tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_loss.tsv.gz tmp_static/Subset8/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset9/footprints.bed data/motifs.motif data/genome tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/motifs.bed tmp_static/Subset9/wellington.tsv.gz tmp_static/Subset9/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 7791\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 7791\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 793 sequences from chr1\n",
      "\tExtracting 338 sequences from chr10\n",
      "\tExtracting 378 sequences from chr11\n",
      "\tExtracting 410 sequences from chr12\n",
      "\tExtracting 145 sequences from chr13\n",
      "\tExtracting 248 sequences from chr14\n",
      "\tExtracting 242 sequences from chr15\n",
      "\tExtracting 387 sequences from chr16\n",
      "\tExtracting 538 sequences from chr17\n",
      "\tExtracting 114 sequences from chr18\n",
      "\tExtracting 602 sequences from chr19\n",
      "\tExtracting 515 sequences from chr2\n",
      "\tExtracting 224 sequences from chr20\n",
      "\tExtracting 80 sequences from chr21\n",
      "\tExtracting 193 sequences from chr22\n",
      "\tExtracting 445 sequences from chr3\n",
      "\tExtracting 252 sequences from chr4\n",
      "\tExtracting 360 sequences from chr5\n",
      "\tExtracting 409 sequences from chr6\n",
      "\tExtracting 371 sequences from chr7\n",
      "\tExtracting 255 sequences from chr8\n",
      "\tExtracting 327 sequences from chr9\n",
      "\tExtracting 159 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t7785 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 7792\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 7792\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 845 sequences from chr1\n",
      "\tExtracting 301 sequences from chr10\n",
      "\tExtracting 401 sequences from chr11\n",
      "\tExtracting 360 sequences from chr12\n",
      "\tExtracting 108 sequences from chr13\n",
      "\tExtracting 260 sequences from chr14\n",
      "\tExtracting 201 sequences from chr15\n",
      "\tExtracting 369 sequences from chr16\n",
      "\tExtracting 584 sequences from chr17\n",
      "\tExtracting 103 sequences from chr18\n",
      "\tExtracting 808 sequences from chr19\n",
      "\tExtracting 490 sequences from chr2\n",
      "\tExtracting 221 sequences from chr20\n",
      "\tExtracting 100 sequences from chr21\n",
      "\tExtracting 199 sequences from chr22\n",
      "\tExtracting 415 sequences from chr3\n",
      "\tExtracting 268 sequences from chr4\n",
      "\tExtracting 284 sequences from chr5\n",
      "\tExtracting 377 sequences from chr6\n",
      "\tExtracting 391 sequences from chr7\n",
      "\tExtracting 250 sequences from chr8\n",
      "\tExtracting 343 sequences from chr9\n",
      "\tExtracting 103 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t7782 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 7792\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 7792\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 852 sequences from chr1\n",
      "\tExtracting 313 sequences from chr10\n",
      "\tExtracting 388 sequences from chr11\n",
      "\tExtracting 403 sequences from chr12\n",
      "\tExtracting 117 sequences from chr13\n",
      "\tExtracting 277 sequences from chr14\n",
      "\tExtracting 203 sequences from chr15\n",
      "\tExtracting 363 sequences from chr16\n",
      "\tExtracting 542 sequences from chr17\n",
      "\tExtracting 108 sequences from chr18\n",
      "\tExtracting 673 sequences from chr19\n",
      "\tExtracting 519 sequences from chr2\n",
      "\tExtracting 212 sequences from chr20\n",
      "\tExtracting 83 sequences from chr21\n",
      "\tExtracting 188 sequences from chr22\n",
      "\tExtracting 423 sequences from chr3\n",
      "\tExtracting 259 sequences from chr4\n",
      "\tExtracting 339 sequences from chr5\n",
      "\tExtracting 416 sequences from chr6\n",
      "\tExtracting 360 sequences from chr7\n",
      "\tExtracting 285 sequences from chr8\n",
      "\tExtracting 337 sequences from chr9\n",
      "\tExtracting 123 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t7783 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 7792\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 7792\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 784 sequences from chr1\n",
      "\tExtracting 303 sequences from chr10\n",
      "\tExtracting 420 sequences from chr11\n",
      "\tExtracting 421 sequences from chr12\n",
      "\tExtracting 136 sequences from chr13\n",
      "\tExtracting 264 sequences from chr14\n",
      "\tExtracting 232 sequences from chr15\n",
      "\tExtracting 353 sequences from chr16\n",
      "\tExtracting 511 sequences from chr17\n",
      "\tExtracting 109 sequences from chr18\n",
      "\tExtracting 711 sequences from chr19\n",
      "\tExtracting 521 sequences from chr2\n",
      "\tExtracting 212 sequences from chr20\n",
      "\tExtracting 97 sequences from chr21\n",
      "\tExtracting 185 sequences from chr22\n",
      "\tExtracting 431 sequences from chr3\n",
      "\tExtracting 276 sequences from chr4\n",
      "\tExtracting 324 sequences from chr5\n",
      "\tExtracting 438 sequences from chr6\n",
      "\tExtracting 368 sequences from chr7\n",
      "\tExtracting 244 sequences from chr8\n",
      "\tExtracting 308 sequences from chr9\n",
      "\tExtracting 138 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t7786 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset9/wellington.tsv.gz tmp_static/Subset9/homer.tsv.gz tmp_static/Subset9/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/wellington.tsv.gz data/gene.bed tmp_static/Subset9/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset9/binding.tsv.gz tmp_static/Subset9/tssdist.tsv.gz tmp_static/Subset9/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset9/linking.tsv.gz tmp_static/Subset9/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/binlinking.tsv.gz tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_loss.tsv.gz tmp_static/Subset9/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset6/footprints.bed data/motifs.motif data/genome tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/motifs.bed tmp_static/Subset6/wellington.tsv.gz tmp_static/Subset6/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 12260\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 12260\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1297 sequences from chr1\n",
      "\tExtracting 479 sequences from chr10\n",
      "\tExtracting 673 sequences from chr11\n",
      "\tExtracting 610 sequences from chr12\n",
      "\tExtracting 213 sequences from chr13\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 443 sequences from chr14\n",
      "\tExtracting 336 sequences from chr15\n",
      "\tExtracting 549 sequences from chr16\n",
      "\tExtracting 802 sequences from chr17\n",
      "\tExtracting 166 sequences from chr18\n",
      "\tExtracting 1024 sequences from chr19\n",
      "\tExtracting 790 sequences from chr2\n",
      "\tExtracting 317 sequences from chr20\n",
      "\tExtracting 191 sequences from chr21\n",
      "\tExtracting 307 sequences from chr22\n",
      "\tExtracting 644 sequences from chr3\n",
      "\tExtracting 395 sequences from chr4\n",
      "\tExtracting 527 sequences from chr5\n",
      "\tExtracting 673 sequences from chr6\n",
      "\tExtracting 600 sequences from chr7\n",
      "\tExtracting 425 sequences from chr8\n",
      "\tExtracting 489 sequences from chr9\n",
      "\tExtracting 277 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t12227 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 12260\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 12260\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1254 sequences from chr1\n",
      "\tExtracting 471 sequences from chr10\n",
      "\tExtracting 692 sequences from chr11\n",
      "\tExtracting 634 sequences from chr12\n",
      "\tExtracting 237 sequences from chr13\n",
      "\tExtracting 392 sequences from chr14\n",
      "\tExtracting 348 sequences from chr15\n",
      "\tExtracting 555 sequences from chr16\n",
      "\tExtracting 854 sequences from chr17\n",
      "\tExtracting 150 sequences from chr18\n",
      "\tExtracting 1045 sequences from chr19\n",
      "\tExtracting 826 sequences from chr2\n",
      "\tExtracting 290 sequences from chr20\n",
      "\tExtracting 233 sequences from chr21\n",
      "\tExtracting 329 sequences from chr22\n",
      "\tExtracting 624 sequences from chr3\n",
      "\tExtracting 405 sequences from chr4\n",
      "\tExtracting 480 sequences from chr5\n",
      "\tExtracting 652 sequences from chr6\n",
      "\tExtracting 596 sequences from chr7\n",
      "\tExtracting 431 sequences from chr8\n",
      "\tExtracting 484 sequences from chr9\n",
      "\tExtracting 235 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t12218 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 12256\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 12256\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1265 sequences from chr1\n",
      "\tExtracting 503 sequences from chr10\n",
      "\tExtracting 639 sequences from chr11\n",
      "\tExtracting 621 sequences from chr12\n",
      "\tExtracting 230 sequences from chr13\n",
      "\tExtracting 417 sequences from chr14\n",
      "\tExtracting 364 sequences from chr15\n",
      "\tExtracting 572 sequences from chr16\n",
      "\tExtracting 845 sequences from chr17\n",
      "\tExtracting 167 sequences from chr18\n",
      "\tExtracting 977 sequences from chr19\n",
      "\tExtracting 781 sequences from chr2\n",
      "\tExtracting 303 sequences from chr20\n",
      "\tExtracting 208 sequences from chr21\n",
      "\tExtracting 299 sequences from chr22\n",
      "\tExtracting 640 sequences from chr3\n",
      "\tExtracting 429 sequences from chr4\n",
      "\tExtracting 503 sequences from chr5\n",
      "\tExtracting 690 sequences from chr6\n",
      "\tExtracting 540 sequences from chr7\n",
      "\tExtracting 446 sequences from chr8\n",
      "\tExtracting 498 sequences from chr9\n",
      "\tExtracting 290 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t12228 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 12260\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 12260\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1301 sequences from chr1\n",
      "\tExtracting 445 sequences from chr10\n",
      "\tExtracting 638 sequences from chr11\n",
      "\tExtracting 652 sequences from chr12\n",
      "\tExtracting 208 sequences from chr13\n",
      "\tExtracting 434 sequences from chr14\n",
      "\tExtracting 327 sequences from chr15\n",
      "\tExtracting 589 sequences from chr16\n",
      "\tExtracting 822 sequences from chr17\n",
      "\tExtracting 150 sequences from chr18\n",
      "\tExtracting 1322 sequences from chr19\n",
      "\tExtracting 778 sequences from chr2\n",
      "\tExtracting 304 sequences from chr20\n",
      "\tExtracting 289 sequences from chr21\n",
      "\tExtracting 273 sequences from chr22\n",
      "\tExtracting 611 sequences from chr3\n",
      "\tExtracting 377 sequences from chr4\n",
      "\tExtracting 477 sequences from chr5\n",
      "\tExtracting 591 sequences from chr6\n",
      "\tExtracting 582 sequences from chr7\n",
      "\tExtracting 360 sequences from chr8\n",
      "\tExtracting 482 sequences from chr9\n",
      "\tExtracting 166 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t12179 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset6/wellington.tsv.gz tmp_static/Subset6/homer.tsv.gz tmp_static/Subset6/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/wellington.tsv.gz data/gene.bed tmp_static/Subset6/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset6/binding.tsv.gz tmp_static/Subset6/tssdist.tsv.gz tmp_static/Subset6/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset6/linking.tsv.gz tmp_static/Subset6/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/binlinking.tsv.gz tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_loss.tsv.gz tmp_static/Subset6/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset5/footprints.bed data/motifs.motif data/genome tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/motifs.bed tmp_static/Subset5/wellington.tsv.gz tmp_static/Subset5/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset7/footprints.bed data/motifs.motif data/genome tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/motifs.bed tmp_static/Subset7/wellington.tsv.gz tmp_static/Subset7/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14384\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14384\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1443 sequences from chr1\n",
      "\tExtracting 516 sequences from chr10\n",
      "\tExtracting 742 sequences from chr11\n",
      "\tExtracting 770 sequences from chr12\n",
      "\tExtracting 238 sequences from chr13\n",
      "\tExtracting 503 sequences from chr14\n",
      "\tExtracting 447 sequences from chr15\n",
      "\tExtracting 722 sequences from chr16\n",
      "\tExtracting 979 sequences from chr17\n",
      "\tExtracting 211 sequences from chr18\n",
      "\tExtracting 1280 sequences from chr19\n",
      "\tExtracting 926 sequences from chr2\n",
      "\tExtracting 407 sequences from chr20\n",
      "\tExtracting 244 sequences from chr21\n",
      "\tExtracting 361 sequences from chr22\n",
      "\tExtracting 738 sequences from chr3\n",
      "\tExtracting 455 sequences from chr4\n",
      "\tExtracting 611 sequences from chr5\n",
      "\tExtracting 755 sequences from chr6\n",
      "\tExtracting 661 sequences from chr7\n",
      "\tExtracting 466 sequences from chr8\n",
      "\tExtracting 565 sequences from chr9\n",
      "\tExtracting 299 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14340 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14384\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14384\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1456 sequences from chr1\n",
      "\tExtracting 587 sequences from chr10\n",
      "\tExtracting 776 sequences from chr11\n",
      "\tExtracting 758 sequences from chr12\n",
      "\tExtracting 250 sequences from chr13\n",
      "\tExtracting 501 sequences from chr14\n",
      "\tExtracting 421 sequences from chr15\n",
      "\tExtracting 697 sequences from chr16\n",
      "\tExtracting 913 sequences from chr17\n",
      "\tExtracting 221 sequences from chr18\n",
      "\tExtracting 1200 sequences from chr19\n",
      "\tExtracting 1033 sequences from chr2\n",
      "\tExtracting 348 sequences from chr20\n",
      "\tExtracting 205 sequences from chr21\n",
      "\tExtracting 379 sequences from chr22\n",
      "\tExtracting 747 sequences from chr3\n",
      "\tExtracting 466 sequences from chr4\n",
      "\tExtracting 588 sequences from chr5\n",
      "\tExtracting 752 sequences from chr6\n",
      "\tExtracting 671 sequences from chr7\n",
      "\tExtracting 466 sequences from chr8\n",
      "\tExtracting 603 sequences from chr9\n",
      "\tExtracting 325 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14363 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\tBED/Header formatted lines: 14384\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14384\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1466 sequences from chr1\n",
      "\tExtracting 503 sequences from chr10\n",
      "\tExtracting 764 sequences from chr11\n",
      "\tExtracting 730 sequences from chr12\n",
      "\tExtracting 217 sequences from chr13\n",
      "\tExtracting 519 sequences from chr14\n",
      "\tExtracting 379 sequences from chr15\n",
      "\tExtracting 714 sequences from chr16\n",
      "\tExtracting 998 sequences from chr17\n",
      "\tExtracting 190 sequences from chr18\n",
      "\tExtracting 1478 sequences from chr19\n",
      "\tExtracting 874 sequences from chr2\n",
      "\tExtracting 334 sequences from chr20\n",
      "\tExtracting 380 sequences from chr21\n",
      "\tExtracting 341 sequences from chr22\n",
      "\tExtracting 711 sequences from chr3\n",
      "\tExtracting 459 sequences from chr4\n",
      "\tExtracting 574 sequences from chr5\n",
      "\tExtracting 695 sequences from chr6\n",
      "\tExtracting 697 sequences from chr7\n",
      "\tExtracting 446 sequences from chr8\n",
      "\tExtracting 582 sequences from chr9\n",
      "\tExtracting 212 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14264 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14383\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14383\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1467 sequences from chr1\n",
      "\tExtracting 552 sequences from chr10\n",
      "\tExtracting 696 sequences from chr11\n",
      "\tExtracting 715 sequences from chr12\n",
      "\tExtracting 249 sequences from chr13\n",
      "\tExtracting 489 sequences from chr14\n",
      "\tExtracting 455 sequences from chr15\n",
      "\tExtracting 681 sequences from chr16\n",
      "\tExtracting 1024 sequences from chr17\n",
      "\tExtracting 212 sequences from chr18\n",
      "\tExtracting 1087 sequences from chr19\n",
      "\tExtracting 994 sequences from chr2\n",
      "\tExtracting 388 sequences from chr20\n",
      "\tExtracting 215 sequences from chr21\n",
      "\tExtracting 356 sequences from chr22\n",
      "\tExtracting 823 sequences from chr3\n",
      "\tExtracting 458 sequences from chr4\n",
      "\tExtracting 619 sequences from chr5\n",
      "\tExtracting 789 sequences from chr6\n",
      "\tExtracting 695 sequences from chr7\n",
      "\tExtracting 473 sequences from chr8\n",
      "\tExtracting 582 sequences from chr9\n",
      "\tExtracting 338 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14357 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset5/wellington.tsv.gz tmp_static/Subset5/homer.tsv.gz tmp_static/Subset5/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/wellington.tsv.gz data/gene.bed tmp_static/Subset5/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset5/binding.tsv.gz tmp_static/Subset5/tssdist.tsv.gz tmp_static/Subset5/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset5/linking.tsv.gz tmp_static/Subset5/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/binlinking.tsv.gz tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_loss.tsv.gz tmp_static/Subset5/net_stats.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14803\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14803\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1513 sequences from chr1\n",
      "\tExtracting 570 sequences from chr10\n",
      "\tExtracting 770 sequences from chr11\n",
      "\tExtracting 760 sequences from chr12\n",
      "\tExtracting 250 sequences from chr13\n",
      "\tExtracting 507 sequences from chr14\n",
      "\tExtracting 445 sequences from chr15\n",
      "\tExtracting 721 sequences from chr16\n",
      "\tExtracting 987 sequences from chr17\n",
      "\tExtracting 200 sequences from chr18\n",
      "\tExtracting 1237 sequences from chr19\n",
      "\tExtracting 980 sequences from chr2\n",
      "\tExtracting 375 sequences from chr20\n",
      "\tExtracting 224 sequences from chr21\n",
      "\tExtracting 386 sequences from chr22\n",
      "\tExtracting 774 sequences from chr3\n",
      "\tExtracting 438 sequences from chr4\n",
      "\tExtracting 616 sequences from chr5\n",
      "\tExtracting 902 sequences from chr6\n",
      "\tExtracting 644 sequences from chr7\n",
      "\tExtracting 491 sequences from chr8\n",
      "\tExtracting 667 sequences from chr9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 296 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14753 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14803\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14803\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1557 sequences from chr1\n",
      "\tExtracting 557 sequences from chr10\n",
      "\tExtracting 803 sequences from chr11\n",
      "\tExtracting 742 sequences from chr12\n",
      "\tExtracting 279 sequences from chr13\n",
      "\tExtracting 514 sequences from chr14\n",
      "\tExtracting 464 sequences from chr15\n",
      "\tExtracting 699 sequences from chr16\n",
      "\tExtracting 976 sequences from chr17\n",
      "\tExtracting 202 sequences from chr18\n",
      "\tExtracting 1197 sequences from chr19\n",
      "\tExtracting 970 sequences from chr2\n",
      "\tExtracting 388 sequences from chr20\n",
      "\tExtracting 233 sequences from chr21\n",
      "\tExtracting 350 sequences from chr22\n",
      "\tExtracting 735 sequences from chr3\n",
      "\tExtracting 477 sequences from chr4\n",
      "\tExtracting 604 sequences from chr5\n",
      "\tExtracting 834 sequences from chr6\n",
      "\tExtracting 726 sequences from chr7\n",
      "\tExtracting 511 sequences from chr8\n",
      "\tExtracting 604 sequences from chr9\n",
      "\tExtracting 344 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14767 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14803\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14803\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1517 sequences from chr1\n",
      "\tExtracting 533 sequences from chr10\n",
      "\tExtracting 781 sequences from chr11\n",
      "\tExtracting 748 sequences from chr12\n",
      "\tExtracting 204 sequences from chr13\n",
      "\tExtracting 488 sequences from chr14\n",
      "\tExtracting 388 sequences from chr15\n",
      "\tExtracting 756 sequences from chr16\n",
      "\tExtracting 1039 sequences from chr17\n",
      "\tExtracting 184 sequences from chr18\n",
      "\tExtracting 1533 sequences from chr19\n",
      "\tExtracting 920 sequences from chr2\n",
      "\tExtracting 388 sequences from chr20\n",
      "\tExtracting 331 sequences from chr21\n",
      "\tExtracting 387 sequences from chr22\n",
      "\tExtracting 767 sequences from chr3\n",
      "\tExtracting 445 sequences from chr4\n",
      "\tExtracting 587 sequences from chr5\n",
      "\tExtracting 742 sequences from chr6\n",
      "\tExtracting 753 sequences from chr7\n",
      "\tExtracting 447 sequences from chr8\n",
      "\tExtracting 558 sequences from chr9\n",
      "\tExtracting 201 sequences from chrX\n",
      "\tExtracting 4 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14701 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 14800\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 14800\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 1478 sequences from chr1\n",
      "\tExtracting 587 sequences from chr10\n",
      "\tExtracting 778 sequences from chr11\n",
      "\tExtracting 712 sequences from chr12\n",
      "\tExtracting 243 sequences from chr13\n",
      "\tExtracting 526 sequences from chr14\n",
      "\tExtracting 444 sequences from chr15\n",
      "\tExtracting 694 sequences from chr16\n",
      "\tExtracting 1020 sequences from chr17\n",
      "\tExtracting 195 sequences from chr18\n",
      "\tExtracting 1186 sequences from chr19\n",
      "\tExtracting 1010 sequences from chr2\n",
      "\tExtracting 376 sequences from chr20\n",
      "\tExtracting 218 sequences from chr21\n",
      "\tExtracting 382 sequences from chr22\n",
      "\tExtracting 823 sequences from chr3\n",
      "\tExtracting 505 sequences from chr4\n",
      "\tExtracting 639 sequences from chr5\n",
      "\tExtracting 792 sequences from chr6\n",
      "\tExtracting 698 sequences from chr7\n",
      "\tExtracting 534 sequences from chr8\n",
      "\tExtracting 592 sequences from chr9\n",
      "\tExtracting 345 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t14777 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset7/wellington.tsv.gz tmp_static/Subset7/homer.tsv.gz tmp_static/Subset7/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/wellington.tsv.gz data/gene.bed tmp_static/Subset7/tssdist.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset7/binding.tsv.gz tmp_static/Subset7/tssdist.tsv.gz tmp_static/Subset7/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset7/linking.tsv.gz tmp_static/Subset7/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/binlinking.tsv.gz tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_loss.tsv.gz tmp_static/Subset7/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset2/footprints.bed data/motifs.motif data/genome tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/motifs.bed tmp_static/Subset2/wellington.tsv.gz tmp_static/Subset2/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset1/footprints.bed data/motifs.motif data/genome tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/motifs.bed tmp_static/Subset1/wellington.tsv.gz tmp_static/Subset1/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23903\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23903\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2572 sequences from chr1\n",
      "\tExtracting 915 sequences from chr10\n",
      "\tExtracting 1270 sequences from chr11\n",
      "\tExtracting 1291 sequences from chr12\n",
      "\tExtracting 436 sequences from chr13\n",
      "\tExtracting 835 sequences from chr14\n",
      "\tExtracting 728 sequences from chr15\n",
      "\tExtracting 1137 sequences from chr16\n",
      "\tExtracting 1562 sequences from chr17\n",
      "\tExtracting 299 sequences from chr18\n",
      "\tExtracting 1896 sequences from chr19\n",
      "\tExtracting 1611 sequences from chr2\n",
      "\tExtracting 592 sequences from chr20\n",
      "\tExtracting 344 sequences from chr21\n",
      "\tExtracting 644 sequences from chr22\n",
      "\tExtracting 1254 sequences from chr3\n",
      "\tExtracting 723 sequences from chr4\n",
      "\tExtracting 998 sequences from chr5\n",
      "\tExtracting 1320 sequences from chr6\n",
      "\tExtracting 1076 sequences from chr7\n",
      "\tExtracting 806 sequences from chr8\n",
      "\tExtracting 964 sequences from chr9\n",
      "\tExtracting 572 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23845 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23901\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23901\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2442 sequences from chr1\n",
      "\tExtracting 987 sequences from chr10\n",
      "\tExtracting 1223 sequences from chr11\n",
      "\tExtracting 1257 sequences from chr12\n",
      "\tExtracting 466 sequences from chr13\n",
      "\tExtracting 827 sequences from chr14\n",
      "\tExtracting 775 sequences from chr15\n",
      "\tExtracting 1046 sequences from chr16\n",
      "\tExtracting 1612 sequences from chr17\n",
      "\tExtracting 343 sequences from chr18\n",
      "\tExtracting 1698 sequences from chr19\n",
      "\tExtracting 1586 sequences from chr2\n",
      "\tExtracting 616 sequences from chr20\n",
      "\tExtracting 330 sequences from chr21\n",
      "\tExtracting 624 sequences from chr22\n",
      "\tExtracting 1310 sequences from chr3\n",
      "\tExtracting 830 sequences from chr4\n",
      "\tExtracting 1005 sequences from chr5\n",
      "\tExtracting 1399 sequences from chr6\n",
      "\tExtracting 1070 sequences from chr7\n",
      "\tExtracting 828 sequences from chr8\n",
      "\tExtracting 947 sequences from chr9\n",
      "\tExtracting 645 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23866 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23903\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23903\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 2444 sequences from chr1\n",
      "\tExtracting 1023 sequences from chr10\n",
      "\tExtracting 1232 sequences from chr11\n",
      "\tExtracting 1205 sequences from chr12\n",
      "\tExtracting 420 sequences from chr13\n",
      "\tExtracting 842 sequences from chr14\n",
      "\tExtracting 738 sequences from chr15\n",
      "\tExtracting 1144 sequences from chr16\n",
      "\tExtracting 1546 sequences from chr17\n",
      "\tExtracting 353 sequences from chr18\n",
      "\tExtracting 1854 sequences from chr19\n",
      "\tExtracting 1582 sequences from chr2\n",
      "\tExtracting 618 sequences from chr20\n",
      "\tExtracting 348 sequences from chr21\n",
      "\tExtracting 585 sequences from chr22\n",
      "\tExtracting 1250 sequences from chr3\n",
      "\tExtracting 790 sequences from chr4\n",
      "\tExtracting 1069 sequences from chr5\n",
      "\tExtracting 1337 sequences from chr6\n",
      "\tExtracting 1076 sequences from chr7\n",
      "\tExtracting 832 sequences from chr8\n",
      "\tExtracting 977 sequences from chr9\n",
      "\tExtracting 600 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23865 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23903\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23903\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2488 sequences from chr1\n",
      "\tExtracting 894 sequences from chr10\n",
      "\tExtracting 1263 sequences from chr11\n",
      "\tExtracting 1264 sequences from chr12\n",
      "\tExtracting 411 sequences from chr13\n",
      "\tExtracting 830 sequences from chr14\n",
      "\tExtracting 669 sequences from chr15\n",
      "\tExtracting 1154 sequences from chr16\n",
      "\tExtracting 1599 sequences from chr17\n",
      "\tExtracting 295 sequences from chr18\n",
      "\tExtracting 2316 sequences from chr19\n",
      "\tExtracting 1494 sequences from chr2\n",
      "\tExtracting 614 sequences from chr20\n",
      "\tExtracting 401 sequences from chr21\n",
      "\tExtracting 541 sequences from chr22\n",
      "\tExtracting 1257 sequences from chr3\n",
      "\tExtracting 746 sequences from chr4\n",
      "\tExtracting 974 sequences from chr5\n",
      "\tExtracting 1253 sequences from chr6\n",
      "\tExtracting 1186 sequences from chr7\n",
      "\tExtracting 767 sequences from chr8\n",
      "\tExtracting 984 sequences from chr9\n",
      "\tExtracting 419 sequences from chrX\n",
      "\tExtracting 3 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23822 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset2/wellington.tsv.gz tmp_static/Subset2/homer.tsv.gz tmp_static/Subset2/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/wellington.tsv.gz data/gene.bed tmp_static/Subset2/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset2/binding.tsv.gz tmp_static/Subset2/tssdist.tsv.gz tmp_static/Subset2/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset2/linking.tsv.gz tmp_static/Subset2/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/binlinking.tsv.gz tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_loss.tsv.gz tmp_static/Subset2/net_stats.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2553 sequences from chr1\n",
      "\tExtracting 986 sequences from chr10\n",
      "\tExtracting 1331 sequences from chr11\n",
      "\tExtracting 1274 sequences from chr12\n",
      "\tExtracting 503 sequences from chr13\n",
      "\tExtracting 894 sequences from chr14\n",
      "\tExtracting 716 sequences from chr15\n",
      "\tExtracting 1170 sequences from chr16\n",
      "\tExtracting 1702 sequences from chr17\n",
      "\tExtracting 326 sequences from chr18\n",
      "\tExtracting 1894 sequences from chr19\n",
      "\tExtracting 1688 sequences from chr2\n",
      "\tExtracting 722 sequences from chr20\n",
      "\tExtracting 354 sequences from chr21\n",
      "\tExtracting 642 sequences from chr22\n",
      "\tExtracting 1300 sequences from chr3\n",
      "\tExtracting 818 sequences from chr4\n",
      "\tExtracting 1019 sequences from chr5\n",
      "\tExtracting 1477 sequences from chr6\n",
      "\tExtracting 1178 sequences from chr7\n",
      "\tExtracting 873 sequences from chr8\n",
      "\tExtracting 929 sequences from chr9\n",
      "\tExtracting 606 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24956 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2606 sequences from chr1\n",
      "\tExtracting 998 sequences from chr10\n",
      "\tExtracting 1266 sequences from chr11\n",
      "\tExtracting 1259 sequences from chr12\n",
      "\tExtracting 463 sequences from chr13\n",
      "\tExtracting 907 sequences from chr14\n",
      "\tExtracting 713 sequences from chr15\n",
      "\tExtracting 1227 sequences from chr16\n",
      "\tExtracting 1648 sequences from chr17\n",
      "\tExtracting 322 sequences from chr18\n",
      "\tExtracting 2059 sequences from chr19\n",
      "\tExtracting 1671 sequences from chr2\n",
      "\tExtracting 625 sequences from chr20\n",
      "\tExtracting 440 sequences from chr21\n",
      "\tExtracting 615 sequences from chr22\n",
      "\tExtracting 1280 sequences from chr3\n",
      "\tExtracting 808 sequences from chr4\n",
      "\tExtracting 1137 sequences from chr5\n",
      "\tExtracting 1366 sequences from chr6\n",
      "\tExtracting 1176 sequences from chr7\n",
      "\tExtracting 838 sequences from chr8\n",
      "\tExtracting 1000 sequences from chr9\n",
      "\tExtracting 503 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24927 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2600 sequences from chr1\n",
      "\tExtracting 916 sequences from chr10\n",
      "\tExtracting 1378 sequences from chr11\n",
      "\tExtracting 1294 sequences from chr12\n",
      "\tExtracting 437 sequences from chr13\n",
      "\tExtracting 859 sequences from chr14\n",
      "\tExtracting 638 sequences from chr15\n",
      "\tExtracting 1218 sequences from chr16\n",
      "\tExtracting 1707 sequences from chr17\n",
      "\tExtracting 303 sequences from chr18\n",
      "\tExtracting 2450 sequences from chr19\n",
      "\tExtracting 1542 sequences from chr2\n",
      "\tExtracting 607 sequences from chr20\n",
      "\tExtracting 565 sequences from chr21\n",
      "\tExtracting 587 sequences from chr22\n",
      "\tExtracting 1271 sequences from chr3\n",
      "\tExtracting 794 sequences from chr4\n",
      "\tExtracting 981 sequences from chr5\n",
      "\tExtracting 1293 sequences from chr6\n",
      "\tExtracting 1169 sequences from chr7\n",
      "\tExtracting 785 sequences from chr8\n",
      "\tExtracting 999 sequences from chr9\n",
      "\tExtracting 435 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24830 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2608 sequences from chr1\n",
      "\tExtracting 983 sequences from chr10\n",
      "\tExtracting 1315 sequences from chr11\n",
      "\tExtracting 1325 sequences from chr12\n",
      "\tExtracting 502 sequences from chr13\n",
      "\tExtracting 835 sequences from chr14\n",
      "\tExtracting 838 sequences from chr15\n",
      "\tExtracting 1116 sequences from chr16\n",
      "\tExtracting 1658 sequences from chr17\n",
      "\tExtracting 348 sequences from chr18\n",
      "\tExtracting 1816 sequences from chr19\n",
      "\tExtracting 1691 sequences from chr2\n",
      "\tExtracting 657 sequences from chr20\n",
      "\tExtracting 339 sequences from chr21\n",
      "\tExtracting 651 sequences from chr22\n",
      "\tExtracting 1327 sequences from chr3\n",
      "\tExtracting 784 sequences from chr4\n",
      "\tExtracting 1072 sequences from chr5\n",
      "\tExtracting 1413 sequences from chr6\n",
      "\tExtracting 1150 sequences from chr7\n",
      "\tExtracting 917 sequences from chr8\n",
      "\tExtracting 1014 sequences from chr9\n",
      "\tExtracting 608 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24967 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset1/wellington.tsv.gz tmp_static/Subset1/homer.tsv.gz tmp_static/Subset1/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/wellington.tsv.gz data/gene.bed tmp_static/Subset1/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset1/binding.tsv.gz tmp_static/Subset1/tssdist.tsv.gz tmp_static/Subset1/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset1/linking.tsv.gz tmp_static/Subset1/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/binlinking.tsv.gz tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_loss.tsv.gz tmp_static/Subset1/net_stats.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset8/net_iweight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_inweight.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset4/footprints.bed data/motifs.motif data/genome tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/motifs.bed tmp_static/Subset4/wellington.tsv.gz tmp_static/Subset4/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 21700\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 21700\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2248 sequences from chr1\n",
      "\tExtracting 843 sequences from chr10\n",
      "\tExtracting 1159 sequences from chr11\n",
      "\tExtracting 1097 sequences from chr12\n",
      "\tExtracting 425 sequences from chr13\n",
      "\tExtracting 755 sequences from chr14\n",
      "\tExtracting 660 sequences from chr15\n",
      "\tExtracting 963 sequences from chr16\n",
      "\tExtracting 1446 sequences from chr17\n",
      "\tExtracting 344 sequences from chr18\n",
      "\tExtracting 1471 sequences from chr19\n",
      "\tExtracting 1494 sequences from chr2\n",
      "\tExtracting 606 sequences from chr20\n",
      "\tExtracting 287 sequences from chr21\n",
      "\tExtracting 559 sequences from chr22\n",
      "\tExtracting 1223 sequences from chr3\n",
      "\tExtracting 708 sequences from chr4\n",
      "\tExtracting 928 sequences from chr5\n",
      "\tExtracting 1208 sequences from chr6\n",
      "\tExtracting 995 sequences from chr7\n",
      "\tExtracting 759 sequences from chr8\n",
      "\tExtracting 994 sequences from chr9\n",
      "\tExtracting 498 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t21670 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 21701\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 21701\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2193 sequences from chr1\n",
      "\tExtracting 873 sequences from chr10\n",
      "\tExtracting 1161 sequences from chr11\n",
      "\tExtracting 1089 sequences from chr12\n",
      "\tExtracting 381 sequences from chr13\n",
      "\tExtracting 696 sequences from chr14\n",
      "\tExtracting 659 sequences from chr15\n",
      "\tExtracting 1011 sequences from chr16\n",
      "\tExtracting 1534 sequences from chr17\n",
      "\tExtracting 316 sequences from chr18\n",
      "\tExtracting 1742 sequences from chr19\n",
      "\tExtracting 1376 sequences from chr2\n",
      "\tExtracting 640 sequences from chr20\n",
      "\tExtracting 267 sequences from chr21\n",
      "\tExtracting 601 sequences from chr22\n",
      "\tExtracting 1151 sequences from chr3\n",
      "\tExtracting 712 sequences from chr4\n",
      "\tExtracting 953 sequences from chr5\n",
      "\tExtracting 1145 sequences from chr6\n",
      "\tExtracting 1024 sequences from chr7\n",
      "\tExtracting 786 sequences from chr8\n",
      "\tExtracting 929 sequences from chr9\n",
      "\tExtracting 426 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t21665 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 21701\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 21701\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2269 sequences from chr1\n",
      "\tExtracting 872 sequences from chr10\n",
      "\tExtracting 1157 sequences from chr11\n",
      "\tExtracting 1147 sequences from chr12\n",
      "\tExtracting 349 sequences from chr13\n",
      "\tExtracting 717 sequences from chr14\n",
      "\tExtracting 643 sequences from chr15\n",
      "\tExtracting 1011 sequences from chr16\n",
      "\tExtracting 1552 sequences from chr17\n",
      "\tExtracting 312 sequences from chr18\n",
      "\tExtracting 1579 sequences from chr19\n",
      "\tExtracting 1455 sequences from chr2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 649 sequences from chr20\n",
      "\tExtracting 289 sequences from chr21\n",
      "\tExtracting 538 sequences from chr22\n",
      "\tExtracting 1189 sequences from chr3\n",
      "\tExtracting 682 sequences from chr4\n",
      "\tExtracting 948 sequences from chr5\n",
      "\tExtracting 1139 sequences from chr6\n",
      "\tExtracting 996 sequences from chr7\n",
      "\tExtracting 801 sequences from chr8\n",
      "\tExtracting 921 sequences from chr9\n",
      "\tExtracting 456 sequences from chrX\n",
      "\tExtracting 3 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t21674 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 21701\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 21701\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2262 sequences from chr1\n",
      "\tExtracting 888 sequences from chr10\n",
      "\tExtracting 1155 sequences from chr11\n",
      "\tExtracting 1093 sequences from chr12\n",
      "\tExtracting 342 sequences from chr13\n",
      "\tExtracting 706 sequences from chr14\n",
      "\tExtracting 582 sequences from chr15\n",
      "\tExtracting 1083 sequences from chr16\n",
      "\tExtracting 1558 sequences from chr17\n",
      "\tExtracting 281 sequences from chr18\n",
      "\tExtracting 2147 sequences from chr19\n",
      "\tExtracting 1379 sequences from chr2\n",
      "\tExtracting 614 sequences from chr20\n",
      "\tExtracting 246 sequences from chr21\n",
      "\tExtracting 574 sequences from chr22\n",
      "\tExtracting 1179 sequences from chr3\n",
      "\tExtracting 675 sequences from chr4\n",
      "\tExtracting 833 sequences from chr5\n",
      "\tExtracting 1069 sequences from chr6\n",
      "\tExtracting 1058 sequences from chr7\n",
      "\tExtracting 693 sequences from chr8\n",
      "\tExtracting 884 sequences from chr9\n",
      "\tExtracting 325 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t21627 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset4/wellington.tsv.gz tmp_static/Subset4/homer.tsv.gz tmp_static/Subset4/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/wellington.tsv.gz data/gene.bed tmp_static/Subset4/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset4/binding.tsv.gz tmp_static/Subset4/tssdist.tsv.gz tmp_static/Subset4/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset4/linking.tsv.gz tmp_static/Subset4/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/binlinking.tsv.gz tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_loss.tsv.gz tmp_static/Subset4/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset3/footprints.bed data/motifs.motif data/genome tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/motifs.bed tmp_static/Subset3/wellington.tsv.gz tmp_static/Subset3/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2544 sequences from chr1\n",
      "\tExtracting 1032 sequences from chr10\n",
      "\tExtracting 1341 sequences from chr11\n",
      "\tExtracting 1344 sequences from chr12\n",
      "\tExtracting 472 sequences from chr13\n",
      "\tExtracting 834 sequences from chr14\n",
      "\tExtracting 733 sequences from chr15\n",
      "\tExtracting 1164 sequences from chr16\n",
      "\tExtracting 1562 sequences from chr17\n",
      "\tExtracting 371 sequences from chr18\n",
      "\tExtracting 1960 sequences from chr19\n",
      "\tExtracting 1617 sequences from chr2\n",
      "\tExtracting 678 sequences from chr20\n",
      "\tExtracting 367 sequences from chr21\n",
      "\tExtracting 659 sequences from chr22\n",
      "\tExtracting 1318 sequences from chr3\n",
      "\tExtracting 841 sequences from chr4\n",
      "\tExtracting 1036 sequences from chr5\n",
      "\tExtracting 1371 sequences from chr6\n",
      "\tExtracting 1213 sequences from chr7\n",
      "\tExtracting 953 sequences from chr8\n",
      "\tExtracting 1019 sequences from chr9\n",
      "\tExtracting 525 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24955 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2490 sequences from chr1\n",
      "\tExtracting 1106 sequences from chr10\n",
      "\tExtracting 1315 sequences from chr11\n",
      "\tExtracting 1310 sequences from chr12\n",
      "\tExtracting 460 sequences from chr13\n",
      "\tExtracting 852 sequences from chr14\n",
      "\tExtracting 743 sequences from chr15\n",
      "\tExtracting 1076 sequences from chr16\n",
      "\tExtracting 1623 sequences from chr17\n",
      "\tExtracting 386 sequences from chr18\n",
      "\tExtracting 1698 sequences from chr19\n",
      "\tExtracting 1754 sequences from chr2\n",
      "\tExtracting 718 sequences from chr20\n",
      "\tExtracting 385 sequences from chr21\n",
      "\tExtracting 587 sequences from chr22\n",
      "\tExtracting 1381 sequences from chr3\n",
      "\tExtracting 807 sequences from chr4\n",
      "\tExtracting 1110 sequences from chr5\n",
      "\tExtracting 1430 sequences from chr6\n",
      "\tExtracting 1211 sequences from chr7\n",
      "\tExtracting 898 sequences from chr8\n",
      "\tExtracting 1070 sequences from chr9\n",
      "\tExtracting 560 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24970 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2554 sequences from chr1\n",
      "\tExtracting 1087 sequences from chr10\n",
      "\tExtracting 1271 sequences from chr11\n",
      "\tExtracting 1294 sequences from chr12\n",
      "\tExtracting 449 sequences from chr13\n",
      "\tExtracting 859 sequences from chr14\n",
      "\tExtracting 846 sequences from chr15\n",
      "\tExtracting 1075 sequences from chr16\n",
      "\tExtracting 1635 sequences from chr17\n",
      "\tExtracting 405 sequences from chr18\n",
      "\tExtracting 1622 sequences from chr19\n",
      "\tExtracting 1717 sequences from chr2\n",
      "\tExtracting 763 sequences from chr20\n",
      "\tExtracting 379 sequences from chr21\n",
      "\tExtracting 632 sequences from chr22\n",
      "\tExtracting 1474 sequences from chr3\n",
      "\tExtracting 856 sequences from chr4\n",
      "\tExtracting 1034 sequences from chr5\n",
      "\tExtracting 1411 sequences from chr6\n",
      "\tExtracting 1123 sequences from chr7\n",
      "\tExtracting 910 sequences from chr8\n",
      "\tExtracting 999 sequences from chr9\n",
      "\tExtracting 562 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24958 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys3k/data/genome/genome.fa)\n",
      "\tExtracting 2590 sequences from chr1\n",
      "\tExtracting 956 sequences from chr10\n",
      "\tExtracting 1327 sequences from chr11\n",
      "\tExtracting 1287 sequences from chr12\n",
      "\tExtracting 404 sequences from chr13\n",
      "\tExtracting 837 sequences from chr14\n",
      "\tExtracting 677 sequences from chr15\n",
      "\tExtracting 1169 sequences from chr16\n",
      "\tExtracting 1825 sequences from chr17\n",
      "\tExtracting 355 sequences from chr18\n",
      "\tExtracting 2365 sequences from chr19\n",
      "\tExtracting 1644 sequences from chr2\n",
      "\tExtracting 669 sequences from chr20\n",
      "\tExtracting 340 sequences from chr21\n",
      "\tExtracting 619 sequences from chr22\n",
      "\tExtracting 1370 sequences from chr3\n",
      "\tExtracting 773 sequences from chr4\n",
      "\tExtracting 1022 sequences from chr5\n",
      "\tExtracting 1299 sequences from chr6\n",
      "\tExtracting 1167 sequences from chr7\n",
      "\tExtracting 804 sequences from chr8\n",
      "\tExtracting 1049 sequences from chr9\n",
      "\tExtracting 393 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24942 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset3/wellington.tsv.gz tmp_static/Subset3/homer.tsv.gz tmp_static/Subset3/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/wellington.tsv.gz data/gene.bed tmp_static/Subset3/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset3/binding.tsv.gz tmp_static/Subset3/tssdist.tsv.gz tmp_static/Subset3/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset3/linking.tsv.gz tmp_static/Subset3/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/binlinking.tsv.gz tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_loss.tsv.gz tmp_static/Subset3/net_stats.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset6/net_iweight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset5/net_iweight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset7/net_iweight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset9/net_iweight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset4/net_iweight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset1/net_iweight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset2/net_iweight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset3/net_iweight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_inweight.tsv.gz\n",
      "make: Nothing to be done for 'cpu'.\n",
      "make: Nothing to be done for 'gpu'.\n",
      "make: Nothing to be done for 'cpu'.\n",
      "mkdir -p output/\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network tofile  data tmp_static data/subsets.txt output/static.h5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"dictys_helper network_inference.sh -j 32 -J 1 static\"\n",
      "\tUser time (seconds): 298030.63\n",
      "\tSystem time (seconds): 11741.26\n",
      "\tPercent of CPU this job got: 1650%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 5:12:53\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 3537696\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 2645\n",
      "\tMinor (reclaiming a frame) page faults: 821627917\n",
      "\tVoluntary context switches: 89403623\n",
      "\tInvoluntary context switches: 50477882\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 39619872\n",
      "\tFile system outputs: 60216152\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd .; \n",
    "/usr/bin/time -v dictys_helper network_inference.sh -j 32 -J 1 static "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "peak memory: 74.64 MiB, increment: 0.00 MiB\n"
     ]
    }
   ],
   "source": [
    "%memit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python (dictys)",
   "language": "python",
   "name": "dictys"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
