{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext memory_profiler"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Preparation of individual input files in `data` folder\n",
    "After data processing with CellRanger, you need the following files:\n",
    "* RNA read count matrix (\"Filtered feature barcode matrix MEX (DIR)\")\n",
    "* ATAC reads (\"ATAC Position-sorted alignments (BAM)\")\n",
    "* Optionally or from other softwares: cell clustering (inside \"Secondary analysis outputs (DIR)\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.conda/envs/dictys/bin:/opt/gridware/apps/gcc/R/4.3.2/bin:/opt/gridware/compilers/gcc/11.2.0/bin:/opt/gridware2/apps/binapps/anaconda3/2019.07/bin:/opt/gridware2/apps/binapps/anaconda3/2019.07/condabin:/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.local/bin:/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/bin:/usr/share/Modules/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin\n"
     ]
    }
   ],
   "source": [
    "#Add path\n",
    "import os\n",
    "os.environ['PATH'] = os.path.expanduser('~/.conda/envs/dictys/bin') + ':' + os.environ['PATH']\n",
    "print(os.getenv('PATH'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys\n"
     ]
    }
   ],
   "source": [
    "# Removes CPU usage limit by some jupyter versions\n",
    "import os\n",
    "os.environ['KMP_AFFINITY'] = ''\n",
    "#Create input data folder\n",
    "current_path = os.getcwd()\n",
    "print(current_path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### expression.tsv.gz\n",
    "Read count matrix of RNA-profiled cells in compressed tsv format. Downloaded and converted from \"Filtered feature barcode matrix MEX (DIR)\"."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:root:Skipped duplicate occurrence of gene names: TBCE,LINC01238,CYB561D2,MATR3,LINC01505,HSPA14,GOLGA8M,GGT1,ARMCX5-GPRASP2,TMSB15B\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Download expression data in mtx.gz format\n",
    "wget -q -o /dev/null https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_filtered_feature_bc_matrix.tar.gz\n",
    "tar xf pbmc_granulocyte_sorted_10k_filtered_feature_bc_matrix.tar.gz\n",
    "rm pbmc_granulocyte_sorted_10k_filtered_feature_bc_matrix.tar.gz\n",
    "# Convert from mtx.gz to tsv.gz format using helper script `expression_mtx.py`.\n",
    "dictys_helper expression_mtx.py filtered_feature_bc_matrix expression.tsv.gz\n",
    "rm -Rf filtered_feature_bc_matrix\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### bams\n",
    "This folder contains one bam file for each cell with chromatin accessibility measurement. File name should be cell name. Downloaded and converted from \"ATAC Position-sorted alignments (BAM)\".\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Setting --progress (progress) to bar:force:noscroll\n",
      "Setting --output-document (outputdocument) to bams.bam\n",
      "DEBUG output created by Wget 1.21.4 on linux-gnu.\n",
      "\n",
      "Reading HSTS entries from /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/.wget-hsts\n",
      "URI encoding = ‘UTF-8’\n",
      "--2024-07-18 23:47:43--  https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_atac_possorted_bam.bam\n",
      "Resolving cf.10xgenomics.com (cf.10xgenomics.com)... 104.18.1.173, 104.18.0.173, 2606:4700::6812:1ad, ...\n",
      "Caching cf.10xgenomics.com => 104.18.1.173 104.18.0.173 2606:4700::6812:1ad 2606:4700::6812:ad\n",
      "Connecting to cf.10xgenomics.com (cf.10xgenomics.com)|104.18.1.173|:443... connected.\n",
      "Created socket 4.\n",
      "Releasing 0x000055ae7779bcf0 (new refcount 1).\n",
      "Initiating SSL handshake.\n",
      "Handshake successful; connected socket 4 to SSL handle 0x000055ae7779d830\n",
      "certificate:\n",
      "  subject: CN=10xgenomics.com\n",
      "  issuer:  CN=E1,O=Let's Encrypt,C=US\n",
      "X509 certificate successfully verified and matches host cf.10xgenomics.com\n",
      "\n",
      "---request begin---\n",
      "GET /samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_atac_possorted_bam.bam HTTP/1.1\n",
      "Host: cf.10xgenomics.com\n",
      "User-Agent: Wget/1.21.4\n",
      "Accept: */*\n",
      "Accept-Encoding: identity\n",
      "Connection: Keep-Alive\n",
      "\n",
      "---request end---\n",
      "HTTP request sent, awaiting response... \n",
      "---response begin---\n",
      "HTTP/1.1 200 OK\n",
      "Date: Thu, 18 Jul 2024 22:47:44 GMT\n",
      "Content-Type: binary/octet-stream\n",
      "Content-Length: 46894629363\n",
      "Connection: keep-alive\n",
      "last-modified: Sun, 06 Sep 2020 06:00:11 GMT\n",
      "etag: \"9e641d777a04a1db59a9433820a2061a-5591\"\n",
      "x-amz-storage-class: INTELLIGENT_TIERING\n",
      "x-amz-version-id: VfR90GBGWVLkI6Lq4DeewOIZt6IZp9Di\n",
      "accept-ranges: bytes\n",
      "x-cache: Miss from cloudfront\n",
      "via: 1.1 6bb8eb5712512d0233fdbd1d8dcf84b6.cloudfront.net (CloudFront)\n",
      "x-amz-cf-pop: LHR5-P5\n",
      "x-amz-cf-id: 0T9Cc99lQ-4_q1lRokGSeo0pEE_TABovPCuzgsKYhy-BHnXHE3nkjA==\n",
      "CF-Cache-Status: DYNAMIC\n",
      "Server: cloudflare\n",
      "CF-RAY: 8a5603e1ad36b101-MAN\n",
      "\n",
      "---response end---\n",
      "200 OK\n",
      "Registered socket 4 for persistent reuse.\n",
      "Length: 46894629363 (44G) [binary/octet-stream]\n",
      "Saving to: ‘bams.bam’\n",
      "\n",
      "bams.bam            100%[===================>]  43.67G  29.0MB/s    in 35m 6s  \n",
      "\n",
      "2024-07-19 00:22:50 (21.2 MB/s) - ‘bams.bam’ saved [46894629363/46894629363]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "set -eo pipefail\n",
    "cd ./data\n",
    "# Download chromatin accessibility reads in bam format\n",
    "wget --debug --progress=bar:force:noscroll -O bams.bam https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_atac_possorted_bam.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "set -eo pipefail\n",
    "cd ./data\n",
    "dictys_helper split_bam.sh bams.bam bams --section \"CB:Z:\" --ref_expression expression.tsv.gz\n",
    "rm bams.bam"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### subsets & subsets.txt\n",
    "* subsets.txt: Names of cell subsets. For each cell subset, a GRN is reconstructed.\n",
    "* subsets: Folder containing one subfolder for each cell subset as in `subsets.txt`. Each subfolder contains two files:\n",
    "    - names_rna.txt: Names of cells that belong to this subset and have transcriptome measurement\n",
    "    - names_atac.txt: Names of cells that belong to this subset and have chromatin accessibility measurement\n",
    "    - For joint measurements of RNA and ATAC, these two files should be identical in every folder.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "wget -q -o /dev/null https://cf.10xgenomics.com/samples/cell-arc/1.0.0/pbmc_granulocyte_sorted_10k/pbmc_granulocyte_sorted_10k_analysis.tar.gz\n",
    "#Extract cell names for each cluster\n",
    "tar xf pbmc_granulocyte_sorted_10k_analysis.tar.gz \n",
    "mv analysis/clustering/gex/graphclust/clusters.csv clusters.csv\n",
    "rm -Rf pbmc_granulocyte_sorted_10k_analysis.tar.gz analysis\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Finally, reformat clusters.csv for input:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "subsets=\"$(tail -n +2 clusters.csv | awk -F , '{print $2}' | sort -u)\"\n",
    "echo \"$subsets\" | awk '{print \"Subset\"$1}' > subsets.txt\n",
    "for x in $subsets; do\n",
    "\tmkdir -p \"subsets/Subset$x\"\n",
    "\tgrep \",$x\"'$' clusters.csv | awk -F , '{print $1}' > \"subsets/Subset$x/names_rna.txt\"\n",
    "\t# RNA and ATAC barcodes are the same for joint quantifications\n",
    "\tcp \"subsets/Subset$x/names_rna.txt\" \"subsets/Subset$x/names_atac.txt\"\n",
    "done\n",
    "rm clusters.csv\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Other files\n",
    "Motif, reference genome, gene transcriptional start site"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: skipped motif file because not found: rorc.motif\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "\n",
    "# Motifs (file motifs.motif)\n",
    "# from HOCOMOCO (https://hocomoco11.autosome.org/)\n",
    "wget -q -o /dev/null -O motifs.motif 'https://hocomoco11.autosome.org/final_bundle/hocomoco11/full/HUMAN/mono/HOCOMOCOv11_full_HUMAN_mono_homer_format_0.0001.motif'\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">dKhGCGTGh\tAHR_HUMAN.H11MO.0.B\t3.3775000000000004\r\n",
      "0.262728374765856\t0.1227600511842322\t0.362725638699551\t0.25178593535036087\r\n",
      "0.07633328991810645\t0.08258130543118362\t0.22593295481662123\t0.6151524498340887\r\n",
      "0.14450570038747923\t0.28392173880411337\t0.13815442099009081\t0.4334181398183167\r\n",
      "0.023935814057894068\t0.016203821748029118\t0.9253278681170539\t0.03453249607702277\r\n",
      "0.007919544273173793\t0.953597675415874\t0.017308392078009837\t0.021174388232942286\r\n",
      "0.02956192959210962\t0.012890110758086997\t0.9474192747166682\t0.010128684933135217\r\n",
      "0.007919544273173797\t0.029561929592109615\t0.012337825593096645\t0.9501807005416201\r\n",
      "0.007919544273173793\t0.007919544273173793\t0.9762413671804787\t0.007919544273173793\r\n",
      "0.27886589130660366\t0.4285328543459993\t0.10955683916661985\t0.18304441518077724\r\n",
      ">hnnGGWWnddWWGGdbWh\tAIRE_HUMAN.H11MO.0.C\t5.64711\r\n",
      "0.38551919443239085\t0.2604245534178759\t0.1353299124033618\t0.21872633974637148\r\n",
      "0.18745267949274294\t0.18745267949274294\t0.14575446582123766\t0.4793401751932764\r\n",
      "0.14575446582123777\t0.14575446582123777\t0.2083017863284958\t0.5001892820290286\r\n",
      "0.0623580384782284\t0.020659824806723626\t0.8337749914010671\t0.08320714531398087\r\n",
      "0.020659824806723633\t0.020659824806723633\t0.7712276708938095\t0.18745267949274314\r\n",
      "0.3125473205072573\t0.062358038478228406\t0.04150893164247602\t0.5835857093720382\r\n",
      "0.37509464101451423\t0.10405625214973326\t0.04150893164247601\t0.4793401751932766\r\n"
     ]
    }
   ],
   "source": [
    "!head -n 18 ./data/motifs.motif"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Process is interrupted.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Reference genome (folder genome)\n",
    "# Download genome from HOMER\n",
    "dictys_helper genome_homer.sh hg38 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 5.3G\n",
      "113K annotations\n",
      "153K chrom.sizes\n",
      "3.7G genome.fa\n",
      "4.2M hg38.aug\n",
      " 51M hg38.basic.annotation\n",
      "822M hg38.full.annotation\n",
      "641K hg38.miRNA\n",
      "616M hg38.repeats\n",
      " 32M hg38.rna\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -h1s ./data/genome | head"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "cd ./data\n",
    "# Bed file for TSS (file gene.bed)\n",
    "# Download gtf file from ensembl\n",
    "wget -q -o /dev/null -O gene.gtf.gz http://ftp.ensembl.org/pub/release-107/gtf/homo_sapiens/Homo_sapiens.GRCh38.107.gtf.gz\n",
    "gunzip gene.gtf.gz\n",
    "# Convert to bed\n",
    "dictys_helper gene_gtf.sh gene.gtf gene.bed\n",
    "rm gene.gtf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1\t11869\t14409\tDDX11L1\t.\t+\r\n",
      "chr1\t14404\t29570\tWASH7P\t.\t-\r\n",
      "chr1\t17369\t17436\tMIR6859-1\t.\t-\r\n",
      "chr1\t29554\t31109\tMIR1302-2HG\t.\t+\r\n",
      "chr1\t30366\t30503\tMIR1302-2\t.\t+\r\n",
      "chr1\t34554\t36081\tFAM138A\t.\t-\r\n",
      "chr1\t52473\t53312\tOR4G4P\t.\t+\r\n",
      "chr1\t57598\t64116\tOR4G11P\t.\t+\r\n",
      "chr1\t65419\t71585\tOR4F5\t.\t+\r\n",
      "chr1\t131025\t134836\tCICP27\t.\t+\r\n"
     ]
    }
   ],
   "source": [
    "!head ./data/gene.bed"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Clean up"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "!rm -Rf ./data/filtered_feature_bc_matrix"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Network inference configuration\n",
    "### Generate configurations\n",
    "Please adjust them for your own machine and dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "# Generate configuration template\n",
    "rm -Rf ./makefiles\n",
    "mkdir ./makefiles\n",
    "cd ./makefiles\n",
    "dictys_helper makefile_template.sh common.mk config.mk env_none.mk static.mk\n",
    "\n",
    "# Update configurations, such as:\n",
    "# DEVICE: pytorch device, e.g. cpu, cuda:0. If you do not have a GPU, use 'cpu' and expect LONG computing time.\n",
    "# GENOME_MACS2: effective genome size for macs2. See https://deeptools.readthedocs.io/en/develop/content/feature/effectiveGenomeSize.html\n",
    "# JOINT: whether dataset is joint profiling of RNA and ATAC.\n",
    "# Other configurations include quality control thresholds, number of threads in each job, number of hidden confounders, etc.\n",
    "# They can be obtained in the full-multiome tutorial.\n",
    "dictys_helper makefile_update.py ../makefiles/config.mk '{\"DEVICE\": \"cpu\", \"GENOME_MACS2\": \"hs\", \"JOINT\": \"1\"}'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Lingfei Wang, 2022. All rights reserved.\r\n",
      "#This file contains parameters for whole run and individual steps to be edited for your dataset\r\n",
      "#This file should be edited to configure the run\r\n",
      "#This file should NOT be directly used for any run with `makefile -f` \r\n",
      "\r\n",
      "############################################################\r\n",
      "# Run environment settings\r\n",
      "############################################################\r\n",
      "#Which environment to use, corresponding to env_$(ENVMODE).mk file\r\n",
      "ENVMODE=none\r\n",
      "#Maximum number of CPU threads for each job\r\n",
      "#This is only nominated and passed through to other softwares without any guarantee.\r\n",
      "NTH=4\r\n",
      "#Device name for pyro/pytorch\r\n",
      "#Note: cuda devices other than cuda:0 could be incompatible with singularity environment\r\n",
      "DEVICE=cpu\r\n",
      "\r\n",
      "############################################################\r\n",
      "# Dataset settings\r\n",
      "############################################################\r\n",
      "\r\n",
      "#Genome size for Macs2, accept shortcuts like mm & hs\r\n",
      "GENOME_MACS2=hs\r\n",
      "#Whether dataset is joint profiling of RNA & ATAC of same cell. Separate measurements of two modalities in different cells: 0. Joint measurements: 1.\r\n",
      "JOINT=1\r\n",
      "\r\n",
      "#######################################################################################\r\n",
      "# Parameters of individual steps shared by cell-type specific and dynamic GRN inference\r\n",
      "#######################################################################################\r\n",
      "\r\n",
      "PARAMS-PREPROC-QC_READS:=50 10 0 200 100 0\r\n",
      "PARAMS-CHROMATIN-MACS2:=$(GENOME_MACS2)\r\n",
      "PARAMS-CHROMATIN-BINLINKING:=20\r\n",
      "KPARAMS-PREPROC-SELECTSC_RNA:=\r\n",
      "KPARAMS-PREPROC-QC_READS:=\r\n",
      "KPARAMS-PREPROC-SELECTSC_ATAC:=\r\n",
      "KPARAMS-CHROMATIN-MACS2:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-WELLINGTON:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-HOMER:=--nth $(NTH)\r\n",
      "KPARAMS-CHROMATIN-BINDING:=\r\n",
      "KPARAMS-CHROMATIN-TSSDIST:=\r\n",
      "KPARAMS-CHROMATIN-LINKING:=\r\n",
      "KPARAMS-CHROMATIN-BINLINKING:=\r\n",
      "KPARAMS-NETWORK-RECONSTRUCT:=--device $(DEVICE) --nth $(NTH)\r\n",
      "KPARAMS-NETWORK-INDIRECT:=--nth $(NTH)\r\n",
      "KPARAMS-NETWORK-NORMALIZE:=--nth $(NTH)\r\n",
      "KPARAMS-NETWORK-TOFILE:=$(KPARAMS-NETWORK-TOFILE_EXTRA)\r\n",
      "\r\n",
      "#######################################################################################\r\n",
      "# Parameters of individual steps only in dynamic GRN inference\r\n",
      "#######################################################################################\r\n",
      "\r\n",
      "PARAMS-DYNAMIC-SUBSETS_RNA=1000 875 0.05\r\n",
      "PARAMS-DYNAMIC-SUBSETS_ATAC=$(firstword $(PARAMS-DYNAMIC-SUBSETS_RNA))\r\n"
     ]
    }
   ],
   "source": [
    "!cat ./makefiles/config.mk"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Validate input data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"cd .\"\n",
      "\tUser time (seconds): 0.00\n",
      "\tSystem time (seconds): 0.00\n",
      "\tPercent of CPU this job got: 33%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:00.00\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 3328\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 0\n",
      "\tMinor (reclaiming a frame) page faults: 189\n",
      "\tVoluntary context switches: 2\n",
      "\tInvoluntary context switches: 0\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 0\n",
      "\tFile system outputs: 0\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n",
      "WARNING:root:Chromosomes not found in reference genome: chrGL000194.1,chrMT\n",
      "WARNING:root:Cannot find dynamic.mk or traj_node.h5. Skipping dynamic network inference checks.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Joint profile: True\n",
      "Found 11909 cells with RNA profile\n",
      "Found 24036 genes with RNA profile\n",
      "Found 11909 cells with ATAC profile\n",
      "Found 769 motifs\n",
      "Found 678 TFs\n",
      "Found 461 TFs in current dataset\n",
      "Missing 217 TFs in current dataset: ANDR,AP2A,AP2B,AP2C,AP2D,ARI3A,ARI5B,ATF6A,BARH1,BARH2,BC11A,BHA15,BHE22,BHE23,BHE40,BHE41,BMAL1,BRAC,BSH,COE1,COT1,COT2,CR3L1,CR3L2,ERR1,ERR2,ERR3,EVI1,GCR,HEN1,HMBX1,HME1,HME2,HNF6,HTF4,HXA1,HXA10,HXA11,HXA13,HXA2,HXA5,HXA7,HXA9,HXB1,HXB13,HXB2,HXB3,HXB4,HXB6,HXB7,HXB8,HXC10,HXC11,HXC12,HXC13,HXC6,HXC8,HXC9,HXD10,HXD11,HXD12,HXD13,HXD3,HXD4,HXD8,HXD9,ITF2,KAISO,MCR,MGAP,MLXPL,MYBA,MYBB,NDF1,NDF2,NF2L1,NF2L2,NFAC1,NFAC2,NFAC3,NFAC4,NGN2,NKX21,NKX22,NKX23,NKX25,NKX28,NKX31,NKX32,NKX61,NKX62,ONEC2,ONEC3,OZF,P53,P5F1B,P63,P73,PEBB,PHX2A,PHX2B,PIT1,PKNX1,PLAL1,PO2F1,PO2F2,PO2F3,PO3F1,PO3F2,PO3F3,PO3F4,PO4F1,PO4F2,PO4F3,PO5F1,PO6F1,PO6F2,PRD14,PRGR,RHXF1,RORG,RX,SMCA1,SMCA5,SRBP1,SRBP2,STA5A,STA5B,STF1,SUH,TF2LX,TF65,TF7L1,TF7L2,TFE2,THA,THA11,THB,TWST1,TYY1,TYY2,UBIP1,UNC4,Z324A,Z354A,ZBT14,ZBT17,ZBT18,ZBT48,ZBT49,ZBT7A,ZBT7B,ZEP1,ZEP2,ZF64A,ZKSC1,ZKSC3,ZN121,ZN134,ZN136,ZN140,ZN143,ZN148,ZN214,ZN219,ZN232,ZN250,ZN257,ZN260,ZN263,ZN264,ZN274,ZN281,ZN282,ZN317,ZN320,ZN322,ZN329,ZN331,ZN333,ZN335,ZN341,ZN350,ZN382,ZN384,ZN394,ZN410,ZN418,ZN423,ZN436,ZN449,ZN467,ZN490,ZN502,ZN524,ZN528,ZN547,ZN549,ZN554,ZN563,ZN582,ZN586,ZN589,ZN652,ZN667,ZN680,ZN708,ZN713,ZN740,ZN768,ZN770,ZN784,ZN816,ZSC16,ZSC22,ZSC31,ZSCA4\n",
      "Found 23533 genes with TSS information\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"dictys_helper makefile_check.py\"\n",
      "\tUser time (seconds): 20.91\n",
      "\tSystem time (seconds): 3.98\n",
      "\tPercent of CPU this job got: 53%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:46.11\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 7925516\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 0\n",
      "\tMinor (reclaiming a frame) page faults: 2119390\n",
      "\tVoluntary context switches: 69043\n",
      "\tInvoluntary context switches: 561\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 6440144\n",
      "\tFile system outputs: 0\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/usr/bin/time -v cd . &&  /usr/bin/time -v dictys_helper makefile_check.py"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Network inference"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "print(os.cpu_count())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "peak memory: 65.46 MiB, increment: 0.25 MiB\n"
     ]
    }
   ],
   "source": [
    "%memit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"cd .\"\n",
      "\tUser time (seconds): 0.00\n",
      "\tSystem time (seconds): 0.00\n",
      "\tPercent of CPU this job got: 100%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:00.00\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 3328\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 0\n",
      "\tMinor (reclaiming a frame) page faults: 188\n",
      "\tVoluntary context switches: 1\n",
      "\tInvoluntary context switches: 0\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 0\n",
      "\tFile system outputs: 0\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir -p tmp_static/Subset1/\n",
      "cp data/subsets/Subset1/names_rna.txt tmp_static/Subset1/names_rna.txt\n",
      "mkdir -p tmp_static/Subset10/\n",
      "mkdir -p tmp_static/Subset11/\n",
      "cp data/subsets/Subset10/names_rna.txt tmp_static/Subset10/names_rna.txt\n",
      "cp data/subsets/Subset11/names_rna.txt tmp_static/Subset11/names_rna.txt\n",
      "mkdir -p tmp_static/Subset12/\n",
      "cp data/subsets/Subset12/names_rna.txt tmp_static/Subset12/names_rna.txt\n",
      "mkdir -p tmp_static/Subset13/\n",
      "cp data/subsets/Subset13/names_rna.txt tmp_static/Subset13/names_rna.txt\n",
      "mkdir -p tmp_static/Subset14/\n",
      "cp data/subsets/Subset14/names_rna.txt tmp_static/Subset14/names_rna.txt\n",
      "mkdir -p tmp_static/Subset15/\n",
      "cp data/subsets/Subset15/names_rna.txt tmp_static/Subset15/names_rna.txt\n",
      "mkdir -p tmp_static/Subset16/\n",
      "mkdir -p tmp_static/Subset17/\n",
      "cp data/subsets/Subset16/names_rna.txt tmp_static/Subset16/names_rna.txt\n",
      "cp data/subsets/Subset17/names_rna.txt tmp_static/Subset17/names_rna.txt\n",
      "mkdir -p tmp_static/Subset2/\n",
      "mkdir -p tmp_static/Subset3/\n",
      "cp data/subsets/Subset2/names_rna.txt tmp_static/Subset2/names_rna.txt\n",
      "cp data/subsets/Subset3/names_rna.txt tmp_static/Subset3/names_rna.txt\n",
      "mkdir -p tmp_static/Subset4/\n",
      "cp data/subsets/Subset4/names_rna.txt tmp_static/Subset4/names_rna.txt\n",
      "mkdir -p tmp_static/Subset5/\n",
      "cp data/subsets/Subset5/names_rna.txt tmp_static/Subset5/names_rna.txt\n",
      "mkdir -p tmp_static/Subset6/\n",
      "cp data/subsets/Subset6/names_rna.txt tmp_static/Subset6/names_rna.txt\n",
      "mkdir -p tmp_static/Subset7/\n",
      "cp data/subsets/Subset7/names_rna.txt tmp_static/Subset7/names_rna.txt\n",
      "mkdir -p tmp_static/Subset8/\n",
      "cp data/subsets/Subset8/names_rna.txt tmp_static/Subset8/names_rna.txt\n",
      "mkdir -p tmp_static/Subset9/\n",
      "cp data/subsets/Subset9/names_rna.txt tmp_static/Subset9/names_rna.txt\n",
      "cp tmp_static/Subset1/names_rna.txt tmp_static/Subset1/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset1/names_rna.txt tmp_static/Subset1/expression0.tsv.gz\n",
      "cp tmp_static/Subset10/names_rna.txt tmp_static/Subset10/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset10/names_rna.txt tmp_static/Subset10/expression0.tsv.gz\n",
      "cp tmp_static/Subset11/names_rna.txt tmp_static/Subset11/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset11/names_rna.txt tmp_static/Subset11/expression0.tsv.gz\n",
      "cp tmp_static/Subset12/names_rna.txt tmp_static/Subset12/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset12/names_rna.txt tmp_static/Subset12/expression0.tsv.gz\n",
      "cp tmp_static/Subset13/names_rna.txt tmp_static/Subset13/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset13/names_rna.txt tmp_static/Subset13/expression0.tsv.gz\n",
      "cp tmp_static/Subset14/names_rna.txt tmp_static/Subset14/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset14/names_rna.txt tmp_static/Subset14/expression0.tsv.gz\n",
      "cp tmp_static/Subset15/names_rna.txt tmp_static/Subset15/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset15/names_rna.txt tmp_static/Subset15/expression0.tsv.gz\n",
      "cp tmp_static/Subset16/names_rna.txt tmp_static/Subset16/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset16/names_rna.txt tmp_static/Subset16/expression0.tsv.gz\n",
      "cp tmp_static/Subset17/names_rna.txt tmp_static/Subset17/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset17/names_rna.txt tmp_static/Subset17/expression0.tsv.gz\n",
      "cp tmp_static/Subset2/names_rna.txt tmp_static/Subset2/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset2/names_rna.txt tmp_static/Subset2/expression0.tsv.gz\n",
      "cp tmp_static/Subset3/names_rna.txt tmp_static/Subset3/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset3/names_rna.txt tmp_static/Subset3/expression0.tsv.gz\n",
      "cp tmp_static/Subset4/names_rna.txt tmp_static/Subset4/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset4/names_rna.txt tmp_static/Subset4/expression0.tsv.gz\n",
      "cp tmp_static/Subset5/names_rna.txt tmp_static/Subset5/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset5/names_rna.txt tmp_static/Subset5/expression0.tsv.gz\n",
      "cp tmp_static/Subset6/names_rna.txt tmp_static/Subset6/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset6/names_rna.txt tmp_static/Subset6/expression0.tsv.gz\n",
      "cp tmp_static/Subset7/names_rna.txt tmp_static/Subset7/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset7/names_rna.txt tmp_static/Subset7/expression0.tsv.gz\n",
      "cp tmp_static/Subset8/names_rna.txt tmp_static/Subset8/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset8/names_rna.txt tmp_static/Subset8/expression0.tsv.gz\n",
      "cp tmp_static/Subset9/names_rna.txt tmp_static/Subset9/names_atac0.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_rna  data/expression.tsv.gz tmp_static/Subset9/names_rna.txt tmp_static/Subset9/expression0.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset17/expression0.tsv.gz tmp_static/Subset17/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset16/expression0.tsv.gz tmp_static/Subset16/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset15/expression0.tsv.gz tmp_static/Subset15/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset17/expression.tsv.gz tmp_static/Subset17/names_atac0.txt tmp_static/Subset17/names_atac.txt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset13/expression0.tsv.gz tmp_static/Subset13/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset10/expression0.tsv.gz tmp_static/Subset10/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset14/expression0.tsv.gz tmp_static/Subset14/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset17/names_atac.txt data/bams tmp_static/Subset17/reads.bam tmp_static/Subset17/reads.bai tmp_static/Subset17/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset9/expression0.tsv.gz tmp_static/Subset9/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset12/expression0.tsv.gz tmp_static/Subset12/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset11/expression0.tsv.gz tmp_static/Subset11/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset6/expression0.tsv.gz tmp_static/Subset6/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset16/expression.tsv.gz tmp_static/Subset16/names_atac0.txt tmp_static/Subset16/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset4/expression0.tsv.gz tmp_static/Subset4/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset8/expression0.tsv.gz tmp_static/Subset8/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset7/expression0.tsv.gz tmp_static/Subset7/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset15/expression.tsv.gz tmp_static/Subset15/names_atac0.txt tmp_static/Subset15/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset16/names_atac.txt data/bams tmp_static/Subset16/reads.bam tmp_static/Subset16/reads.bai tmp_static/Subset16/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset3/expression0.tsv.gz tmp_static/Subset3/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset13/expression.tsv.gz tmp_static/Subset13/names_atac0.txt tmp_static/Subset13/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset5/expression0.tsv.gz tmp_static/Subset5/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset15/names_atac.txt data/bams tmp_static/Subset15/reads.bam tmp_static/Subset15/reads.bai tmp_static/Subset15/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset10/expression.tsv.gz tmp_static/Subset10/names_atac0.txt tmp_static/Subset10/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset13/names_atac.txt data/bams tmp_static/Subset13/reads.bam tmp_static/Subset13/reads.bai tmp_static/Subset13/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset14/expression.tsv.gz tmp_static/Subset14/names_atac0.txt tmp_static/Subset14/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset2/expression0.tsv.gz tmp_static/Subset2/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset10/names_atac.txt data/bams tmp_static/Subset10/reads.bam tmp_static/Subset10/reads.bai tmp_static/Subset10/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/names_atac0.txt tmp_static/Subset9/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/names_atac0.txt tmp_static/Subset7/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset12/expression.tsv.gz tmp_static/Subset12/names_atac0.txt tmp_static/Subset12/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset11/expression.tsv.gz tmp_static/Subset11/names_atac0.txt tmp_static/Subset11/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/names_atac0.txt tmp_static/Subset8/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/names_atac0.txt tmp_static/Subset6/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset14/names_atac.txt data/bams tmp_static/Subset14/reads.bam tmp_static/Subset14/reads.bai tmp_static/Subset14/peaks.bed hs\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/names_atac0.txt tmp_static/Subset4/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset9/names_atac.txt data/bams tmp_static/Subset9/reads.bam tmp_static/Subset9/reads.bai tmp_static/Subset9/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset7/names_atac.txt data/bams tmp_static/Subset7/reads.bam tmp_static/Subset7/reads.bai tmp_static/Subset7/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset12/names_atac.txt data/bams tmp_static/Subset12/reads.bam tmp_static/Subset12/reads.bai tmp_static/Subset12/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset11/names_atac.txt data/bams tmp_static/Subset11/reads.bam tmp_static/Subset11/reads.bai tmp_static/Subset11/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/names_atac0.txt tmp_static/Subset3/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset6/names_atac.txt data/bams tmp_static/Subset6/reads.bam tmp_static/Subset6/reads.bai tmp_static/Subset6/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset8/names_atac.txt data/bams tmp_static/Subset8/reads.bam tmp_static/Subset8/reads.bai tmp_static/Subset8/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset4/names_atac.txt data/bams tmp_static/Subset4/reads.bam tmp_static/Subset4/reads.bai tmp_static/Subset4/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/names_atac0.txt tmp_static/Subset5/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc qc_reads  tmp_static/Subset1/expression0.tsv.gz tmp_static/Subset1/expression.tsv.gz 50 10 0 200 100 0\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset3/names_atac.txt data/bams tmp_static/Subset3/reads.bam tmp_static/Subset3/reads.bai tmp_static/Subset3/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/names_atac0.txt tmp_static/Subset2/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset5/names_atac.txt data/bams tmp_static/Subset5/reads.bam tmp_static/Subset5/reads.bai tmp_static/Subset5/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  preproc selects_atac  tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/names_atac0.txt tmp_static/Subset1/names_atac.txt\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset2/names_atac.txt data/bams tmp_static/Subset2/reads.bam tmp_static/Subset2/reads.bai tmp_static/Subset2/peaks.bed hs\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin macs2 --nth 4 tmp_static/Subset1/names_atac.txt data/bams tmp_static/Subset1/reads.bam tmp_static/Subset1/reads.bai tmp_static/Subset1/peaks.bed hs\n",
      "[bam_sort_core] merging from 1 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:22:06: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset17/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset17/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:06: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:06: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:22:06: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:06: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:06: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:09:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:12:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:15:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:18:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: 4924143 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #1 tag size is determined as 50 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #1 tag size = 50.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #1  total tags in treatment: 4924143 \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:21: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:22:21: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:22:26: access pq hash for 10979677 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:22:26: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:23:06: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:23:06: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:23:07: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:23:07: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset17/reads.bam tmp_static/Subset17/reads.bai tmp_static/Subset17/peaks.bed tmp_static/Subset17/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:25:22: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset16/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset16/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:22: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:22: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:25:22: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:22: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:22: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:25:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:28:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:31:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:34:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:37:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:40:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:43:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:46:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: 8749075 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #1  total tags in treatment: 8749075 \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:49: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:25:49: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:25:59: access pq hash for 16389906 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:25:59: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:26:25: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:26:25: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:26:25: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:26:25: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset16/reads.bam tmp_static/Subset16/reads.bai tmp_static/Subset16/peaks.bed tmp_static/Subset16/footprints.bed\n",
      "[bam_sort_core] merging from 2 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:27:23: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset13/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset13/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:23: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:23: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:27:23: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:23: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:23: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:26:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:29:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:32:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:35:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:38:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:41:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:44:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:47:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:50:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:53:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: 10748254 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #1  total tags in treatment: 10748254 \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:27:56: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:27:56: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:28:10: access pq hash for 19005632 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:10: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:37: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:38: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:38: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:38: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset13/reads.bam tmp_static/Subset13/reads.bai tmp_static/Subset13/peaks.bed tmp_static/Subset13/footprints.bed\n",
      "[bam_sort_core] merging from 3 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:28:06: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset15/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset15/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:06: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:06: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:28:06: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:06: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:06: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:09:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:12:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:15:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:19:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:22:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:25:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:28:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:31:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:34:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:38:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:41:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: 11955586 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #1 tag size is determined as 49 bps \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #1  total tags in treatment: 11955586 \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:44: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:28:44: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:28:59: access pq hash for 21232508 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:28:59: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:29:25: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:29:25: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:29:25: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:29:25: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset15/reads.bam tmp_static/Subset15/reads.bai tmp_static/Subset15/peaks.bed tmp_static/Subset15/footprints.bed\n",
      "[bam_sort_core] merging from 3 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:30:01: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset10/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset10/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:01: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:01: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:30:01: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:01: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:01: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:04:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:07:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:10:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:13:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:16:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:19:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:22:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:25:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:28:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:31:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:34:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:37:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:39: 12731137 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #1  total tags in treatment: 12731137 \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:40: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:30:40: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:30:55: access pq hash for 21592680 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:30:55: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:20: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:20: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:21: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:21: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset10/reads.bam tmp_static/Subset10/reads.bai tmp_static/Subset10/peaks.bed tmp_static/Subset10/footprints.bed\n",
      "[bam_sort_core] merging from 3 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:31:44: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset11/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset11/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:44: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:44: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:31:44: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:44: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:44: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:47:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:50:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:53:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:56:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:31:59:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:02:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:05:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:08:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:11:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:14:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:17:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:20:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:23:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: 13961329 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #1  total tags in treatment: 13961329 \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:26: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:32:26: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:32:44: access pq hash for 25300891 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:32:44: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:33:17: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:33:17: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:33:18: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:33:18: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset11/reads.bam tmp_static/Subset11/reads.bai tmp_static/Subset11/peaks.bed tmp_static/Subset11/footprints.bed\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:34:04: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset14/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset14/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:04: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:04: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:34:05: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:05: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:05: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:08:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:11:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:14:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:17:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:20:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:23:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:26:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:29:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:32:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:35:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:38:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:41:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:44:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:47:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:50:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:53:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:56:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:34:59:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: 18409803 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #1  total tags in treatment: 18409803 \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:00: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:35:00: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:35:24: access pq hash for 33583381 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:35:24: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:07: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:08: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:08: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:08: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset14/reads.bam tmp_static/Subset14/reads.bai tmp_static/Subset14/peaks.bed tmp_static/Subset14/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:36:04: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset7/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset7/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:04: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:04: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:36:04: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:05: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:05: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:08:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:11:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:14:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:17:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:20:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:23:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:26:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:29:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:32:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:35:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:39:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:42:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:45:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:48:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:54:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:57:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:58: 17496080 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #1 tag size is determined as 45 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #1 tag size = 45.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #1  total tags in treatment: 17496080 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:59: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:36:59: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:37:24: access pq hash for 28086559 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:24: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:54: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:55: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:55: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:55: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset7/reads.bam tmp_static/Subset7/reads.bai tmp_static/Subset7/peaks.bed tmp_static/Subset7/footprints.bed\n",
      "[bam_sort_core] merging from 4 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset8/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset8/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:36:51: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:51: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:55:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:36:58:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:01:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:04:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:07:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:10:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:14:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:17:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:20:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:23:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:26:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:29:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:32:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:35:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:39:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:42:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:45:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:48:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:51:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: 19570117 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #1  total tags in treatment: 19570117 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:53: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:37:53: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:38:22: access pq hash for 31653268 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:22: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:02: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:02: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:02: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:02: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset8/reads.bam tmp_static/Subset8/reads.bai tmp_static/Subset8/peaks.bed tmp_static/Subset8/footprints.bed\n",
      "[bam_sort_core] merging from 5 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:37:31: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset9/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset9/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:31: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:31: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:37:31: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:31: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:31: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:34:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:37:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:40:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:43:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:46:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:49:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:52:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:55:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:37:58:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:01:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:04:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:08:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:11:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:14:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:17:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:20:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:23:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:26:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:30:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:33:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:36:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:39:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: 22269535 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #1  total tags in treatment: 22269535 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:38:41: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:39:17: access pq hash for 36071646 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:17: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:13: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:13: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:14: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:14: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset9/reads.bam tmp_static/Subset9/reads.bai tmp_static/Subset9/peaks.bed tmp_static/Subset9/footprints.bed\n",
      "[bam_sort_core] merging from 5 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:38:12: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset6/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset6/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:12: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:12: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:38:12: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:12: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:12: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:16:  1000000 \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 02 Aug 2024 12:38:20:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:23:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:26:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:29:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:33:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:36:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:39:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:43:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:46:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:49:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:53:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:57:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:01:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:04:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:08:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:12:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:15:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:19:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:27:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:31:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: 22508003 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #1  total tags in treatment: 22508003 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:39:33: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:40:15: access pq hash for 34514871 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:15: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:06: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:06: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:07: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:07: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset6/reads.bam tmp_static/Subset6/reads.bai tmp_static/Subset6/peaks.bed tmp_static/Subset6/footprints.bed\n",
      "[bam_sort_core] merging from 5 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset4/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset4/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:38:38: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:44:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:47:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:52:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:56:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:59:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:03:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:06:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:10:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:13:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:16:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:21:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:24:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:28:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:32:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:36:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:41:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:45:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:50:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:55:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:58:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:02:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:06:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:08: 23437221 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #1  total tags in treatment: 23437221 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:09: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:40:09: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:40:58: access pq hash for 35292945 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:58: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:53: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:54: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:54: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:54: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset4/reads.bam tmp_static/Subset4/reads.bai tmp_static/Subset4/peaks.bed tmp_static/Subset4/footprints.bed\n",
      "[bam_sort_core] merging from 6 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset12/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset12/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:38:38: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:38: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:41:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:45:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:48:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:52:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:38:56:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:00:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:03:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:07:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:10:  9000000 \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 02 Aug 2024 12:39:14:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:18:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:22:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:25:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:29:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:33:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:37:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:45:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:49:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:54:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:58:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:02:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:08:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:12:  24000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:16:  25000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:21:  26000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: 26390825 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #1  total tags in treatment: 26390825 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:40:22: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:41:15: access pq hash for 45006259 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:15: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:29: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:30: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:30: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:31: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset12/reads.bam tmp_static/Subset12/reads.bai tmp_static/Subset12/peaks.bed tmp_static/Subset12/footprints.bed\n",
      "[bam_sort_core] merging from 6 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset5/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset5/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:39:23: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:23: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:27:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:31:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:34:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:38:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:42:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:46:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:50:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:54:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:58:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:02:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:07:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:11:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:14:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:19:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:22:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:27:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:31:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:35:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:39:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:43:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:47:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:51:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:56:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:59:  24000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:03:  25000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:09:  26000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: 26722135 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #1 tag size is determined as 47 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #1 tag size = 47.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #1  total tags in treatment: 26722135 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:12: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:41:12: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:42:14: access pq hash for 44300252 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:14: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:47: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: Done! \n",
      "\n",
      "[bam_sort_core] merging from 7 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset3/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset3/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:39:40: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:40: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:44:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:49:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:52:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:39:56:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:01:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:05:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:10:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:14:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:18:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:23:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:26:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:30:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:35:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:40:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:44:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:49:  16000000 \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 02 Aug 2024 12:40:53:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:40:57:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:01:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:06:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:11:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:16:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:21:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:26:  24000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:31:  25000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:35:  26000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:38:  27000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:42:  28000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: 28263367 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #1  total tags in treatment: 28263367 \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:41:43: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:41:43: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:42:43: access pq hash for 40959046 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:42:43: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:48: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:43:49: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset5/reads.bam tmp_static/Subset5/reads.bai tmp_static/Subset5/peaks.bed tmp_static/Subset5/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset3/reads.bam tmp_static/Subset3/reads.bai tmp_static/Subset3/peaks.bed tmp_static/Subset3/footprints.bed\n",
      "[bam_sort_core] merging from 12 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 12:48:22: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset2/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset2/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:22: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:22: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:48:22: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:22: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:22: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:29:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:34:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:40:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:45:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:51:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:48:56:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:01:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:06:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:13:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:20:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:25:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:31:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:38:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:44:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:49:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:54:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:49:59:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:03:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:08:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:13:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:17:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:23:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:28:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:33:  24000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:39:  25000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:45:  26000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:51:  27000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:50:56:  28000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:00:  29000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:05:  30000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:10:  31000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:13:  32000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:18:  33000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:22:  34000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:27:  35000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:33:  36000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:39:  37000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:44:  38000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:48:  39000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:51:53:  40000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:01:  41000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:06:  42000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:12:  43000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:17:  44000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:22:  45000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:28:  46000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:33:  47000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:37:  48000000 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: 48688034 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #1  total tags in treatment: 48688034 \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #3 Call peaks... \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 12:52:41: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:52:41: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 12:54:30: access pq hash for 65086932 times \n",
      "INFO  @ Fri, 02 Aug 2024 12:54:30: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 12:55:36: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 12:55:37: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 12:55:37: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 12:55:37: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset2/reads.bam tmp_static/Subset2/reads.bai tmp_static/Subset2/peaks.bed tmp_static/Subset2/footprints.bed\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset17/footprints.bed data/motifs.motif data/genome tmp_static/Subset17/expression.tsv.gz tmp_static/Subset17/motifs.bed tmp_static/Subset17/wellington.tsv.gz tmp_static/Subset17/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6747\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6747\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 21 (avg size of targets)\n",
      "\tBackground files for 21 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 705 sequences from chr1\n",
      "\tExtracting 228 sequences from chr10\n",
      "\tExtracting 354 sequences from chr11\n",
      "\tExtracting 304 sequences from chr12\n",
      "\tExtracting 96 sequences from chr13\n",
      "\tExtracting 223 sequences from chr14\n",
      "\tExtracting 155 sequences from chr15\n",
      "\tExtracting 354 sequences from chr16\n",
      "\tExtracting 485 sequences from chr17\n",
      "\tExtracting 76 sequences from chr18\n",
      "\tExtracting 812 sequences from chr19\n",
      "\tExtracting 381 sequences from chr2\n",
      "\tExtracting 163 sequences from chr20\n",
      "\tExtracting 155 sequences from chr21\n",
      "\tExtracting 159 sequences from chr22\n",
      "\tExtracting 369 sequences from chr3\n",
      "\tExtracting 216 sequences from chr4\n",
      "\tExtracting 229 sequences from chr5\n",
      "\tExtracting 299 sequences from chr6\n",
      "\tExtracting 337 sequences from chr7\n",
      "\tExtracting 191 sequences from chr8\n",
      "\tExtracting 295 sequences from chr9\n",
      "\tExtracting 91 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6677 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6747\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6747\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 660 sequences from chr1\n",
      "\tExtracting 237 sequences from chr10\n",
      "\tExtracting 389 sequences from chr11\n",
      "\tExtracting 338 sequences from chr12\n",
      "\tExtracting 104 sequences from chr13\n",
      "\tExtracting 226 sequences from chr14\n",
      "\tExtracting 169 sequences from chr15\n",
      "\tExtracting 340 sequences from chr16\n",
      "\tExtracting 489 sequences from chr17\n",
      "\tExtracting 83 sequences from chr18\n",
      "\tExtracting 723 sequences from chr19\n",
      "\tExtracting 446 sequences from chr2\n",
      "\tExtracting 141 sequences from chr20\n",
      "\tExtracting 116 sequences from chr21\n",
      "\tExtracting 164 sequences from chr22\n",
      "\tExtracting 306 sequences from chr3\n",
      "\tExtracting 205 sequences from chr4\n",
      "\tExtracting 289 sequences from chr5\n",
      "\tExtracting 328 sequences from chr6\n",
      "\tExtracting 328 sequences from chr7\n",
      "\tExtracting 212 sequences from chr8\n",
      "\tExtracting 290 sequences from chr9\n",
      "\tExtracting 122 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6707 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6747\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6747\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 684 sequences from chr1\n",
      "\tExtracting 239 sequences from chr10\n",
      "\tExtracting 350 sequences from chr11\n",
      "\tExtracting 349 sequences from chr12\n",
      "\tExtracting 119 sequences from chr13\n",
      "\tExtracting 248 sequences from chr14\n",
      "\tExtracting 188 sequences from chr15\n",
      "\tExtracting 292 sequences from chr16\n",
      "\tExtracting 428 sequences from chr17\n",
      "\tExtracting 82 sequences from chr18\n",
      "\tExtracting 614 sequences from chr19\n",
      "\tExtracting 446 sequences from chr2\n",
      "\tExtracting 165 sequences from chr20\n",
      "\tExtracting 111 sequences from chr21\n",
      "\tExtracting 191 sequences from chr22\n",
      "\tExtracting 397 sequences from chr3\n",
      "\tExtracting 233 sequences from chr4\n",
      "\tExtracting 256 sequences from chr5\n",
      "\tExtracting 357 sequences from chr6\n",
      "\tExtracting 306 sequences from chr7\n",
      "\tExtracting 223 sequences from chr8\n",
      "\tExtracting 293 sequences from chr9\n",
      "\tExtracting 142 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6714 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 6745\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 6745\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 626 sequences from chr1\n",
      "\tExtracting 259 sequences from chr10\n",
      "\tExtracting 387 sequences from chr11\n",
      "\tExtracting 336 sequences from chr12\n",
      "\tExtracting 123 sequences from chr13\n",
      "\tExtracting 212 sequences from chr14\n",
      "\tExtracting 197 sequences from chr15\n",
      "\tExtracting 330 sequences from chr16\n",
      "\tExtracting 463 sequences from chr17\n",
      "\tExtracting 101 sequences from chr18\n",
      "\tExtracting 642 sequences from chr19\n",
      "\tExtracting 440 sequences from chr2\n",
      "\tExtracting 187 sequences from chr20\n",
      "\tExtracting 118 sequences from chr21\n",
      "\tExtracting 180 sequences from chr22\n",
      "\tExtracting 319 sequences from chr3\n",
      "\tExtracting 211 sequences from chr4\n",
      "\tExtracting 288 sequences from chr5\n",
      "\tExtracting 344 sequences from chr6\n",
      "\tExtracting 301 sequences from chr7\n",
      "\tExtracting 248 sequences from chr8\n",
      "\tExtracting 248 sequences from chr9\n",
      "\tExtracting 150 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t6710 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset17/wellington.tsv.gz tmp_static/Subset17/homer.tsv.gz tmp_static/Subset17/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset17/expression.tsv.gz tmp_static/Subset17/wellington.tsv.gz data/gene.bed tmp_static/Subset17/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset17/binding.tsv.gz tmp_static/Subset17/tssdist.tsv.gz tmp_static/Subset17/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset17/linking.tsv.gz tmp_static/Subset17/binlinking.tsv.gz 20\n",
      "[bam_sort_core] merging from 19 files and 4 in-memory blocks...\n",
      "INFO  @ Fri, 02 Aug 2024 13:00:01: \n",
      "# Command line: callpeak -t /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset1/reads.bam -n 04 -g hs --nomodel --shift -75 --extsize 150 --keep-dup all --verbose 4 --call-summits -q 0.05\n",
      "# ARGUMENTS LIST:\n",
      "# name = 04\n",
      "# format = AUTO\n",
      "# ChIP-seq file = ['/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/tmp_static/Subset1/reads.bam']\n",
      "# control file = None\n",
      "# effective genome size = 2.70e+09\n",
      "# band width = 300\n",
      "# model fold = [5, 50]\n",
      "# qvalue cutoff = 5.00e-02\n",
      "# The maximum gap between significant sites is assigned as the read length/tag size.\n",
      "# The minimum length of peaks is assigned as the predicted fragment length \"d\".\n",
      "# Larger dataset will be scaled towards smaller dataset.\n",
      "# Range for calculating regional lambda is: 10000 bps\n",
      "# Broad region calling is off\n",
      "# Paired-End mode is off\n",
      "# Searching for subpeak summits is on\n",
      " \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:01: #1 read tag files... \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:01: #1 read treatment tags... \n",
      "DEBUG @ Fri, 02 Aug 2024 13:00:01: Testing format BAM \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:01: Detected format is: BAM \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:01: * Input file is gzipped. \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:05:  1000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:09:  2000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:14:  3000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:17:  4000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:22:  5000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:26:  6000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:30:  7000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:34:  8000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:38:  9000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:42:  10000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:47:  11000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:51:  12000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:00:56:  13000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:00:  14000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:04:  15000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:07:  16000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:12:  17000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:17:  18000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:22:  19000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:27:  20000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:32:  21000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:37:  22000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:42:  23000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:46:  24000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:50:  25000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:54:  26000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:01:59:  27000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:04:  28000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:07:  29000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:13:  30000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:18:  31000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:23:  32000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:27:  33000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:33:  34000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:37:  35000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:43:  36000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:49:  37000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:53:  38000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:02:58:  39000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:04:  40000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:10:  41000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:16:  42000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:19:  43000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:24:  44000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:29:  45000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:34:  46000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:39:  47000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:46:  48000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:52:  49000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:03:56:  50000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:01:  51000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:06:  52000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:11:  53000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:18:  54000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:23:  55000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:31:  56000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:37:  57000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:43:  58000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:50:  59000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:55:  60000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:04:59:  61000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:04:  62000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:09:  63000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:13:  64000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:18:  65000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:23:  66000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:29:  67000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:35:  68000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:39:  69000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:44:  70000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:50:  71000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:05:55:  72000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:00:  73000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:06:  74000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:10:  75000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:15:  76000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:21:  77000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:25:  78000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:32:  79000000 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:35: 79431551 reads have been read. \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #1 tag size is determined as 49 bps \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #1 tag size = 49.0 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #1  total tags in treatment: 79431551 \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #1 finished! \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #2 Build Peak Model... \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #2 Skipped... \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #2 Use 150 as fragment length \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #2 Sequencing ends will be shifted towards 5' by 75 bp(s) \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #3 Call peaks... \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #3 Going to call summits inside each peak ... \n",
      "INFO  @ Fri, 02 Aug 2024 13:06:36: #3 Pre-compute pvalue-qvalue table... \n",
      "DEBUG @ Fri, 02 Aug 2024 13:06:36: Start to calculate pvalue stat... \n",
      "DEBUG @ Fri, 02 Aug 2024 13:10:27: access pq hash for 115417840 times \n",
      "INFO  @ Fri, 02 Aug 2024 13:10:27: #3 Call peaks for each chromosome... \n",
      "INFO  @ Fri, 02 Aug 2024 13:13:27: #4 Write output xls file... 04_peaks.xls \n",
      "INFO  @ Fri, 02 Aug 2024 13:13:28: #4 Write peak in narrowPeak format file... 04_peaks.narrowPeak \n",
      "INFO  @ Fri, 02 Aug 2024 13:13:29: #4 Write summits bed file... 04_summits.bed \n",
      "INFO  @ Fri, 02 Aug 2024 13:13:29: Done! \n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin wellington --nth 4 tmp_static/Subset1/reads.bam tmp_static/Subset1/reads.bai tmp_static/Subset1/peaks.bed tmp_static/Subset1/footprints.bed\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset17/expression.tsv.gz tmp_static/Subset17/binlinking.tsv.gz tmp_static/Subset17/net_weight.tsv.gz tmp_static/Subset17/net_meanvar.tsv.gz tmp_static/Subset17/net_covfactor.tsv.gz tmp_static/Subset17/net_loss.tsv.gz tmp_static/Subset17/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset16/footprints.bed data/motifs.motif data/genome tmp_static/Subset16/expression.tsv.gz tmp_static/Subset16/motifs.bed tmp_static/Subset16/wellington.tsv.gz tmp_static/Subset16/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23794\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23794\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2450 sequences from chr1\n",
      "\tExtracting 949 sequences from chr10\n",
      "\tExtracting 1233 sequences from chr11\n",
      "\tExtracting 1196 sequences from chr12\n",
      "\tExtracting 437 sequences from chr13\n",
      "\tExtracting 823 sequences from chr14\n",
      "\tExtracting 744 sequences from chr15\n",
      "\tExtracting 1064 sequences from chr16\n",
      "\tExtracting 1685 sequences from chr17\n",
      "\tExtracting 308 sequences from chr18\n",
      "\tExtracting 1804 sequences from chr19\n",
      "\tExtracting 1618 sequences from chr2\n",
      "\tExtracting 621 sequences from chr20\n",
      "\tExtracting 371 sequences from chr21\n",
      "\tExtracting 601 sequences from chr22\n",
      "\tExtracting 1261 sequences from chr3\n",
      "\tExtracting 756 sequences from chr4\n",
      "\tExtracting 1020 sequences from chr5\n",
      "\tExtracting 1330 sequences from chr6\n",
      "\tExtracting 1114 sequences from chr7\n",
      "\tExtracting 869 sequences from chr8\n",
      "\tExtracting 940 sequences from chr9\n",
      "\tExtracting 567 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23762 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23791\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23791\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2506 sequences from chr1\n",
      "\tExtracting 1034 sequences from chr10\n",
      "\tExtracting 1173 sequences from chr11\n",
      "\tExtracting 1227 sequences from chr12\n",
      "\tExtracting 466 sequences from chr13\n",
      "\tExtracting 846 sequences from chr14\n",
      "\tExtracting 749 sequences from chr15\n",
      "\tExtracting 1063 sequences from chr16\n",
      "\tExtracting 1575 sequences from chr17\n",
      "\tExtracting 317 sequences from chr18\n",
      "\tExtracting 1682 sequences from chr19\n",
      "\tExtracting 1610 sequences from chr2\n",
      "\tExtracting 641 sequences from chr20\n",
      "\tExtracting 318 sequences from chr21\n",
      "\tExtracting 659 sequences from chr22\n",
      "\tExtracting 1272 sequences from chr3\n",
      "\tExtracting 799 sequences from chr4\n",
      "\tExtracting 1059 sequences from chr5\n",
      "\tExtracting 1299 sequences from chr6\n",
      "\tExtracting 1042 sequences from chr7\n",
      "\tExtracting 831 sequences from chr8\n",
      "\tExtracting 923 sequences from chr9\n",
      "\tExtracting 661 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23752 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23794\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23794\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 2520 sequences from chr1\n",
      "\tExtracting 920 sequences from chr10\n",
      "\tExtracting 1314 sequences from chr11\n",
      "\tExtracting 1209 sequences from chr12\n",
      "\tExtracting 438 sequences from chr13\n",
      "\tExtracting 842 sequences from chr14\n",
      "\tExtracting 714 sequences from chr15\n",
      "\tExtracting 1152 sequences from chr16\n",
      "\tExtracting 1572 sequences from chr17\n",
      "\tExtracting 339 sequences from chr18\n",
      "\tExtracting 1983 sequences from chr19\n",
      "\tExtracting 1517 sequences from chr2\n",
      "\tExtracting 598 sequences from chr20\n",
      "\tExtracting 380 sequences from chr21\n",
      "\tExtracting 618 sequences from chr22\n",
      "\tExtracting 1217 sequences from chr3\n",
      "\tExtracting 786 sequences from chr4\n",
      "\tExtracting 1030 sequences from chr5\n",
      "\tExtracting 1330 sequences from chr6\n",
      "\tExtracting 1074 sequences from chr7\n",
      "\tExtracting 799 sequences from chr8\n",
      "\tExtracting 899 sequences from chr9\n",
      "\tExtracting 477 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23730 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 23794\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 23794\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2493 sequences from chr1\n",
      "\tExtracting 855 sequences from chr10\n",
      "\tExtracting 1268 sequences from chr11\n",
      "\tExtracting 1245 sequences from chr12\n",
      "\tExtracting 406 sequences from chr13\n",
      "\tExtracting 829 sequences from chr14\n",
      "\tExtracting 652 sequences from chr15\n",
      "\tExtracting 1197 sequences from chr16\n",
      "\tExtracting 1597 sequences from chr17\n",
      "\tExtracting 271 sequences from chr18\n",
      "\tExtracting 2317 sequences from chr19\n",
      "\tExtracting 1500 sequences from chr2\n",
      "\tExtracting 618 sequences from chr20\n",
      "\tExtracting 428 sequences from chr21\n",
      "\tExtracting 572 sequences from chr22\n",
      "\tExtracting 1208 sequences from chr3\n",
      "\tExtracting 736 sequences from chr4\n",
      "\tExtracting 918 sequences from chr5\n",
      "\tExtracting 1219 sequences from chr6\n",
      "\tExtracting 1201 sequences from chr7\n",
      "\tExtracting 767 sequences from chr8\n",
      "\tExtracting 967 sequences from chr9\n",
      "\tExtracting 424 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t23688 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset16/wellington.tsv.gz tmp_static/Subset16/homer.tsv.gz tmp_static/Subset16/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset16/expression.tsv.gz tmp_static/Subset16/wellington.tsv.gz data/gene.bed tmp_static/Subset16/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset16/binding.tsv.gz tmp_static/Subset16/tssdist.tsv.gz tmp_static/Subset16/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset16/linking.tsv.gz tmp_static/Subset16/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset16/expression.tsv.gz tmp_static/Subset16/binlinking.tsv.gz tmp_static/Subset16/net_weight.tsv.gz tmp_static/Subset16/net_meanvar.tsv.gz tmp_static/Subset16/net_covfactor.tsv.gz tmp_static/Subset16/net_loss.tsv.gz tmp_static/Subset16/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset13/footprints.bed data/motifs.motif data/genome tmp_static/Subset13/expression.tsv.gz tmp_static/Subset13/motifs.bed tmp_static/Subset13/wellington.tsv.gz tmp_static/Subset13/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset15/footprints.bed data/motifs.motif data/genome tmp_static/Subset15/expression.tsv.gz tmp_static/Subset15/motifs.bed tmp_static/Subset15/wellington.tsv.gz tmp_static/Subset15/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2593 sequences from chr1\n",
      "\tExtracting 1027 sequences from chr10\n",
      "\tExtracting 1330 sequences from chr11\n",
      "\tExtracting 1266 sequences from chr12\n",
      "\tExtracting 433 sequences from chr13\n",
      "\tExtracting 867 sequences from chr14\n",
      "\tExtracting 794 sequences from chr15\n",
      "\tExtracting 1102 sequences from chr16\n",
      "\tExtracting 1663 sequences from chr17\n",
      "\tExtracting 361 sequences from chr18\n",
      "\tExtracting 1836 sequences from chr19\n",
      "\tExtracting 1665 sequences from chr2\n",
      "\tExtracting 630 sequences from chr20\n",
      "\tExtracting 382 sequences from chr21\n",
      "\tExtracting 662 sequences from chr22\n",
      "\tExtracting 1327 sequences from chr3\n",
      "\tExtracting 859 sequences from chr4\n",
      "\tExtracting 1040 sequences from chr5\n",
      "\tExtracting 1472 sequences from chr6\n",
      "\tExtracting 1182 sequences from chr7\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 890 sequences from chr8\n",
      "\tExtracting 988 sequences from chr9\n",
      "\tExtracting 597 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24966 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2563 sequences from chr1\n",
      "\tExtracting 956 sequences from chr10\n",
      "\tExtracting 1315 sequences from chr11\n",
      "\tExtracting 1275 sequences from chr12\n",
      "\tExtracting 435 sequences from chr13\n",
      "\tExtracting 859 sequences from chr14\n",
      "\tExtracting 756 sequences from chr15\n",
      "\tExtracting 1182 sequences from chr16\n",
      "\tExtracting 1637 sequences from chr17\n",
      "\tExtracting 366 sequences from chr18\n",
      "\tExtracting 2082 sequences from chr19\n",
      "\tExtracting 1650 sequences from chr2\n",
      "\tExtracting 636 sequences from chr20\n",
      "\tExtracting 410 sequences from chr21\n",
      "\tExtracting 640 sequences from chr22\n",
      "\tExtracting 1302 sequences from chr3\n",
      "\tExtracting 786 sequences from chr4\n",
      "\tExtracting 1085 sequences from chr5\n",
      "\tExtracting 1476 sequences from chr6\n",
      "\tExtracting 1122 sequences from chr7\n",
      "\tExtracting 883 sequences from chr8\n",
      "\tExtracting 987 sequences from chr9\n",
      "\tExtracting 543 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24946 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2570 sequences from chr1\n",
      "\tExtracting 1030 sequences from chr10\n",
      "\tExtracting 1304 sequences from chr11\n",
      "\tExtracting 1316 sequences from chr12\n",
      "\tExtracting 428 sequences from chr13\n",
      "\tExtracting 878 sequences from chr14\n",
      "\tExtracting 839 sequences from chr15\n",
      "\tExtracting 1105 sequences from chr16\n",
      "\tExtracting 1612 sequences from chr17\n",
      "\tExtracting 362 sequences from chr18\n",
      "\tExtracting 1752 sequences from chr19\n",
      "\tExtracting 1654 sequences from chr2\n",
      "\tExtracting 638 sequences from chr20\n",
      "\tExtracting 349 sequences from chr21\n",
      "\tExtracting 617 sequences from chr22\n",
      "\tExtracting 1427 sequences from chr3\n",
      "\tExtracting 824 sequences from chr4\n",
      "\tExtracting 1087 sequences from chr5\n",
      "\tExtracting 1556 sequences from chr6\n",
      "\tExtracting 1156 sequences from chr7\n",
      "\tExtracting 890 sequences from chr8\n",
      "\tExtracting 1007 sequences from chr9\n",
      "\tExtracting 565 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24966 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2532 sequences from chr1\n",
      "\tExtracting 860 sequences from chr10\n",
      "\tExtracting 1341 sequences from chr11\n",
      "\tExtracting 1258 sequences from chr12\n",
      "\tExtracting 385 sequences from chr13\n",
      "\tExtracting 869 sequences from chr14\n",
      "\tExtracting 677 sequences from chr15\n",
      "\tExtracting 1215 sequences from chr16\n",
      "\tExtracting 1743 sequences from chr17\n",
      "\tExtracting 316 sequences from chr18\n",
      "\tExtracting 2464 sequences from chr19\n",
      "\tExtracting 1577 sequences from chr2\n",
      "\tExtracting 641 sequences from chr20\n",
      "\tExtracting 466 sequences from chr21\n",
      "\tExtracting 652 sequences from chr22\n",
      "\tExtracting 1301 sequences from chr3\n",
      "\tExtracting 797 sequences from chr4\n",
      "\tExtracting 975 sequences from chr5\n",
      "\tExtracting 1338 sequences from chr6\n",
      "\tExtracting 1216 sequences from chr7\n",
      "\tExtracting 851 sequences from chr8\n",
      "\tExtracting 1013 sequences from chr9\n",
      "\tExtracting 367 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24856 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2602 sequences from chr1\n",
      "\tExtracting 1005 sequences from chr10\n",
      "\tExtracting 1205 sequences from chr11\n",
      "\tExtracting 1252 sequences from chr12\n",
      "\tExtracting 479 sequences from chr13\n",
      "\tExtracting 861 sequences from chr14\n",
      "\tExtracting 828 sequences from chr15\n",
      "\tExtracting 1165 sequences from chr16\n",
      "\tExtracting 1694 sequences from chr17\n",
      "\tExtracting 400 sequences from chr18\n",
      "\tExtracting 1933 sequences from chr19\n",
      "\tExtracting 1694 sequences from chr2\n",
      "\tExtracting 655 sequences from chr20\n",
      "\tExtracting 346 sequences from chr21\n",
      "\tExtracting 676 sequences from chr22\n",
      "\tExtracting 1308 sequences from chr3\n",
      "\tExtracting 850 sequences from chr4\n",
      "\tExtracting 1095 sequences from chr5\n",
      "\tExtracting 1306 sequences from chr6\n",
      "\tExtracting 1185 sequences from chr7\n",
      "\tExtracting 865 sequences from chr8\n",
      "\tExtracting 935 sequences from chr9\n",
      "\tExtracting 621 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24960 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2489 sequences from chr1\n",
      "\tExtracting 898 sequences from chr10\n",
      "\tExtracting 1297 sequences from chr11\n",
      "\tExtracting 1300 sequences from chr12\n",
      "\tExtracting 429 sequences from chr13\n",
      "\tExtracting 905 sequences from chr14\n",
      "\tExtracting 695 sequences from chr15\n",
      "\tExtracting 1228 sequences from chr16\n",
      "\tExtracting 1691 sequences from chr17\n",
      "\tExtracting 341 sequences from chr18\n",
      "\tExtracting 2448 sequences from chr19\n",
      "\tExtracting 1623 sequences from chr2\n",
      "\tExtracting 636 sequences from chr20\n",
      "\tExtracting 496 sequences from chr21\n",
      "\tExtracting 607 sequences from chr22\n",
      "\tExtracting 1266 sequences from chr3\n",
      "\tExtracting 767 sequences from chr4\n",
      "\tExtracting 1015 sequences from chr5\n",
      "\tExtracting 1289 sequences from chr6\n",
      "\tExtracting 1201 sequences from chr7\n",
      "\tExtracting 757 sequences from chr8\n",
      "\tExtracting 1029 sequences from chr9\n",
      "\tExtracting 453 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24860 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2518 sequences from chr1\n",
      "\tExtracting 976 sequences from chr10\n",
      "\tExtracting 1265 sequences from chr11\n",
      "\tExtracting 1235 sequences from chr12\n",
      "\tExtracting 427 sequences from chr13\n",
      "\tExtracting 914 sequences from chr14\n",
      "\tExtracting 775 sequences from chr15\n",
      "\tExtracting 1175 sequences from chr16\n",
      "\tExtracting 1709 sequences from chr17\n",
      "\tExtracting 382 sequences from chr18\n",
      "\tExtracting 2119 sequences from chr19\n",
      "\tExtracting 1679 sequences from chr2\n",
      "\tExtracting 650 sequences from chr20\n",
      "\tExtracting 389 sequences from chr21\n",
      "\tExtracting 647 sequences from chr22\n",
      "\tExtracting 1277 sequences from chr3\n",
      "\tExtracting 792 sequences from chr4\n",
      "\tExtracting 1065 sequences from chr5\n",
      "\tExtracting 1354 sequences from chr6\n",
      "\tExtracting 1154 sequences from chr7\n",
      "\tExtracting 865 sequences from chr8\n",
      "\tExtracting 989 sequences from chr9\n",
      "\tExtracting 584 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24940 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2610 sequences from chr1\n",
      "\tExtracting 1012 sequences from chr10\n",
      "\tExtracting 1301 sequences from chr11\n",
      "\tExtracting 1238 sequences from chr12\n",
      "\tExtracting 492 sequences from chr13\n",
      "\tExtracting 918 sequences from chr14\n",
      "\tExtracting 821 sequences from chr15\n",
      "\tExtracting 1148 sequences from chr16\n",
      "\tExtracting 1670 sequences from chr17\n",
      "\tExtracting 378 sequences from chr18\n",
      "\tExtracting 1773 sequences from chr19\n",
      "\tExtracting 1687 sequences from chr2\n",
      "\tExtracting 663 sequences from chr20\n",
      "\tExtracting 316 sequences from chr21\n",
      "\tExtracting 626 sequences from chr22\n",
      "\tExtracting 1310 sequences from chr3\n",
      "\tExtracting 786 sequences from chr4\n",
      "\tExtracting 1120 sequences from chr5\n",
      "\tExtracting 1366 sequences from chr6\n",
      "\tExtracting 1258 sequences from chr7\n",
      "\tExtracting 836 sequences from chr8\n",
      "\tExtracting 996 sequences from chr9\n",
      "\tExtracting 643 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24969 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset13/wellington.tsv.gz tmp_static/Subset13/homer.tsv.gz tmp_static/Subset13/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset13/expression.tsv.gz tmp_static/Subset13/wellington.tsv.gz data/gene.bed tmp_static/Subset13/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset15/wellington.tsv.gz tmp_static/Subset15/homer.tsv.gz tmp_static/Subset15/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset15/expression.tsv.gz tmp_static/Subset15/wellington.tsv.gz data/gene.bed tmp_static/Subset15/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset15/binding.tsv.gz tmp_static/Subset15/tssdist.tsv.gz tmp_static/Subset15/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset13/binding.tsv.gz tmp_static/Subset13/tssdist.tsv.gz tmp_static/Subset13/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset15/linking.tsv.gz tmp_static/Subset15/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset15/expression.tsv.gz tmp_static/Subset15/binlinking.tsv.gz tmp_static/Subset15/net_weight.tsv.gz tmp_static/Subset15/net_meanvar.tsv.gz tmp_static/Subset15/net_covfactor.tsv.gz tmp_static/Subset15/net_loss.tsv.gz tmp_static/Subset15/net_stats.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset13/linking.tsv.gz tmp_static/Subset13/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset13/expression.tsv.gz tmp_static/Subset13/binlinking.tsv.gz tmp_static/Subset13/net_weight.tsv.gz tmp_static/Subset13/net_meanvar.tsv.gz tmp_static/Subset13/net_covfactor.tsv.gz tmp_static/Subset13/net_loss.tsv.gz tmp_static/Subset13/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset10/footprints.bed data/motifs.motif data/genome tmp_static/Subset10/expression.tsv.gz tmp_static/Subset10/motifs.bed tmp_static/Subset10/wellington.tsv.gz tmp_static/Subset10/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2576 sequences from chr1\n",
      "\tExtracting 1105 sequences from chr10\n",
      "\tExtracting 1325 sequences from chr11\n",
      "\tExtracting 1240 sequences from chr12\n",
      "\tExtracting 482 sequences from chr13\n",
      "\tExtracting 858 sequences from chr14\n",
      "\tExtracting 817 sequences from chr15\n",
      "\tExtracting 1136 sequences from chr16\n",
      "\tExtracting 1594 sequences from chr17\n",
      "\tExtracting 326 sequences from chr18\n",
      "\tExtracting 1796 sequences from chr19\n",
      "\tExtracting 1699 sequences from chr2\n",
      "\tExtracting 647 sequences from chr20\n",
      "\tExtracting 341 sequences from chr21\n",
      "\tExtracting 634 sequences from chr22\n",
      "\tExtracting 1354 sequences from chr3\n",
      "\tExtracting 839 sequences from chr4\n",
      "\tExtracting 1108 sequences from chr5\n",
      "\tExtracting 1454 sequences from chr6\n",
      "\tExtracting 1143 sequences from chr7\n",
      "\tExtracting 844 sequences from chr8\n",
      "\tExtracting 954 sequences from chr9\n",
      "\tExtracting 690 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24962 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2595 sequences from chr1\n",
      "\tExtracting 1039 sequences from chr10\n",
      "\tExtracting 1278 sequences from chr11\n",
      "\tExtracting 1223 sequences from chr12\n",
      "\tExtracting 507 sequences from chr13\n",
      "\tExtracting 857 sequences from chr14\n",
      "\tExtracting 819 sequences from chr15\n",
      "\tExtracting 1143 sequences from chr16\n",
      "\tExtracting 1658 sequences from chr17\n",
      "\tExtracting 322 sequences from chr18\n",
      "\tExtracting 1883 sequences from chr19\n",
      "\tExtracting 1621 sequences from chr2\n",
      "\tExtracting 631 sequences from chr20\n",
      "\tExtracting 381 sequences from chr21\n",
      "\tExtracting 622 sequences from chr22\n",
      "\tExtracting 1377 sequences from chr3\n",
      "\tExtracting 792 sequences from chr4\n",
      "\tExtracting 1144 sequences from chr5\n",
      "\tExtracting 1414 sequences from chr6\n",
      "\tExtracting 1160 sequences from chr7\n",
      "\tExtracting 887 sequences from chr8\n",
      "\tExtracting 965 sequences from chr9\n",
      "\tExtracting 642 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24961 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2633 sequences from chr1\n",
      "\tExtracting 965 sequences from chr10\n",
      "\tExtracting 1352 sequences from chr11\n",
      "\tExtracting 1345 sequences from chr12\n",
      "\tExtracting 440 sequences from chr13\n",
      "\tExtracting 911 sequences from chr14\n",
      "\tExtracting 652 sequences from chr15\n",
      "\tExtracting 1232 sequences from chr16\n",
      "\tExtracting 1655 sequences from chr17\n",
      "\tExtracting 327 sequences from chr18\n",
      "\tExtracting 2341 sequences from chr19\n",
      "\tExtracting 1582 sequences from chr2\n",
      "\tExtracting 626 sequences from chr20\n",
      "\tExtracting 394 sequences from chr21\n",
      "\tExtracting 592 sequences from chr22\n",
      "\tExtracting 1283 sequences from chr3\n",
      "\tExtracting 796 sequences from chr4\n",
      "\tExtracting 1016 sequences from chr5\n",
      "\tExtracting 1291 sequences from chr6\n",
      "\tExtracting 1226 sequences from chr7\n",
      "\tExtracting 789 sequences from chr8\n",
      "\tExtracting 1018 sequences from chr9\n",
      "\tExtracting 449 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24916 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2652 sequences from chr1\n",
      "\tExtracting 970 sequences from chr10\n",
      "\tExtracting 1294 sequences from chr11\n",
      "\tExtracting 1306 sequences from chr12\n",
      "\tExtracting 477 sequences from chr13\n",
      "\tExtracting 871 sequences from chr14\n",
      "\tExtracting 772 sequences from chr15\n",
      "\tExtracting 1173 sequences from chr16\n",
      "\tExtracting 1654 sequences from chr17\n",
      "\tExtracting 336 sequences from chr18\n",
      "\tExtracting 1968 sequences from chr19\n",
      "\tExtracting 1671 sequences from chr2\n",
      "\tExtracting 600 sequences from chr20\n",
      "\tExtracting 375 sequences from chr21\n",
      "\tExtracting 630 sequences from chr22\n",
      "\tExtracting 1310 sequences from chr3\n",
      "\tExtracting 801 sequences from chr4\n",
      "\tExtracting 1031 sequences from chr5\n",
      "\tExtracting 1413 sequences from chr6\n",
      "\tExtracting 1155 sequences from chr7\n",
      "\tExtracting 902 sequences from chr8\n",
      "\tExtracting 1010 sequences from chr9\n",
      "\tExtracting 558 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24930 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset10/wellington.tsv.gz tmp_static/Subset10/homer.tsv.gz tmp_static/Subset10/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset10/expression.tsv.gz tmp_static/Subset10/wellington.tsv.gz data/gene.bed tmp_static/Subset10/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset10/binding.tsv.gz tmp_static/Subset10/tssdist.tsv.gz tmp_static/Subset10/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset10/linking.tsv.gz tmp_static/Subset10/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset10/expression.tsv.gz tmp_static/Subset10/binlinking.tsv.gz tmp_static/Subset10/net_weight.tsv.gz tmp_static/Subset10/net_meanvar.tsv.gz tmp_static/Subset10/net_covfactor.tsv.gz tmp_static/Subset10/net_loss.tsv.gz tmp_static/Subset10/net_stats.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset7/footprints.bed data/motifs.motif data/genome tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/motifs.bed tmp_static/Subset7/wellington.tsv.gz tmp_static/Subset7/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2577 sequences from chr1\n",
      "\tExtracting 980 sequences from chr10\n",
      "\tExtracting 1369 sequences from chr11\n",
      "\tExtracting 1281 sequences from chr12\n",
      "\tExtracting 490 sequences from chr13\n",
      "\tExtracting 901 sequences from chr14\n",
      "\tExtracting 693 sequences from chr15\n",
      "\tExtracting 1164 sequences from chr16\n",
      "\tExtracting 1621 sequences from chr17\n",
      "\tExtracting 362 sequences from chr18\n",
      "\tExtracting 2080 sequences from chr19\n",
      "\tExtracting 1674 sequences from chr2\n",
      "\tExtracting 651 sequences from chr20\n",
      "\tExtracting 401 sequences from chr21\n",
      "\tExtracting 611 sequences from chr22\n",
      "\tExtracting 1286 sequences from chr3\n",
      "\tExtracting 817 sequences from chr4\n",
      "\tExtracting 1060 sequences from chr5\n",
      "\tExtracting 1349 sequences from chr6\n",
      "\tExtracting 1173 sequences from chr7\n",
      "\tExtracting 869 sequences from chr8\n",
      "\tExtracting 967 sequences from chr9\n",
      "\tExtracting 560 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24937 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2602 sequences from chr1\n",
      "\tExtracting 879 sequences from chr10\n",
      "\tExtracting 1313 sequences from chr11\n",
      "\tExtracting 1255 sequences from chr12\n",
      "\tExtracting 435 sequences from chr13\n",
      "\tExtracting 886 sequences from chr14\n",
      "\tExtracting 650 sequences from chr15\n",
      "\tExtracting 1284 sequences from chr16\n",
      "\tExtracting 1702 sequences from chr17\n",
      "\tExtracting 286 sequences from chr18\n",
      "\tExtracting 2481 sequences from chr19\n",
      "\tExtracting 1519 sequences from chr2\n",
      "\tExtracting 635 sequences from chr20\n",
      "\tExtracting 683 sequences from chr21\n",
      "\tExtracting 588 sequences from chr22\n",
      "\tExtracting 1227 sequences from chr3\n",
      "\tExtracting 786 sequences from chr4\n",
      "\tExtracting 940 sequences from chr5\n",
      "\tExtracting 1298 sequences from chr6\n",
      "\tExtracting 1162 sequences from chr7\n",
      "\tExtracting 747 sequences from chr8\n",
      "\tExtracting 1038 sequences from chr9\n",
      "\tExtracting 389 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24786 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2610 sequences from chr1\n",
      "\tExtracting 1019 sequences from chr10\n",
      "\tExtracting 1283 sequences from chr11\n",
      "\tExtracting 1258 sequences from chr12\n",
      "\tExtracting 499 sequences from chr13\n",
      "\tExtracting 888 sequences from chr14\n",
      "\tExtracting 770 sequences from chr15\n",
      "\tExtracting 1167 sequences from chr16\n",
      "\tExtracting 1597 sequences from chr17\n",
      "\tExtracting 356 sequences from chr18\n",
      "\tExtracting 1786 sequences from chr19\n",
      "\tExtracting 1738 sequences from chr2\n",
      "\tExtracting 668 sequences from chr20\n",
      "\tExtracting 340 sequences from chr21\n",
      "\tExtracting 689 sequences from chr22\n",
      "\tExtracting 1278 sequences from chr3\n",
      "\tExtracting 854 sequences from chr4\n",
      "\tExtracting 1080 sequences from chr5\n",
      "\tExtracting 1412 sequences from chr6\n",
      "\tExtracting 1127 sequences from chr7\n",
      "\tExtracting 924 sequences from chr8\n",
      "\tExtracting 985 sequences from chr9\n",
      "\tExtracting 642 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24970 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2513 sequences from chr1\n",
      "\tExtracting 1031 sequences from chr10\n",
      "\tExtracting 1293 sequences from chr11\n",
      "\tExtracting 1349 sequences from chr12\n",
      "\tExtracting 498 sequences from chr13\n",
      "\tExtracting 873 sequences from chr14\n",
      "\tExtracting 771 sequences from chr15\n",
      "\tExtracting 1095 sequences from chr16\n",
      "\tExtracting 1727 sequences from chr17\n",
      "\tExtracting 322 sequences from chr18\n",
      "\tExtracting 1938 sequences from chr19\n",
      "\tExtracting 1572 sequences from chr2\n",
      "\tExtracting 646 sequences from chr20\n",
      "\tExtracting 322 sequences from chr21\n",
      "\tExtracting 610 sequences from chr22\n",
      "\tExtracting 1301 sequences from chr3\n",
      "\tExtracting 808 sequences from chr4\n",
      "\tExtracting 1159 sequences from chr5\n",
      "\tExtracting 1434 sequences from chr6\n",
      "\tExtracting 1196 sequences from chr7\n",
      "\tExtracting 883 sequences from chr8\n",
      "\tExtracting 986 sequences from chr9\n",
      "\tExtracting 645 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24972 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset7/wellington.tsv.gz tmp_static/Subset7/homer.tsv.gz tmp_static/Subset7/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/wellington.tsv.gz data/gene.bed tmp_static/Subset7/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset7/binding.tsv.gz tmp_static/Subset7/tssdist.tsv.gz tmp_static/Subset7/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset7/linking.tsv.gz tmp_static/Subset7/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset7/expression.tsv.gz tmp_static/Subset7/binlinking.tsv.gz tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_loss.tsv.gz tmp_static/Subset7/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset8/footprints.bed data/motifs.motif data/genome tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/motifs.bed tmp_static/Subset8/wellington.tsv.gz tmp_static/Subset8/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset11/footprints.bed data/motifs.motif data/genome tmp_static/Subset11/expression.tsv.gz tmp_static/Subset11/motifs.bed tmp_static/Subset11/wellington.tsv.gz tmp_static/Subset11/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2622 sequences from chr1\n",
      "\tExtracting 980 sequences from chr10\n",
      "\tExtracting 1290 sequences from chr11\n",
      "\tExtracting 1297 sequences from chr12\n",
      "\tExtracting 461 sequences from chr13\n",
      "\tExtracting 891 sequences from chr14\n",
      "\tExtracting 733 sequences from chr15\n",
      "\tExtracting 1132 sequences from chr16\n",
      "\tExtracting 1688 sequences from chr17\n",
      "\tExtracting 345 sequences from chr18\n",
      "\tExtracting 2032 sequences from chr19\n",
      "\tExtracting 1597 sequences from chr2\n",
      "\tExtracting 597 sequences from chr20\n",
      "\tExtracting 433 sequences from chr21\n",
      "\tExtracting 635 sequences from chr22\n",
      "\tExtracting 1336 sequences from chr3\n",
      "\tExtracting 837 sequences from chr4\n",
      "\tExtracting 1062 sequences from chr5\n",
      "\tExtracting 1377 sequences from chr6\n",
      "\tExtracting 1136 sequences from chr7\n",
      "\tExtracting 874 sequences from chr8\n",
      "\tExtracting 969 sequences from chr9\n",
      "\tExtracting 611 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24936 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2655 sequences from chr1\n",
      "\tExtracting 963 sequences from chr10\n",
      "\tExtracting 1307 sequences from chr11\n",
      "\tExtracting 1288 sequences from chr12\n",
      "\tExtracting 453 sequences from chr13\n",
      "\tExtracting 899 sequences from chr14\n",
      "\tExtracting 648 sequences from chr15\n",
      "\tExtracting 1213 sequences from chr16\n",
      "\tExtracting 1632 sequences from chr17\n",
      "\tExtracting 309 sequences from chr18\n",
      "\tExtracting 2428 sequences from chr19\n",
      "\tExtracting 1555 sequences from chr2\n",
      "\tExtracting 595 sequences from chr20\n",
      "\tExtracting 464 sequences from chr21\n",
      "\tExtracting 617 sequences from chr22\n",
      "\tExtracting 1254 sequences from chr3\n",
      "\tExtracting 797 sequences from chr4\n",
      "\tExtracting 1001 sequences from chr5\n",
      "\tExtracting 1264 sequences from chr6\n",
      "\tExtracting 1215 sequences from chr7\n",
      "\tExtracting 803 sequences from chr8\n",
      "\tExtracting 1031 sequences from chr9\n",
      "\tExtracting 469 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24861 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2608 sequences from chr1\n",
      "\tExtracting 1034 sequences from chr10\n",
      "\tExtracting 1329 sequences from chr11\n",
      "\tExtracting 1255 sequences from chr12\n",
      "\tExtracting 516 sequences from chr13\n",
      "\tExtracting 841 sequences from chr14\n",
      "\tExtracting 830 sequences from chr15\n",
      "\tExtracting 1085 sequences from chr16\n",
      "\tExtracting 1601 sequences from chr17\n",
      "\tExtracting 361 sequences from chr18\n",
      "\tExtracting 1801 sequences from chr19\n",
      "\tExtracting 1724 sequences from chr2\n",
      "\tExtracting 679 sequences from chr20\n",
      "\tExtracting 343 sequences from chr21\n",
      "\tExtracting 618 sequences from chr22\n",
      "\tExtracting 1320 sequences from chr3\n",
      "\tExtracting 867 sequences from chr4\n",
      "\tExtracting 1089 sequences from chr5\n",
      "\tExtracting 1417 sequences from chr6\n",
      "\tExtracting 1187 sequences from chr7\n",
      "\tExtracting 854 sequences from chr8\n",
      "\tExtracting 956 sequences from chr9\n",
      "\tExtracting 655 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24970 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2563 sequences from chr1\n",
      "\tExtracting 1043 sequences from chr10\n",
      "\tExtracting 1349 sequences from chr11\n",
      "\tExtracting 1248 sequences from chr12\n",
      "\tExtracting 454 sequences from chr13\n",
      "\tExtracting 886 sequences from chr14\n",
      "\tExtracting 757 sequences from chr15\n",
      "\tExtracting 1193 sequences from chr16\n",
      "\tExtracting 1626 sequences from chr17\n",
      "\tExtracting 326 sequences from chr18\n",
      "\tExtracting 1859 sequences from chr19\n",
      "\tExtracting 1693 sequences from chr2\n",
      "\tExtracting 668 sequences from chr20\n",
      "\tExtracting 392 sequences from chr21\n",
      "\tExtracting 628 sequences from chr22\n",
      "\tExtracting 1322 sequences from chr3\n",
      "\tExtracting 789 sequences from chr4\n",
      "\tExtracting 1112 sequences from chr5\n",
      "\tExtracting 1342 sequences from chr6\n",
      "\tExtracting 1174 sequences from chr7\n",
      "\tExtracting 904 sequences from chr8\n",
      "\tExtracting 995 sequences from chr9\n",
      "\tExtracting 634 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24957 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset8/wellington.tsv.gz tmp_static/Subset8/homer.tsv.gz tmp_static/Subset8/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/wellington.tsv.gz data/gene.bed tmp_static/Subset8/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset8/binding.tsv.gz tmp_static/Subset8/tssdist.tsv.gz tmp_static/Subset8/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset8/linking.tsv.gz tmp_static/Subset8/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2479 sequences from chr1\n",
      "\tExtracting 1016 sequences from chr10\n",
      "\tExtracting 1300 sequences from chr11\n",
      "\tExtracting 1248 sequences from chr12\n",
      "\tExtracting 420 sequences from chr13\n",
      "\tExtracting 847 sequences from chr14\n",
      "\tExtracting 709 sequences from chr15\n",
      "\tExtracting 1210 sequences from chr16\n",
      "\tExtracting 1716 sequences from chr17\n",
      "\tExtracting 384 sequences from chr18\n",
      "\tExtracting 2319 sequences from chr19\n",
      "\tExtracting 1656 sequences from chr2\n",
      "\tExtracting 669 sequences from chr20\n",
      "\tExtracting 310 sequences from chr21\n",
      "\tExtracting 583 sequences from chr22\n",
      "\tExtracting 1381 sequences from chr3\n",
      "\tExtracting 841 sequences from chr4\n",
      "\tExtracting 1035 sequences from chr5\n",
      "\tExtracting 1310 sequences from chr6\n",
      "\tExtracting 1203 sequences from chr7\n",
      "\tExtracting 846 sequences from chr8\n",
      "\tExtracting 1083 sequences from chr9\n",
      "\tExtracting 408 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24973 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2505 sequences from chr1\n",
      "\tExtracting 1158 sequences from chr10\n",
      "\tExtracting 1334 sequences from chr11\n",
      "\tExtracting 1291 sequences from chr12\n",
      "\tExtracting 498 sequences from chr13\n",
      "\tExtracting 892 sequences from chr14\n",
      "\tExtracting 784 sequences from chr15\n",
      "\tExtracting 1097 sequences from chr16\n",
      "\tExtracting 1582 sequences from chr17\n",
      "\tExtracting 400 sequences from chr18\n",
      "\tExtracting 1704 sequences from chr19\n",
      "\tExtracting 1742 sequences from chr2\n",
      "\tExtracting 623 sequences from chr20\n",
      "\tExtracting 375 sequences from chr21\n",
      "\tExtracting 597 sequences from chr22\n",
      "\tExtracting 1355 sequences from chr3\n",
      "\tExtracting 832 sequences from chr4\n",
      "\tExtracting 1085 sequences from chr5\n",
      "\tExtracting 1428 sequences from chr6\n",
      "\tExtracting 1169 sequences from chr7\n",
      "\tExtracting 916 sequences from chr8\n",
      "\tExtracting 1014 sequences from chr9\n",
      "\tExtracting 588 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24970 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2510 sequences from chr1\n",
      "\tExtracting 1083 sequences from chr10\n",
      "\tExtracting 1338 sequences from chr11\n",
      "\tExtracting 1348 sequences from chr12\n",
      "\tExtracting 427 sequences from chr13\n",
      "\tExtracting 862 sequences from chr14\n",
      "\tExtracting 783 sequences from chr15\n",
      "\tExtracting 1142 sequences from chr16\n",
      "\tExtracting 1677 sequences from chr17\n",
      "\tExtracting 351 sequences from chr18\n",
      "\tExtracting 1844 sequences from chr19\n",
      "\tExtracting 1755 sequences from chr2\n",
      "\tExtracting 645 sequences from chr20\n",
      "\tExtracting 337 sequences from chr21\n",
      "\tExtracting 602 sequences from chr22\n",
      "\tExtracting 1366 sequences from chr3\n",
      "\tExtracting 868 sequences from chr4\n",
      "\tExtracting 1100 sequences from chr5\n",
      "\tExtracting 1310 sequences from chr6\n",
      "\tExtracting 1180 sequences from chr7\n",
      "\tExtracting 869 sequences from chr8\n",
      "\tExtracting 991 sequences from chr9\n",
      "\tExtracting 576 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24965 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2529 sequences from chr1\n",
      "\tExtracting 1086 sequences from chr10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 1329 sequences from chr11\n",
      "\tExtracting 1284 sequences from chr12\n",
      "\tExtracting 481 sequences from chr13\n",
      "\tExtracting 832 sequences from chr14\n",
      "\tExtracting 752 sequences from chr15\n",
      "\tExtracting 1050 sequences from chr16\n",
      "\tExtracting 1510 sequences from chr17\n",
      "\tExtracting 392 sequences from chr18\n",
      "\tExtracting 1533 sequences from chr19\n",
      "\tExtracting 1776 sequences from chr2\n",
      "\tExtracting 690 sequences from chr20\n",
      "\tExtracting 378 sequences from chr21\n",
      "\tExtracting 584 sequences from chr22\n",
      "\tExtracting 1385 sequences from chr3\n",
      "\tExtracting 858 sequences from chr4\n",
      "\tExtracting 1163 sequences from chr5\n",
      "\tExtracting 1464 sequences from chr6\n",
      "\tExtracting 1219 sequences from chr7\n",
      "\tExtracting 934 sequences from chr8\n",
      "\tExtracting 1104 sequences from chr9\n",
      "\tExtracting 624 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24957 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset8/expression.tsv.gz tmp_static/Subset8/binlinking.tsv.gz tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_loss.tsv.gz tmp_static/Subset8/net_stats.tsv.gz\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset11/wellington.tsv.gz tmp_static/Subset11/homer.tsv.gz tmp_static/Subset11/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset11/expression.tsv.gz tmp_static/Subset11/wellington.tsv.gz data/gene.bed tmp_static/Subset11/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset11/binding.tsv.gz tmp_static/Subset11/tssdist.tsv.gz tmp_static/Subset11/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset11/linking.tsv.gz tmp_static/Subset11/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset11/expression.tsv.gz tmp_static/Subset11/binlinking.tsv.gz tmp_static/Subset11/net_weight.tsv.gz tmp_static/Subset11/net_meanvar.tsv.gz tmp_static/Subset11/net_covfactor.tsv.gz tmp_static/Subset11/net_loss.tsv.gz tmp_static/Subset11/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset4/footprints.bed data/motifs.motif data/genome tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/motifs.bed tmp_static/Subset4/wellington.tsv.gz tmp_static/Subset4/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset6/footprints.bed data/motifs.motif data/genome tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/motifs.bed tmp_static/Subset6/wellington.tsv.gz tmp_static/Subset6/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2582 sequences from chr1\n",
      "\tExtracting 1017 sequences from chr10\n",
      "\tExtracting 1321 sequences from chr11\n",
      "\tExtracting 1299 sequences from chr12\n",
      "\tExtracting 541 sequences from chr13\n",
      "\tExtracting 814 sequences from chr14\n",
      "\tExtracting 747 sequences from chr15\n",
      "\tExtracting 1098 sequences from chr16\n",
      "\tExtracting 1655 sequences from chr17\n",
      "\tExtracting 365 sequences from chr18\n",
      "\tExtracting 1873 sequences from chr19\n",
      "\tExtracting 1715 sequences from chr2\n",
      "\tExtracting 645 sequences from chr20\n",
      "\tExtracting 366 sequences from chr21\n",
      "\tExtracting 630 sequences from chr22\n",
      "\tExtracting 1315 sequences from chr3\n",
      "\tExtracting 817 sequences from chr4\n",
      "\tExtracting 1067 sequences from chr5\n",
      "\tExtracting 1469 sequences from chr6\n",
      "\tExtracting 1181 sequences from chr7\n",
      "\tExtracting 860 sequences from chr8\n",
      "\tExtracting 1009 sequences from chr9\n",
      "\tExtracting 585 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24971 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2580 sequences from chr1\n",
      "\tExtracting 1054 sequences from chr10\n",
      "\tExtracting 1336 sequences from chr11\n",
      "\tExtracting 1338 sequences from chr12\n",
      "\tExtracting 466 sequences from chr13\n",
      "\tExtracting 872 sequences from chr14\n",
      "\tExtracting 814 sequences from chr15\n",
      "\tExtracting 1103 sequences from chr16\n",
      "\tExtracting 1697 sequences from chr17\n",
      "\tExtracting 369 sequences from chr18\n",
      "\tExtracting 1724 sequences from chr19\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 1720 sequences from chr2\n",
      "\tExtracting 680 sequences from chr20\n",
      "\tExtracting 341 sequences from chr21\n",
      "\tExtracting 661 sequences from chr22\n",
      "\tExtracting 1296 sequences from chr3\n",
      "\tExtracting 858 sequences from chr4\n",
      "\tExtracting 1073 sequences from chr5\n",
      "\tExtracting 1407 sequences from chr6\n",
      "\tExtracting 1137 sequences from chr7\n",
      "\tExtracting 861 sequences from chr8\n",
      "\tExtracting 941 sequences from chr9\n",
      "\tExtracting 636 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24964 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2541 sequences from chr1\n",
      "\tExtracting 987 sequences from chr10\n",
      "\tExtracting 1289 sequences from chr11\n",
      "\tExtracting 1265 sequences from chr12\n",
      "\tExtracting 471 sequences from chr13\n",
      "\tExtracting 906 sequences from chr14\n",
      "\tExtracting 737 sequences from chr15\n",
      "\tExtracting 1196 sequences from chr16\n",
      "\tExtracting 1611 sequences from chr17\n",
      "\tExtracting 315 sequences from chr18\n",
      "\tExtracting 2084 sequences from chr19\n",
      "\tExtracting 1631 sequences from chr2\n",
      "\tExtracting 654 sequences from chr20\n",
      "\tExtracting 437 sequences from chr21\n",
      "\tExtracting 617 sequences from chr22\n",
      "\tExtracting 1241 sequences from chr3\n",
      "\tExtracting 776 sequences from chr4\n",
      "\tExtracting 1101 sequences from chr5\n",
      "\tExtracting 1433 sequences from chr6\n",
      "\tExtracting 1200 sequences from chr7\n",
      "\tExtracting 906 sequences from chr8\n",
      "\tExtracting 955 sequences from chr9\n",
      "\tExtracting 581 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24936 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2615 sequences from chr1\n",
      "\tExtracting 907 sequences from chr10\n",
      "\tExtracting 1316 sequences from chr11\n",
      "\tExtracting 1311 sequences from chr12\n",
      "\tExtracting 430 sequences from chr13\n",
      "\tExtracting 843 sequences from chr14\n",
      "\tExtracting 635 sequences from chr15\n",
      "\tExtracting 1216 sequences from chr16\n",
      "\tExtracting 1645 sequences from chr17\n",
      "\tExtracting 297 sequences from chr18\n",
      "\tExtracting 2447 sequences from chr19\n",
      "\tExtracting 1556 sequences from chr2\n",
      "\tExtracting 616 sequences from chr20\n",
      "\tExtracting 631 sequences from chr21\n",
      "\tExtracting 554 sequences from chr22\n",
      "\tExtracting 1258 sequences from chr3\n",
      "\tExtracting 829 sequences from chr4\n",
      "\tExtracting 995 sequences from chr5\n",
      "\tExtracting 1261 sequences from chr6\n",
      "\tExtracting 1221 sequences from chr7\n",
      "\tExtracting 804 sequences from chr8\n",
      "\tExtracting 1025 sequences from chr9\n",
      "\tExtracting 392 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24805 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset4/wellington.tsv.gz tmp_static/Subset4/homer.tsv.gz tmp_static/Subset4/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/wellington.tsv.gz data/gene.bed tmp_static/Subset4/tssdist.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2674 sequences from chr1\n",
      "\tExtracting 958 sequences from chr10\n",
      "\tExtracting 1280 sequences from chr11\n",
      "\tExtracting 1332 sequences from chr12\n",
      "\tExtracting 443 sequences from chr13\n",
      "\tExtracting 858 sequences from chr14\n",
      "\tExtracting 743 sequences from chr15\n",
      "\tExtracting 1142 sequences from chr16\n",
      "\tExtracting 1659 sequences from chr17\n",
      "\tExtracting 329 sequences from chr18\n",
      "\tExtracting 2089 sequences from chr19\n",
      "\tExtracting 1682 sequences from chr2\n",
      "\tExtracting 585 sequences from chr20\n",
      "\tExtracting 429 sequences from chr21\n",
      "\tExtracting 565 sequences from chr22\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 1288 sequences from chr3\n",
      "\tExtracting 793 sequences from chr4\n",
      "\tExtracting 1084 sequences from chr5\n",
      "\tExtracting 1385 sequences from chr6\n",
      "\tExtracting 1160 sequences from chr7\n",
      "\tExtracting 873 sequences from chr8\n",
      "\tExtracting 951 sequences from chr9\n",
      "\tExtracting 620 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24922 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2600 sequences from chr1\n",
      "\tExtracting 1028 sequences from chr10\n",
      "\tExtracting 1305 sequences from chr11\n",
      "\tExtracting 1284 sequences from chr12\n",
      "\tExtracting 506 sequences from chr13\n",
      "\tExtracting 846 sequences from chr14\n",
      "\tExtracting 788 sequences from chr15\n",
      "\tExtracting 1129 sequences from chr16\n",
      "\tExtracting 1626 sequences from chr17\n",
      "\tExtracting 359 sequences from chr18\n",
      "\tExtracting 1834 sequences from chr19\n",
      "\tExtracting 1672 sequences from chr2\n",
      "\tExtracting 663 sequences from chr20\n",
      "\tExtracting 366 sequences from chr21\n",
      "\tExtracting 659 sequences from chr22\n",
      "\tExtracting 1353 sequences from chr3\n",
      "\tExtracting 824 sequences from chr4\n",
      "\tExtracting 1023 sequences from chr5\n",
      "\tExtracting 1417 sequences from chr6\n",
      "\tExtracting 1166 sequences from chr7\n",
      "\tExtracting 864 sequences from chr8\n",
      "\tExtracting 990 sequences from chr9\n",
      "\tExtracting 644 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24947 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2519 sequences from chr1\n",
      "\tExtracting 1101 sequences from chr10\n",
      "\tExtracting 1306 sequences from chr11\n",
      "\tExtracting 1284 sequences from chr12\n",
      "\tExtracting 533 sequences from chr13\n",
      "\tExtracting 899 sequences from chr14\n",
      "\tExtracting 766 sequences from chr15\n",
      "\tExtracting 1099 sequences from chr16\n",
      "\tExtracting 1669 sequences from chr17\n",
      "\tExtracting 350 sequences from chr18\n",
      "\tExtracting 1784 sequences from chr19\n",
      "\tExtracting 1692 sequences from chr2\n",
      "\tExtracting 691 sequences from chr20\n",
      "\tExtracting 362 sequences from chr21\n",
      "\tExtracting 658 sequences from chr22\n",
      "\tExtracting 1311 sequences from chr3\n",
      "\tExtracting 800 sequences from chr4\n",
      "\tExtracting 1077 sequences from chr5\n",
      "\tExtracting 1437 sequences from chr6\n",
      "\tExtracting 1168 sequences from chr7\n",
      "\tExtracting 827 sequences from chr8\n",
      "\tExtracting 980 sequences from chr9\n",
      "\tExtracting 649 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24962 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2639 sequences from chr1\n",
      "\tExtracting 915 sequences from chr10\n",
      "\tExtracting 1338 sequences from chr11\n",
      "\tExtracting 1289 sequences from chr12\n",
      "\tExtracting 428 sequences from chr13\n",
      "\tExtracting 901 sequences from chr14\n",
      "\tExtracting 658 sequences from chr15\n",
      "\tExtracting 1242 sequences from chr16\n",
      "\tExtracting 1685 sequences from chr17\n",
      "\tExtracting 310 sequences from chr18\n",
      "\tExtracting 2409 sequences from chr19\n",
      "\tExtracting 1533 sequences from chr2\n",
      "\tExtracting 628 sequences from chr20\n",
      "\tExtracting 481 sequences from chr21\n",
      "\tExtracting 588 sequences from chr22\n",
      "\tExtracting 1270 sequences from chr3\n",
      "\tExtracting 805 sequences from chr4\n",
      "\tExtracting 991 sequences from chr5\n",
      "\tExtracting 1310 sequences from chr6\n",
      "\tExtracting 1189 sequences from chr7\n",
      "\tExtracting 802 sequences from chr8\n",
      "\tExtracting 1039 sequences from chr9\n",
      "\tExtracting 424 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24875 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset4/binding.tsv.gz tmp_static/Subset4/tssdist.tsv.gz tmp_static/Subset4/linking.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset6/wellington.tsv.gz tmp_static/Subset6/homer.tsv.gz tmp_static/Subset6/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/wellington.tsv.gz data/gene.bed tmp_static/Subset6/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset4/linking.tsv.gz tmp_static/Subset4/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset4/expression.tsv.gz tmp_static/Subset4/binlinking.tsv.gz tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_loss.tsv.gz tmp_static/Subset4/net_stats.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset6/binding.tsv.gz tmp_static/Subset6/tssdist.tsv.gz tmp_static/Subset6/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset6/linking.tsv.gz tmp_static/Subset6/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset6/expression.tsv.gz tmp_static/Subset6/binlinking.tsv.gz tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_loss.tsv.gz tmp_static/Subset6/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset3/footprints.bed data/motifs.motif data/genome tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/motifs.bed tmp_static/Subset3/wellington.tsv.gz tmp_static/Subset3/homer.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset9/footprints.bed data/motifs.motif data/genome tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/motifs.bed tmp_static/Subset9/wellington.tsv.gz tmp_static/Subset9/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2653 sequences from chr1\n",
      "\tExtracting 1001 sequences from chr10\n",
      "\tExtracting 1260 sequences from chr11\n",
      "\tExtracting 1212 sequences from chr12\n",
      "\tExtracting 477 sequences from chr13\n",
      "\tExtracting 884 sequences from chr14\n",
      "\tExtracting 827 sequences from chr15\n",
      "\tExtracting 1163 sequences from chr16\n",
      "\tExtracting 1679 sequences from chr17\n",
      "\tExtracting 344 sequences from chr18\n",
      "\tExtracting 1928 sequences from chr19\n",
      "\tExtracting 1772 sequences from chr2\n",
      "\tExtracting 655 sequences from chr20\n",
      "\tExtracting 338 sequences from chr21\n",
      "\tExtracting 636 sequences from chr22\n",
      "\tExtracting 1306 sequences from chr3\n",
      "\tExtracting 806 sequences from chr4\n",
      "\tExtracting 1102 sequences from chr5\n",
      "\tExtracting 1389 sequences from chr6\n",
      "\tExtracting 1122 sequences from chr7\n",
      "\tExtracting 801 sequences from chr8\n",
      "\tExtracting 989 sequences from chr9\n",
      "\tExtracting 625 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24969 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2537 sequences from chr1\n",
      "\tExtracting 1104 sequences from chr10\n",
      "\tExtracting 1294 sequences from chr11\n",
      "\tExtracting 1293 sequences from chr12\n",
      "\tExtracting 490 sequences from chr13\n",
      "\tExtracting 887 sequences from chr14\n",
      "\tExtracting 787 sequences from chr15\n",
      "\tExtracting 1167 sequences from chr16\n",
      "\tExtracting 1733 sequences from chr17\n",
      "\tExtracting 376 sequences from chr18\n",
      "\tExtracting 1787 sequences from chr19\n",
      "\tExtracting 1686 sequences from chr2\n",
      "\tExtracting 672 sequences from chr20\n",
      "\tExtracting 291 sequences from chr21\n",
      "\tExtracting 632 sequences from chr22\n",
      "\tExtracting 1298 sequences from chr3\n",
      "\tExtracting 832 sequences from chr4\n",
      "\tExtracting 1068 sequences from chr5\n",
      "\tExtracting 1407 sequences from chr6\n",
      "\tExtracting 1163 sequences from chr7\n",
      "\tExtracting 872 sequences from chr8\n",
      "\tExtracting 947 sequences from chr9\n",
      "\tExtracting 640 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24964 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2631 sequences from chr1\n",
      "\tExtracting 937 sequences from chr10\n",
      "\tExtracting 1285 sequences from chr11\n",
      "\tExtracting 1235 sequences from chr12\n",
      "\tExtracting 433 sequences from chr13\n",
      "\tExtracting 863 sequences from chr14\n",
      "\tExtracting 756 sequences from chr15\n",
      "\tExtracting 1203 sequences from chr16\n",
      "\tExtracting 1709 sequences from chr17\n",
      "\tExtracting 364 sequences from chr18\n",
      "\tExtracting 2041 sequences from chr19\n",
      "\tExtracting 1698 sequences from chr2\n",
      "\tExtracting 675 sequences from chr20\n",
      "\tExtracting 395 sequences from chr21\n",
      "\tExtracting 643 sequences from chr22\n",
      "\tExtracting 1273 sequences from chr3\n",
      "\tExtracting 791 sequences from chr4\n",
      "\tExtracting 1100 sequences from chr5\n",
      "\tExtracting 1407 sequences from chr6\n",
      "\tExtracting 1136 sequences from chr7\n",
      "\tExtracting 773 sequences from chr8\n",
      "\tExtracting 1035 sequences from chr9\n",
      "\tExtracting 563 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24946 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2506 sequences from chr1\n",
      "\tExtracting 926 sequences from chr10\n",
      "\tExtracting 1295 sequences from chr11\n",
      "\tExtracting 1336 sequences from chr12\n",
      "\tExtracting 419 sequences from chr13\n",
      "\tExtracting 857 sequences from chr14\n",
      "\tExtracting 646 sequences from chr15\n",
      "\tExtracting 1238 sequences from chr16\n",
      "\tExtracting 1648 sequences from chr17\n",
      "\tExtracting 325 sequences from chr18\n",
      "\tExtracting 2408 sequences from chr19\n",
      "\tExtracting 1569 sequences from chr2\n",
      "\tExtracting 611 sequences from chr20\n",
      "\tExtracting 686 sequences from chr21\n",
      "\tExtracting 588 sequences from chr22\n",
      "\tExtracting 1255 sequences from chr3\n",
      "\tExtracting 792 sequences from chr4\n",
      "\tExtracting 953 sequences from chr5\n",
      "\tExtracting 1243 sequences from chr6\n",
      "\tExtracting 1214 sequences from chr7\n",
      "\tExtracting 793 sequences from chr8\n",
      "\tExtracting 1017 sequences from chr9\n",
      "\tExtracting 410 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24736 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2586 sequences from chr1\n",
      "\tExtracting 952 sequences from chr10\n",
      "\tExtracting 1313 sequences from chr11\n",
      "\tExtracting 1239 sequences from chr12\n",
      "\tExtracting 431 sequences from chr13\n",
      "\tExtracting 879 sequences from chr14\n",
      "\tExtracting 785 sequences from chr15\n",
      "\tExtracting 1135 sequences from chr16\n",
      "\tExtracting 1693 sequences from chr17\n",
      "\tExtracting 344 sequences from chr18\n",
      "\tExtracting 1908 sequences from chr19\n",
      "\tExtracting 1706 sequences from chr2\n",
      "\tExtracting 679 sequences from chr20\n",
      "\tExtracting 340 sequences from chr21\n",
      "\tExtracting 645 sequences from chr22\n",
      "\tExtracting 1338 sequences from chr3\n",
      "\tExtracting 812 sequences from chr4\n",
      "\tExtracting 1071 sequences from chr5\n",
      "\tExtracting 1476 sequences from chr6\n",
      "\tExtracting 1155 sequences from chr7\n",
      "\tExtracting 898 sequences from chr8\n",
      "\tExtracting 1002 sequences from chr9\n",
      "\tExtracting 568 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24955 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2614 sequences from chr1\n",
      "\tExtracting 1008 sequences from chr10\n",
      "\tExtracting 1322 sequences from chr11\n",
      "\tExtracting 1214 sequences from chr12\n",
      "\tExtracting 448 sequences from chr13\n",
      "\tExtracting 910 sequences from chr14\n",
      "\tExtracting 777 sequences from chr15\n",
      "\tExtracting 1193 sequences from chr16\n",
      "\tExtracting 1684 sequences from chr17\n",
      "\tExtracting 364 sequences from chr18\n",
      "\tExtracting 2022 sequences from chr19\n",
      "\tExtracting 1699 sequences from chr2\n",
      "\tExtracting 620 sequences from chr20\n",
      "\tExtracting 411 sequences from chr21\n",
      "\tExtracting 655 sequences from chr22\n",
      "\tExtracting 1316 sequences from chr3\n",
      "\tExtracting 793 sequences from chr4\n",
      "\tExtracting 1062 sequences from chr5\n",
      "\tExtracting 1383 sequences from chr6\n",
      "\tExtracting 1133 sequences from chr7\n",
      "\tExtracting 875 sequences from chr8\n",
      "\tExtracting 938 sequences from chr9\n",
      "\tExtracting 496 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24937 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2595 sequences from chr1\n",
      "\tExtracting 1029 sequences from chr10\n",
      "\tExtracting 1283 sequences from chr11\n",
      "\tExtracting 1262 sequences from chr12\n",
      "\tExtracting 455 sequences from chr13\n",
      "\tExtracting 865 sequences from chr14\n",
      "\tExtracting 808 sequences from chr15\n",
      "\tExtracting 1065 sequences from chr16\n",
      "\tExtracting 1649 sequences from chr17\n",
      "\tExtracting 356 sequences from chr18\n",
      "\tExtracting 1802 sequences from chr19\n",
      "\tExtracting 1654 sequences from chr2\n",
      "\tExtracting 670 sequences from chr20\n",
      "\tExtracting 332 sequences from chr21\n",
      "\tExtracting 709 sequences from chr22\n",
      "\tExtracting 1350 sequences from chr3\n",
      "\tExtracting 841 sequences from chr4\n",
      "\tExtracting 1027 sequences from chr5\n",
      "\tExtracting 1572 sequences from chr6\n",
      "\tExtracting 1205 sequences from chr7\n",
      "\tExtracting 821 sequences from chr8\n",
      "\tExtracting 995 sequences from chr9\n",
      "\tExtracting 625 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24970 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2600 sequences from chr1\n",
      "\tExtracting 853 sequences from chr10\n",
      "\tExtracting 1297 sequences from chr11\n",
      "\tExtracting 1303 sequences from chr12\n",
      "\tExtracting 405 sequences from chr13\n",
      "\tExtracting 827 sequences from chr14\n",
      "\tExtracting 650 sequences from chr15\n",
      "\tExtracting 1235 sequences from chr16\n",
      "\tExtracting 1649 sequences from chr17\n",
      "\tExtracting 302 sequences from chr18\n",
      "\tExtracting 2441 sequences from chr19\n",
      "\tExtracting 1582 sequences from chr2\n",
      "\tExtracting 630 sequences from chr20\n",
      "\tExtracting 594 sequences from chr21\n",
      "\tExtracting 613 sequences from chr22\n",
      "\tExtracting 1267 sequences from chr3\n",
      "\tExtracting 795 sequences from chr4\n",
      "\tExtracting 975 sequences from chr5\n",
      "\tExtracting 1355 sequences from chr6\n",
      "\tExtracting 1185 sequences from chr7\n",
      "\tExtracting 834 sequences from chr8\n",
      "\tExtracting 1021 sequences from chr9\n",
      "\tExtracting 395 sequences from chrX\n",
      "\tExtracting 3 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24811 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset3/wellington.tsv.gz tmp_static/Subset3/homer.tsv.gz tmp_static/Subset3/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/wellington.tsv.gz data/gene.bed tmp_static/Subset3/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset3/binding.tsv.gz tmp_static/Subset3/tssdist.tsv.gz tmp_static/Subset3/linking.tsv.gz\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset9/wellington.tsv.gz tmp_static/Subset9/homer.tsv.gz tmp_static/Subset9/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/wellington.tsv.gz data/gene.bed tmp_static/Subset9/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset3/linking.tsv.gz tmp_static/Subset3/binlinking.tsv.gz 20\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset3/expression.tsv.gz tmp_static/Subset3/binlinking.tsv.gz tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_loss.tsv.gz tmp_static/Subset3/net_stats.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset9/binding.tsv.gz tmp_static/Subset9/tssdist.tsv.gz tmp_static/Subset9/linking.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset14/footprints.bed data/motifs.motif data/genome tmp_static/Subset14/expression.tsv.gz tmp_static/Subset14/motifs.bed tmp_static/Subset14/wellington.tsv.gz tmp_static/Subset14/homer.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset9/linking.tsv.gz tmp_static/Subset9/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset9/expression.tsv.gz tmp_static/Subset9/binlinking.tsv.gz tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_loss.tsv.gz tmp_static/Subset9/net_stats.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2451 sequences from chr1\n",
      "\tExtracting 1089 sequences from chr10\n",
      "\tExtracting 1297 sequences from chr11\n",
      "\tExtracting 1280 sequences from chr12\n",
      "\tExtracting 497 sequences from chr13\n",
      "\tExtracting 831 sequences from chr14\n",
      "\tExtracting 747 sequences from chr15\n",
      "\tExtracting 1163 sequences from chr16\n",
      "\tExtracting 1591 sequences from chr17\n",
      "\tExtracting 366 sequences from chr18\n",
      "\tExtracting 1810 sequences from chr19\n",
      "\tExtracting 1771 sequences from chr2\n",
      "\tExtracting 711 sequences from chr20\n",
      "\tExtracting 356 sequences from chr21\n",
      "\tExtracting 656 sequences from chr22\n",
      "\tExtracting 1406 sequences from chr3\n",
      "\tExtracting 788 sequences from chr4\n",
      "\tExtracting 1050 sequences from chr5\n",
      "\tExtracting 1398 sequences from chr6\n",
      "\tExtracting 1185 sequences from chr7\n",
      "\tExtracting 904 sequences from chr8\n",
      "\tExtracting 1028 sequences from chr9\n",
      "\tExtracting 583 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24959 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2469 sequences from chr1\n",
      "\tExtracting 1131 sequences from chr10\n",
      "\tExtracting 1257 sequences from chr11\n",
      "\tExtracting 1266 sequences from chr12\n",
      "\tExtracting 448 sequences from chr13\n",
      "\tExtracting 835 sequences from chr14\n",
      "\tExtracting 815 sequences from chr15\n",
      "\tExtracting 1110 sequences from chr16\n",
      "\tExtracting 1611 sequences from chr17\n",
      "\tExtracting 429 sequences from chr18\n",
      "\tExtracting 1723 sequences from chr19\n",
      "\tExtracting 1767 sequences from chr2\n",
      "\tExtracting 735 sequences from chr20\n",
      "\tExtracting 325 sequences from chr21\n",
      "\tExtracting 644 sequences from chr22\n",
      "\tExtracting 1399 sequences from chr3\n",
      "\tExtracting 847 sequences from chr4\n",
      "\tExtracting 1048 sequences from chr5\n",
      "\tExtracting 1408 sequences from chr6\n",
      "\tExtracting 1170 sequences from chr7\n",
      "\tExtracting 885 sequences from chr8\n",
      "\tExtracting 1061 sequences from chr9\n",
      "\tExtracting 578 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24961 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 2567 sequences from chr1\n",
      "\tExtracting 1052 sequences from chr10\n",
      "\tExtracting 1290 sequences from chr11\n",
      "\tExtracting 1301 sequences from chr12\n",
      "\tExtracting 482 sequences from chr13\n",
      "\tExtracting 855 sequences from chr14\n",
      "\tExtracting 724 sequences from chr15\n",
      "\tExtracting 1182 sequences from chr16\n",
      "\tExtracting 1676 sequences from chr17\n",
      "\tExtracting 392 sequences from chr18\n",
      "\tExtracting 1857 sequences from chr19\n",
      "\tExtracting 1705 sequences from chr2\n",
      "\tExtracting 706 sequences from chr20\n",
      "\tExtracting 380 sequences from chr21\n",
      "\tExtracting 616 sequences from chr22\n",
      "\tExtracting 1399 sequences from chr3\n",
      "\tExtracting 847 sequences from chr4\n",
      "\tExtracting 1033 sequences from chr5\n",
      "\tExtracting 1323 sequences from chr6\n",
      "\tExtracting 1157 sequences from chr7\n",
      "\tExtracting 815 sequences from chr8\n",
      "\tExtracting 1080 sequences from chr9\n",
      "\tExtracting 504 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24943 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2613 sequences from chr1\n",
      "\tExtracting 910 sequences from chr10\n",
      "\tExtracting 1321 sequences from chr11\n",
      "\tExtracting 1268 sequences from chr12\n",
      "\tExtracting 405 sequences from chr13\n",
      "\tExtracting 814 sequences from chr14\n",
      "\tExtracting 684 sequences from chr15\n",
      "\tExtracting 1220 sequences from chr16\n",
      "\tExtracting 1771 sequences from chr17\n",
      "\tExtracting 329 sequences from chr18\n",
      "\tExtracting 2359 sequences from chr19\n",
      "\tExtracting 1610 sequences from chr2\n",
      "\tExtracting 611 sequences from chr20\n",
      "\tExtracting 529 sequences from chr21\n",
      "\tExtracting 596 sequences from chr22\n",
      "\tExtracting 1292 sequences from chr3\n",
      "\tExtracting 791 sequences from chr4\n",
      "\tExtracting 1001 sequences from chr5\n",
      "\tExtracting 1267 sequences from chr6\n",
      "\tExtracting 1206 sequences from chr7\n",
      "\tExtracting 803 sequences from chr8\n",
      "\tExtracting 1027 sequences from chr9\n",
      "\tExtracting 436 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24864 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset14/wellington.tsv.gz tmp_static/Subset14/homer.tsv.gz tmp_static/Subset14/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset14/expression.tsv.gz tmp_static/Subset14/wellington.tsv.gz data/gene.bed tmp_static/Subset14/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset14/binding.tsv.gz tmp_static/Subset14/tssdist.tsv.gz tmp_static/Subset14/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset14/linking.tsv.gz tmp_static/Subset14/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset14/expression.tsv.gz tmp_static/Subset14/binlinking.tsv.gz tmp_static/Subset14/net_weight.tsv.gz tmp_static/Subset14/net_meanvar.tsv.gz tmp_static/Subset14/net_covfactor.tsv.gz tmp_static/Subset14/net_loss.tsv.gz tmp_static/Subset14/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset12/footprints.bed data/motifs.motif data/genome tmp_static/Subset12/expression.tsv.gz tmp_static/Subset12/motifs.bed tmp_static/Subset12/wellington.tsv.gz tmp_static/Subset12/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2572 sequences from chr1\n",
      "\tExtracting 973 sequences from chr10\n",
      "\tExtracting 1321 sequences from chr11\n",
      "\tExtracting 1349 sequences from chr12\n",
      "\tExtracting 455 sequences from chr13\n",
      "\tExtracting 873 sequences from chr14\n",
      "\tExtracting 782 sequences from chr15\n",
      "\tExtracting 1165 sequences from chr16\n",
      "\tExtracting 1646 sequences from chr17\n",
      "\tExtracting 371 sequences from chr18\n",
      "\tExtracting 1822 sequences from chr19\n",
      "\tExtracting 1701 sequences from chr2\n",
      "\tExtracting 650 sequences from chr20\n",
      "\tExtracting 387 sequences from chr21\n",
      "\tExtracting 667 sequences from chr22\n",
      "\tExtracting 1329 sequences from chr3\n",
      "\tExtracting 801 sequences from chr4\n",
      "\tExtracting 1122 sequences from chr5\n",
      "\tExtracting 1346 sequences from chr6\n",
      "\tExtracting 1131 sequences from chr7\n",
      "\tExtracting 885 sequences from chr8\n",
      "\tExtracting 1020 sequences from chr9\n",
      "\tExtracting 579 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24947 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2670 sequences from chr1\n",
      "\tExtracting 1002 sequences from chr10\n",
      "\tExtracting 1321 sequences from chr11\n",
      "\tExtracting 1255 sequences from chr12\n",
      "\tExtracting 439 sequences from chr13\n",
      "\tExtracting 805 sequences from chr14\n",
      "\tExtracting 679 sequences from chr15\n",
      "\tExtracting 1167 sequences from chr16\n",
      "\tExtracting 1699 sequences from chr17\n",
      "\tExtracting 309 sequences from chr18\n",
      "\tExtracting 2136 sequences from chr19\n",
      "\tExtracting 1620 sequences from chr2\n",
      "\tExtracting 676 sequences from chr20\n",
      "\tExtracting 424 sequences from chr21\n",
      "\tExtracting 616 sequences from chr22\n",
      "\tExtracting 1287 sequences from chr3\n",
      "\tExtracting 804 sequences from chr4\n",
      "\tExtracting 1061 sequences from chr5\n",
      "\tExtracting 1399 sequences from chr6\n",
      "\tExtracting 1145 sequences from chr7\n",
      "\tExtracting 852 sequences from chr8\n",
      "\tExtracting 1015 sequences from chr9\n",
      "\tExtracting 535 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24917 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2500 sequences from chr1\n",
      "\tExtracting 1033 sequences from chr10\n",
      "\tExtracting 1357 sequences from chr11\n",
      "\tExtracting 1251 sequences from chr12\n",
      "\tExtracting 470 sequences from chr13\n",
      "\tExtracting 849 sequences from chr14\n",
      "\tExtracting 841 sequences from chr15\n",
      "\tExtracting 1111 sequences from chr16\n",
      "\tExtracting 1687 sequences from chr17\n",
      "\tExtracting 347 sequences from chr18\n",
      "\tExtracting 1764 sequences from chr19\n",
      "\tExtracting 1727 sequences from chr2\n",
      "\tExtracting 678 sequences from chr20\n",
      "\tExtracting 364 sequences from chr21\n",
      "\tExtracting 652 sequences from chr22\n",
      "\tExtracting 1363 sequences from chr3\n",
      "\tExtracting 821 sequences from chr4\n",
      "\tExtracting 1108 sequences from chr5\n",
      "\tExtracting 1382 sequences from chr6\n",
      "\tExtracting 1111 sequences from chr7\n",
      "\tExtracting 913 sequences from chr8\n",
      "\tExtracting 1023 sequences from chr9\n",
      "\tExtracting 603 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24955 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2595 sequences from chr1\n",
      "\tExtracting 920 sequences from chr10\n",
      "\tExtracting 1329 sequences from chr11\n",
      "\tExtracting 1267 sequences from chr12\n",
      "\tExtracting 414 sequences from chr13\n",
      "\tExtracting 875 sequences from chr14\n",
      "\tExtracting 673 sequences from chr15\n",
      "\tExtracting 1235 sequences from chr16\n",
      "\tExtracting 1783 sequences from chr17\n",
      "\tExtracting 316 sequences from chr18\n",
      "\tExtracting 2498 sequences from chr19\n",
      "\tExtracting 1568 sequences from chr2\n",
      "\tExtracting 630 sequences from chr20\n",
      "\tExtracting 477 sequences from chr21\n",
      "\tExtracting 591 sequences from chr22\n",
      "\tExtracting 1315 sequences from chr3\n",
      "\tExtracting 781 sequences from chr4\n",
      "\tExtracting 960 sequences from chr5\n",
      "\tExtracting 1231 sequences from chr6\n",
      "\tExtracting 1199 sequences from chr7\n",
      "\tExtracting 782 sequences from chr8\n",
      "\tExtracting 1002 sequences from chr9\n",
      "\tExtracting 408 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24850 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset12/wellington.tsv.gz tmp_static/Subset12/homer.tsv.gz tmp_static/Subset12/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset12/expression.tsv.gz tmp_static/Subset12/wellington.tsv.gz data/gene.bed tmp_static/Subset12/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset12/binding.tsv.gz tmp_static/Subset12/tssdist.tsv.gz tmp_static/Subset12/linking.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset12/linking.tsv.gz tmp_static/Subset12/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset12/expression.tsv.gz tmp_static/Subset12/binlinking.tsv.gz tmp_static/Subset12/net_weight.tsv.gz tmp_static/Subset12/net_meanvar.tsv.gz tmp_static/Subset12/net_covfactor.tsv.gz tmp_static/Subset12/net_loss.tsv.gz tmp_static/Subset12/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset2/footprints.bed data/motifs.motif data/genome tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/motifs.bed tmp_static/Subset2/wellington.tsv.gz tmp_static/Subset2/homer.tsv.gz\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2579 sequences from chr1\n",
      "\tExtracting 1006 sequences from chr10\n",
      "\tExtracting 1288 sequences from chr11\n",
      "\tExtracting 1331 sequences from chr12\n",
      "\tExtracting 479 sequences from chr13\n",
      "\tExtracting 809 sequences from chr14\n",
      "\tExtracting 788 sequences from chr15\n",
      "\tExtracting 1088 sequences from chr16\n",
      "\tExtracting 1647 sequences from chr17\n",
      "\tExtracting 334 sequences from chr18\n",
      "\tExtracting 1916 sequences from chr19\n",
      "\tExtracting 1771 sequences from chr2\n",
      "\tExtracting 705 sequences from chr20\n",
      "\tExtracting 364 sequences from chr21\n",
      "\tExtracting 640 sequences from chr22\n",
      "\tExtracting 1275 sequences from chr3\n",
      "\tExtracting 811 sequences from chr4\n",
      "\tExtracting 1090 sequences from chr5\n",
      "\tExtracting 1397 sequences from chr6\n",
      "\tExtracting 1199 sequences from chr7\n",
      "\tExtracting 870 sequences from chr8\n",
      "\tExtracting 1007 sequences from chr9\n",
      "\tExtracting 566 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24960 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2617 sequences from chr1\n",
      "\tExtracting 903 sequences from chr10\n",
      "\tExtracting 1323 sequences from chr11\n",
      "\tExtracting 1296 sequences from chr12\n",
      "\tExtracting 427 sequences from chr13\n",
      "\tExtracting 858 sequences from chr14\n",
      "\tExtracting 644 sequences from chr15\n",
      "\tExtracting 1222 sequences from chr16\n",
      "\tExtracting 1675 sequences from chr17\n",
      "\tExtracting 313 sequences from chr18\n",
      "\tExtracting 2461 sequences from chr19\n",
      "\tExtracting 1521 sequences from chr2\n",
      "\tExtracting 621 sequences from chr20\n",
      "\tExtracting 665 sequences from chr21\n",
      "\tExtracting 586 sequences from chr22\n",
      "\tExtracting 1239 sequences from chr3\n",
      "\tExtracting 762 sequences from chr4\n",
      "\tExtracting 990 sequences from chr5\n",
      "\tExtracting 1270 sequences from chr6\n",
      "\tExtracting 1206 sequences from chr7\n",
      "\tExtracting 802 sequences from chr8\n",
      "\tExtracting 995 sequences from chr9\n",
      "\tExtracting 387 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24785 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2642 sequences from chr1\n",
      "\tExtracting 932 sequences from chr10\n",
      "\tExtracting 1335 sequences from chr11\n",
      "\tExtracting 1300 sequences from chr12\n",
      "\tExtracting 468 sequences from chr13\n",
      "\tExtracting 897 sequences from chr14\n",
      "\tExtracting 737 sequences from chr15\n",
      "\tExtracting 1246 sequences from chr16\n",
      "\tExtracting 1644 sequences from chr17\n",
      "\tExtracting 316 sequences from chr18\n",
      "\tExtracting 2114 sequences from chr19\n",
      "\tExtracting 1664 sequences from chr2\n",
      "\tExtracting 621 sequences from chr20\n",
      "\tExtracting 443 sequences from chr21\n",
      "\tExtracting 631 sequences from chr22\n",
      "\tExtracting 1198 sequences from chr3\n",
      "\tExtracting 793 sequences from chr4\n",
      "\tExtracting 1032 sequences from chr5\n",
      "\tExtracting 1351 sequences from chr6\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 1163 sequences from chr7\n",
      "\tExtracting 855 sequences from chr8\n",
      "\tExtracting 996 sequences from chr9\n",
      "\tExtracting 545 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24923 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 18 (avg size of targets)\n",
      "\tBackground files for 18 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2528 sequences from chr1\n",
      "\tExtracting 987 sequences from chr10\n",
      "\tExtracting 1285 sequences from chr11\n",
      "\tExtracting 1286 sequences from chr12\n",
      "\tExtracting 542 sequences from chr13\n",
      "\tExtracting 825 sequences from chr14\n",
      "\tExtracting 776 sequences from chr15\n",
      "\tExtracting 1110 sequences from chr16\n",
      "\tExtracting 1715 sequences from chr17\n",
      "\tExtracting 360 sequences from chr18\n",
      "\tExtracting 1853 sequences from chr19\n",
      "\tExtracting 1672 sequences from chr2\n",
      "\tExtracting 718 sequences from chr20\n",
      "\tExtracting 362 sequences from chr21\n",
      "\tExtracting 616 sequences from chr22\n",
      "\tExtracting 1315 sequences from chr3\n",
      "\tExtracting 851 sequences from chr4\n",
      "\tExtracting 1067 sequences from chr5\n",
      "\tExtracting 1426 sequences from chr6\n",
      "\tExtracting 1197 sequences from chr7\n",
      "\tExtracting 874 sequences from chr8\n",
      "\tExtracting 980 sequences from chr9\n",
      "\tExtracting 625 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24971 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset5/footprints.bed data/motifs.motif data/genome tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/motifs.bed tmp_static/Subset5/wellington.tsv.gz tmp_static/Subset5/homer.tsv.gz\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset2/wellington.tsv.gz tmp_static/Subset2/homer.tsv.gz tmp_static/Subset2/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/wellington.tsv.gz data/gene.bed tmp_static/Subset2/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset2/binding.tsv.gz tmp_static/Subset2/tssdist.tsv.gz tmp_static/Subset2/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset2/linking.tsv.gz tmp_static/Subset2/binlinking.tsv.gz 20\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2456 sequences from chr1\n",
      "\tExtracting 1030 sequences from chr10\n",
      "\tExtracting 1345 sequences from chr11\n",
      "\tExtracting 1290 sequences from chr12\n",
      "\tExtracting 472 sequences from chr13\n",
      "\tExtracting 832 sequences from chr14\n",
      "\tExtracting 745 sequences from chr15\n",
      "\tExtracting 1131 sequences from chr16\n",
      "\tExtracting 1665 sequences from chr17\n",
      "\tExtracting 392 sequences from chr18\n",
      "\tExtracting 1952 sequences from chr19\n",
      "\tExtracting 1674 sequences from chr2\n",
      "\tExtracting 659 sequences from chr20\n",
      "\tExtracting 375 sequences from chr21\n",
      "\tExtracting 629 sequences from chr22\n",
      "\tExtracting 1344 sequences from chr3\n",
      "\tExtracting 844 sequences from chr4\n",
      "\tExtracting 1093 sequences from chr5\n",
      "\tExtracting 1397 sequences from chr6\n",
      "\tExtracting 1126 sequences from chr7\n",
      "\tExtracting 911 sequences from chr8\n",
      "\tExtracting 1052 sequences from chr9\n",
      "\tExtracting 532 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24947 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tExtracting 2478 sequences from chr1\n",
      "\tExtracting 1047 sequences from chr10\n",
      "\tExtracting 1315 sequences from chr11\n",
      "\tExtracting 1316 sequences from chr12\n",
      "\tExtracting 496 sequences from chr13\n",
      "\tExtracting 818 sequences from chr14\n",
      "\tExtracting 766 sequences from chr15\n",
      "\tExtracting 1073 sequences from chr16\n",
      "\tExtracting 1648 sequences from chr17\n",
      "\tExtracting 385 sequences from chr18\n",
      "\tExtracting 1803 sequences from chr19\n",
      "\tExtracting 1737 sequences from chr2\n",
      "\tExtracting 740 sequences from chr20\n",
      "\tExtracting 347 sequences from chr21\n",
      "\tExtracting 619 sequences from chr22\n",
      "\tExtracting 1391 sequences from chr3\n",
      "\tExtracting 808 sequences from chr4\n",
      "\tExtracting 1111 sequences from chr5\n",
      "\tExtracting 1418 sequences from chr6\n",
      "\tExtracting 1182 sequences from chr7\n",
      "\tExtracting 886 sequences from chr8\n",
      "\tExtracting 1000 sequences from chr9\n",
      "\tExtracting 567 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24951 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2559 sequences from chr1\n",
      "\tExtracting 1110 sequences from chr10\n",
      "\tExtracting 1289 sequences from chr11\n",
      "\tExtracting 1318 sequences from chr12\n",
      "\tExtracting 435 sequences from chr13\n",
      "\tExtracting 878 sequences from chr14\n",
      "\tExtracting 753 sequences from chr15\n",
      "\tExtracting 1090 sequences from chr16\n",
      "\tExtracting 1587 sequences from chr17\n",
      "\tExtracting 393 sequences from chr18\n",
      "\tExtracting 1629 sequences from chr19\n",
      "\tExtracting 1715 sequences from chr2\n",
      "\tExtracting 693 sequences from chr20\n",
      "\tExtracting 387 sequences from chr21\n",
      "\tExtracting 611 sequences from chr22\n",
      "\tExtracting 1367 sequences from chr3\n",
      "\tExtracting 887 sequences from chr4\n",
      "\tExtracting 1081 sequences from chr5\n",
      "\tExtracting 1364 sequences from chr6\n",
      "\tExtracting 1226 sequences from chr7\n",
      "\tExtracting 950 sequences from chr8\n",
      "\tExtracting 1033 sequences from chr9\n",
      "\tExtracting 598 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24953 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2603 sequences from chr1\n",
      "\tExtracting 998 sequences from chr10\n",
      "\tExtracting 1314 sequences from chr11\n",
      "\tExtracting 1261 sequences from chr12\n",
      "\tExtracting 397 sequences from chr13\n",
      "\tExtracting 834 sequences from chr14\n",
      "\tExtracting 702 sequences from chr15\n",
      "\tExtracting 1196 sequences from chr16\n",
      "\tExtracting 1779 sequences from chr17\n",
      "\tExtracting 339 sequences from chr18\n",
      "\tExtracting 2338 sequences from chr19\n",
      "\tExtracting 1639 sequences from chr2\n",
      "\tExtracting 661 sequences from chr20\n",
      "\tExtracting 335 sequences from chr21\n",
      "\tExtracting 617 sequences from chr22\n",
      "\tExtracting 1366 sequences from chr3\n",
      "\tExtracting 827 sequences from chr4\n",
      "\tExtracting 1000 sequences from chr5\n",
      "\tExtracting 1264 sequences from chr6\n",
      "\tExtracting 1205 sequences from chr7\n",
      "\tExtracting 815 sequences from chr8\n",
      "\tExtracting 1039 sequences from chr9\n",
      "\tExtracting 421 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24950 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset2/expression.tsv.gz tmp_static/Subset2/binlinking.tsv.gz tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_loss.tsv.gz tmp_static/Subset2/net_stats.tsv.gz\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset5/wellington.tsv.gz tmp_static/Subset5/homer.tsv.gz tmp_static/Subset5/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/wellington.tsv.gz data/gene.bed tmp_static/Subset5/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset5/binding.tsv.gz tmp_static/Subset5/tssdist.tsv.gz tmp_static/Subset5/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset5/linking.tsv.gz tmp_static/Subset5/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset5/expression.tsv.gz tmp_static/Subset5/binlinking.tsv.gz tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_loss.tsv.gz tmp_static/Subset5/net_stats.tsv.gz\n",
      "Reading BED File...\n",
      "Calculating footprints...\n",
      "Waiting for the last 30 jobs to finish...\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin homer --nth 4 tmp_static/Subset1/footprints.bed data/motifs.motif data/genome tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/motifs.bed tmp_static/Subset1/wellington.tsv.gz tmp_static/Subset1/homer.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tPosition file = 14-reform-split/aaaac\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaac\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2497 sequences from chr1\n",
      "\tExtracting 1036 sequences from chr10\n",
      "\tExtracting 1326 sequences from chr11\n",
      "\tExtracting 1240 sequences from chr12\n",
      "\tExtracting 470 sequences from chr13\n",
      "\tExtracting 840 sequences from chr14\n",
      "\tExtracting 804 sequences from chr15\n",
      "\tExtracting 1094 sequences from chr16\n",
      "\tExtracting 1676 sequences from chr17\n",
      "\tExtracting 379 sequences from chr18\n",
      "\tExtracting 1840 sequences from chr19\n",
      "\tExtracting 1679 sequences from chr2\n",
      "\tExtracting 710 sequences from chr20\n",
      "\tExtracting 358 sequences from chr21\n",
      "\tExtracting 640 sequences from chr22\n",
      "\tExtracting 1463 sequences from chr3\n",
      "\tExtracting 764 sequences from chr4\n",
      "\tExtracting 1132 sequences from chr5\n",
      "\tExtracting 1328 sequences from chr6\n",
      "\tExtracting 1151 sequences from chr7\n",
      "\tExtracting 895 sequences from chr8\n",
      "\tExtracting 1059 sequences from chr9\n",
      "\tExtracting 551 sequences from chrX\n",
      "\tExtracting 3 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24935 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaad\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaad\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 24997\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 24997\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2533 sequences from chr1\n",
      "\tExtracting 1077 sequences from chr10\n",
      "\tExtracting 1306 sequences from chr11\n",
      "\tExtracting 1317 sequences from chr12\n",
      "\tExtracting 472 sequences from chr13\n",
      "\tExtracting 753 sequences from chr14\n",
      "\tExtracting 821 sequences from chr15\n",
      "\tExtracting 1120 sequences from chr16\n",
      "\tExtracting 1653 sequences from chr17\n",
      "\tExtracting 373 sequences from chr18\n",
      "\tExtracting 1688 sequences from chr19\n",
      "\tExtracting 1683 sequences from chr2\n",
      "\tExtracting 732 sequences from chr20\n",
      "\tExtracting 348 sequences from chr21\n",
      "\tExtracting 642 sequences from chr22\n",
      "\tExtracting 1382 sequences from chr3\n",
      "\tExtracting 847 sequences from chr4\n",
      "\tExtracting 1065 sequences from chr5\n",
      "\tExtracting 1396 sequences from chr6\n",
      "\tExtracting 1206 sequences from chr7\n",
      "\tExtracting 922 sequences from chr8\n",
      "\tExtracting 1049 sequences from chr9\n",
      "\tExtracting 559 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24945 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaab\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaab\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n",
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 19 (avg size of targets)\n",
      "\tBackground files for 19 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2563 sequences from chr1\n",
      "\tExtracting 979 sequences from chr10\n",
      "\tExtracting 1315 sequences from chr11\n",
      "\tExtracting 1290 sequences from chr12\n",
      "\tExtracting 409 sequences from chr13\n",
      "\tExtracting 836 sequences from chr14\n",
      "\tExtracting 720 sequences from chr15\n",
      "\tExtracting 1182 sequences from chr16\n",
      "\tExtracting 1688 sequences from chr17\n",
      "\tExtracting 361 sequences from chr18\n",
      "\tExtracting 1950 sequences from chr19\n",
      "\tExtracting 1623 sequences from chr2\n",
      "\tExtracting 669 sequences from chr20\n",
      "\tExtracting 386 sequences from chr21\n",
      "\tExtracting 645 sequences from chr22\n",
      "\tExtracting 1388 sequences from chr3\n",
      "\tExtracting 840 sequences from chr4\n",
      "\tExtracting 1080 sequences from chr5\n",
      "\tExtracting 1327 sequences from chr6\n",
      "\tExtracting 1151 sequences from chr7\n",
      "\tExtracting 910 sequences from chr8\n",
      "\tExtracting 1063 sequences from chr9\n",
      "\tExtracting 552 sequences from chrX\n",
      "\tExtracting 1 sequences from chrY\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24928 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "\tPosition file = 14-reform-split/aaaaa\n",
      "\tGenome = /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\tOutput Directory = 15-motifscan/aaaaa\n",
      "\tUsing actual sizes of regions (-size given)\n",
      "\tFragment size set to given\n",
      "\tWill use repeat masked sequences\n",
      "\tWill find motif(s) in /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/motifs.motif\n",
      "\tUsing Custom Genome\n",
      "\tPeak/BED file conversion summary:\n",
      "\t\tBED/Header formatted lines: 25001\n",
      "\t\tpeakfile formatted lines: 0\n",
      "\n",
      "\tPeak File Statistics:\n",
      "\t\tTotal Peaks: 25001\n",
      "\t\tRedundant Peak IDs: 0\n",
      "\t\tPeaks lacking information: 0 (need at least 5 columns per peak)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\t\tPeaks with misformatted coordinates: 0 (should be integer)\n",
      "\t\tPeaks with misformatted strand: 0 (should be either +/- or 0/1)\n",
      "\n",
      "\tPeak file looks good!\n",
      "\n",
      "\tBackground fragment size set to 20 (avg size of targets)\n",
      "\tBackground files for 20 bp fragments found.\n",
      "\tCustom genome sequence directory: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome\n",
      "\n",
      "\tExtracting sequences from file: /mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa\n",
      "\tLooking for peak sequences in a single file (/mnt/iusers01/fatpou01/bmh01/msc-healthdatasci-2023-2024/z89953zj/models/dictys/data/genome/genome.fa)\n",
      "\tExtracting 2653 sequences from chr1\n",
      "\tExtracting 989 sequences from chr10\n",
      "\tExtracting 1289 sequences from chr11\n",
      "\tExtracting 1237 sequences from chr12\n",
      "\tExtracting 392 sequences from chr13\n",
      "\tExtracting 796 sequences from chr14\n",
      "\tExtracting 659 sequences from chr15\n",
      "\tExtracting 1209 sequences from chr16\n",
      "\tExtracting 1838 sequences from chr17\n",
      "\tExtracting 344 sequences from chr18\n",
      "\tExtracting 2411 sequences from chr19\n",
      "\tExtracting 1556 sequences from chr2\n",
      "\tExtracting 717 sequences from chr20\n",
      "\tExtracting 337 sequences from chr21\n",
      "\tExtracting 614 sequences from chr22\n",
      "\tExtracting 1331 sequences from chr3\n",
      "\tExtracting 833 sequences from chr4\n",
      "\tExtracting 978 sequences from chr5\n",
      "\tExtracting 1249 sequences from chr6\n",
      "\tExtracting 1212 sequences from chr7\n",
      "\tExtracting 827 sequences from chr8\n",
      "\tExtracting 1060 sequences from chr9\n",
      "\tExtracting 395 sequences from chrX\n",
      "\n",
      "\n",
      "\tReading input files...\n",
      "\t24926 total sequences read\n",
      "\t769 motifs loaded\n",
      "\tFinding instances of 769 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tCleaning up tmp files...\n",
      "\n",
      "\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binding  tmp_static/Subset1/wellington.tsv.gz tmp_static/Subset1/homer.tsv.gz tmp_static/Subset1/binding.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin tssdist  tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/wellington.tsv.gz data/gene.bed tmp_static/Subset1/tssdist.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin linking  tmp_static/Subset1/binding.tsv.gz tmp_static/Subset1/tssdist.tsv.gz tmp_static/Subset1/linking.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  chromatin binlinking  tmp_static/Subset1/linking.tsv.gz tmp_static/Subset1/binlinking.tsv.gz 20\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network reconstruct --device cpu --nth 4 tmp_static/Subset1/expression.tsv.gz tmp_static/Subset1/binlinking.tsv.gz tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_loss.tsv.gz tmp_static/Subset1/net_stats.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset17/net_weight.tsv.gz tmp_static/Subset17/net_meanvar.tsv.gz tmp_static/Subset17/net_covfactor.tsv.gz tmp_static/Subset17/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset17/net_meanvar.tsv.gz tmp_static/Subset17/net_weight.tsv.gz tmp_static/Subset17/net_covfactor.tsv.gz tmp_static/Subset17/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset17/net_iweight.tsv.gz tmp_static/Subset17/net_meanvar.tsv.gz tmp_static/Subset17/net_covfactor.tsv.gz tmp_static/Subset17/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset16/net_weight.tsv.gz tmp_static/Subset16/net_meanvar.tsv.gz tmp_static/Subset16/net_covfactor.tsv.gz tmp_static/Subset16/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset16/net_meanvar.tsv.gz tmp_static/Subset16/net_weight.tsv.gz tmp_static/Subset16/net_covfactor.tsv.gz tmp_static/Subset16/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset16/net_iweight.tsv.gz tmp_static/Subset16/net_meanvar.tsv.gz tmp_static/Subset16/net_covfactor.tsv.gz tmp_static/Subset16/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset15/net_weight.tsv.gz tmp_static/Subset15/net_meanvar.tsv.gz tmp_static/Subset15/net_covfactor.tsv.gz tmp_static/Subset15/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset15/net_meanvar.tsv.gz tmp_static/Subset15/net_weight.tsv.gz tmp_static/Subset15/net_covfactor.tsv.gz tmp_static/Subset15/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset15/net_iweight.tsv.gz tmp_static/Subset15/net_meanvar.tsv.gz tmp_static/Subset15/net_covfactor.tsv.gz tmp_static/Subset15/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset13/net_weight.tsv.gz tmp_static/Subset13/net_meanvar.tsv.gz tmp_static/Subset13/net_covfactor.tsv.gz tmp_static/Subset13/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset13/net_meanvar.tsv.gz tmp_static/Subset13/net_weight.tsv.gz tmp_static/Subset13/net_covfactor.tsv.gz tmp_static/Subset13/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset13/net_iweight.tsv.gz tmp_static/Subset13/net_meanvar.tsv.gz tmp_static/Subset13/net_covfactor.tsv.gz tmp_static/Subset13/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_weight.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset7/net_iweight.tsv.gz tmp_static/Subset7/net_meanvar.tsv.gz tmp_static/Subset7/net_covfactor.tsv.gz tmp_static/Subset7/net_inweight.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_weight.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset8/net_iweight.tsv.gz tmp_static/Subset8/net_meanvar.tsv.gz tmp_static/Subset8/net_covfactor.tsv.gz tmp_static/Subset8/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_weight.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset9/net_iweight.tsv.gz tmp_static/Subset9/net_meanvar.tsv.gz tmp_static/Subset9/net_covfactor.tsv.gz tmp_static/Subset9/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset10/net_weight.tsv.gz tmp_static/Subset10/net_meanvar.tsv.gz tmp_static/Subset10/net_covfactor.tsv.gz tmp_static/Subset10/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset10/net_meanvar.tsv.gz tmp_static/Subset10/net_weight.tsv.gz tmp_static/Subset10/net_covfactor.tsv.gz tmp_static/Subset10/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset10/net_iweight.tsv.gz tmp_static/Subset10/net_meanvar.tsv.gz tmp_static/Subset10/net_covfactor.tsv.gz tmp_static/Subset10/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset12/net_weight.tsv.gz tmp_static/Subset12/net_meanvar.tsv.gz tmp_static/Subset12/net_covfactor.tsv.gz tmp_static/Subset12/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset12/net_meanvar.tsv.gz tmp_static/Subset12/net_weight.tsv.gz tmp_static/Subset12/net_covfactor.tsv.gz tmp_static/Subset12/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset12/net_iweight.tsv.gz tmp_static/Subset12/net_meanvar.tsv.gz tmp_static/Subset12/net_covfactor.tsv.gz tmp_static/Subset12/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_weight.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_weight.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset3/net_iweight.tsv.gz tmp_static/Subset3/net_meanvar.tsv.gz tmp_static/Subset3/net_covfactor.tsv.gz tmp_static/Subset3/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset6/net_iweight.tsv.gz tmp_static/Subset6/net_meanvar.tsv.gz tmp_static/Subset6/net_covfactor.tsv.gz tmp_static/Subset6/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset11/net_weight.tsv.gz tmp_static/Subset11/net_meanvar.tsv.gz tmp_static/Subset11/net_covfactor.tsv.gz tmp_static/Subset11/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset11/net_meanvar.tsv.gz tmp_static/Subset11/net_weight.tsv.gz tmp_static/Subset11/net_covfactor.tsv.gz tmp_static/Subset11/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset11/net_iweight.tsv.gz tmp_static/Subset11/net_meanvar.tsv.gz tmp_static/Subset11/net_covfactor.tsv.gz tmp_static/Subset11/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_weight.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset4/net_iweight.tsv.gz tmp_static/Subset4/net_meanvar.tsv.gz tmp_static/Subset4/net_covfactor.tsv.gz tmp_static/Subset4/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset14/net_weight.tsv.gz tmp_static/Subset14/net_meanvar.tsv.gz tmp_static/Subset14/net_covfactor.tsv.gz tmp_static/Subset14/net_nweight.tsv.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset14/net_meanvar.tsv.gz tmp_static/Subset14/net_weight.tsv.gz tmp_static/Subset14/net_covfactor.tsv.gz tmp_static/Subset14/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset14/net_iweight.tsv.gz tmp_static/Subset14/net_meanvar.tsv.gz tmp_static/Subset14/net_covfactor.tsv.gz tmp_static/Subset14/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_weight.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset5/net_iweight.tsv.gz tmp_static/Subset5/net_meanvar.tsv.gz tmp_static/Subset5/net_covfactor.tsv.gz tmp_static/Subset5/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_weight.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset2/net_iweight.tsv.gz tmp_static/Subset2/net_meanvar.tsv.gz tmp_static/Subset2/net_covfactor.tsv.gz tmp_static/Subset2/net_inweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_nweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network indirect --nth 4 --fi_meanvar tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_weight.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_iweight.tsv.gz\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network normalize --nth 4 tmp_static/Subset1/net_iweight.tsv.gz tmp_static/Subset1/net_meanvar.tsv.gz tmp_static/Subset1/net_covfactor.tsv.gz tmp_static/Subset1/net_inweight.tsv.gz\n",
      "make: Nothing to be done for 'cpu'.\n",
      "make: Nothing to be done for 'gpu'.\n",
      "make: Nothing to be done for 'cpu'.\n",
      "mkdir -p output/\n",
      "OPENBLAS_NUM_THREADS=1 NUMEXPR_NUM_THREADS=1 MKL_NUM_THREADS=1 OPENBLAS_MAX_THREADS=1 NUMEXPR_MAX_THREADS=1 MKL_MAX_THREADS=1 python3 -m dictys  network tofile  data tmp_static data/subsets.txt output/static.h5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tCommand being timed: \"dictys_helper network_inference.sh -j 32 -J 1 static\"\n",
      "\tUser time (seconds): 1888306.40\n",
      "\tSystem time (seconds): 60382.78\n",
      "\tPercent of CPU this job got: 2316%\n",
      "\tElapsed (wall clock) time (h:mm:ss or m:ss): 23:22:18\n",
      "\tAverage shared text size (kbytes): 0\n",
      "\tAverage unshared data size (kbytes): 0\n",
      "\tAverage stack size (kbytes): 0\n",
      "\tAverage total size (kbytes): 0\n",
      "\tMaximum resident set size (kbytes): 6561684\n",
      "\tAverage resident set size (kbytes): 0\n",
      "\tMajor (requiring I/O) page faults: 2551\n",
      "\tMinor (reclaiming a frame) page faults: 3570822734\n",
      "\tVoluntary context switches: 370192451\n",
      "\tInvoluntary context switches: 223650176\n",
      "\tSwaps: 0\n",
      "\tFile system inputs: 268634144\n",
      "\tFile system outputs: 322552624\n",
      "\tSocket messages sent: 0\n",
      "\tSocket messages received: 0\n",
      "\tSignals delivered: 0\n",
      "\tPage size (bytes): 4096\n",
      "\tExit status: 0\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/usr/bin/time -v cd .; \n",
    "/usr/bin/time -v dictys_helper network_inference.sh -j 32 -J 1 static "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
